# This Makefile is for the Bio extension to perl.
#
# It was generated automatically by MakeMaker version
# 6.56 (Revision: 65600) from the contents of
# Makefile.PL. Don't edit this file, edit Makefile.PL instead.
#
#       ANY CHANGES MADE HERE WILL BE LOST!
#
#   MakeMaker ARGV: ()
#

#   MakeMaker Parameters:

#     ABSTRACT => q[Bioinformatics Toolkit]
#     AUTHOR => q[Bioperl Team (bioperl-l@bioperl.org)]
#     BUILD_REQUIRES => {  }
#     DIR => []
#     DISTNAME => q[bioperl]
#     NAME => q[Bio]
#     PL_FILES => { biodatabases.PL=>q[biodatabases.pod], biodesign.PL=>q[biodesign.pod], bioperl.PL=>q[bioperl.pod], bioscripts.PL=>q[bioscripts.pod] }
#     PREREQ_PM => {  }
#     VERSION => q[1.2.3]
#     dist => { DIST_DEFAULT=>q[all tardist], COMPRESS=>q[gzip -9f], SUFFIX=>q[.gz] }
#     realclean => { FILES=>q[./Seq.pm.bak] }

# --- MakeMaker post_initialize section:


# --- MakeMaker const_config section:

# These definitions are from config.sh (via /usr/lib/perl/5.12/Config.pm).
# They may have been overridden via Makefile.PL or on the command line.
AR = ar
CC = cc
CCCDLFLAGS = -fPIC
CCDLFLAGS = -Wl,-E
DLEXT = so
DLSRC = dl_dlopen.xs
EXE_EXT = 
FULL_AR = /usr/bin/ar
LD = cc
LDDLFLAGS = -shared -O2 -g -L/usr/local/lib -fstack-protector
LDFLAGS =  -fstack-protector -L/usr/local/lib
LIBC = 
LIB_EXT = .a
OBJ_EXT = .o
OSNAME = linux
OSVERS = 2.6.24-28-server
RANLIB = :
SITELIBEXP = /usr/local/share/perl/5.12.4
SITEARCHEXP = /usr/local/lib/perl/5.12.4
SO = so
VENDORARCHEXP = /usr/lib/perl5
VENDORLIBEXP = /usr/share/perl5


# --- MakeMaker constants section:
AR_STATIC_ARGS = cr
DIRFILESEP = /
DFSEP = $(DIRFILESEP)
NAME = Bio
NAME_SYM = Bio
VERSION = 1.2.3
VERSION_MACRO = VERSION
VERSION_SYM = 1_2_3
DEFINE_VERSION = -D$(VERSION_MACRO)=\"$(VERSION)\"
XS_VERSION = 1.2.3
XS_VERSION_MACRO = XS_VERSION
XS_DEFINE_VERSION = -D$(XS_VERSION_MACRO)=\"$(XS_VERSION)\"
INST_ARCHLIB = blib/arch
INST_SCRIPT = blib/script
INST_BIN = blib/bin
INST_LIB = blib/lib
INST_MAN1DIR = blib/man1
INST_MAN3DIR = blib/man3
MAN1EXT = 1p
MAN3EXT = 3pm
INSTALLDIRS = site
DESTDIR = 
PREFIX = /usr
PERLPREFIX = $(PREFIX)
SITEPREFIX = $(PREFIX)/local
VENDORPREFIX = $(PREFIX)
INSTALLPRIVLIB = $(PERLPREFIX)/share/perl/5.12
DESTINSTALLPRIVLIB = $(DESTDIR)$(INSTALLPRIVLIB)
INSTALLSITELIB = $(SITEPREFIX)/share/perl/5.12.4
DESTINSTALLSITELIB = $(DESTDIR)$(INSTALLSITELIB)
INSTALLVENDORLIB = $(VENDORPREFIX)/share/perl5
DESTINSTALLVENDORLIB = $(DESTDIR)$(INSTALLVENDORLIB)
INSTALLARCHLIB = $(PERLPREFIX)/lib/perl/5.12
DESTINSTALLARCHLIB = $(DESTDIR)$(INSTALLARCHLIB)
INSTALLSITEARCH = $(SITEPREFIX)/lib/perl/5.12.4
DESTINSTALLSITEARCH = $(DESTDIR)$(INSTALLSITEARCH)
INSTALLVENDORARCH = $(VENDORPREFIX)/lib/perl5
DESTINSTALLVENDORARCH = $(DESTDIR)$(INSTALLVENDORARCH)
INSTALLBIN = $(PERLPREFIX)/bin
DESTINSTALLBIN = $(DESTDIR)$(INSTALLBIN)
INSTALLSITEBIN = $(SITEPREFIX)/bin
DESTINSTALLSITEBIN = $(DESTDIR)$(INSTALLSITEBIN)
INSTALLVENDORBIN = $(VENDORPREFIX)/bin
DESTINSTALLVENDORBIN = $(DESTDIR)$(INSTALLVENDORBIN)
INSTALLSCRIPT = $(PERLPREFIX)/bin
DESTINSTALLSCRIPT = $(DESTDIR)$(INSTALLSCRIPT)
INSTALLSITESCRIPT = $(SITEPREFIX)/bin
DESTINSTALLSITESCRIPT = $(DESTDIR)$(INSTALLSITESCRIPT)
INSTALLVENDORSCRIPT = $(VENDORPREFIX)/bin
DESTINSTALLVENDORSCRIPT = $(DESTDIR)$(INSTALLVENDORSCRIPT)
INSTALLMAN1DIR = $(PERLPREFIX)/share/man/man1
DESTINSTALLMAN1DIR = $(DESTDIR)$(INSTALLMAN1DIR)
INSTALLSITEMAN1DIR = $(SITEPREFIX)/man/man1
DESTINSTALLSITEMAN1DIR = $(DESTDIR)$(INSTALLSITEMAN1DIR)
INSTALLVENDORMAN1DIR = $(VENDORPREFIX)/share/man/man1
DESTINSTALLVENDORMAN1DIR = $(DESTDIR)$(INSTALLVENDORMAN1DIR)
INSTALLMAN3DIR = $(PERLPREFIX)/share/man/man3
DESTINSTALLMAN3DIR = $(DESTDIR)$(INSTALLMAN3DIR)
INSTALLSITEMAN3DIR = $(SITEPREFIX)/man/man3
DESTINSTALLSITEMAN3DIR = $(DESTDIR)$(INSTALLSITEMAN3DIR)
INSTALLVENDORMAN3DIR = $(VENDORPREFIX)/share/man/man3
DESTINSTALLVENDORMAN3DIR = $(DESTDIR)$(INSTALLVENDORMAN3DIR)
PERL_LIB = /usr/share/perl/5.12
PERL_ARCHLIB = /usr/lib/perl/5.12
LIBPERL_A = libperl.a
FIRST_MAKEFILE = Makefile
MAKEFILE_OLD = Makefile.old
MAKE_APERL_FILE = Makefile.aperl
PERLMAINCC = $(CC)
PERL_INC = /usr/lib/perl/5.12/CORE
PERL = /usr/bin/perl
FULLPERL = /usr/bin/perl
ABSPERL = $(PERL)
PERLRUN = $(PERL)
FULLPERLRUN = $(FULLPERL)
ABSPERLRUN = $(ABSPERL)
PERLRUNINST = $(PERLRUN) "-I$(INST_ARCHLIB)" "-I$(INST_LIB)"
FULLPERLRUNINST = $(FULLPERLRUN) "-I$(INST_ARCHLIB)" "-I$(INST_LIB)"
ABSPERLRUNINST = $(ABSPERLRUN) "-I$(INST_ARCHLIB)" "-I$(INST_LIB)"
PERL_CORE = 0
PERM_DIR = 755
PERM_RW = 644
PERM_RWX = 755

MAKEMAKER   = /usr/share/perl/5.12/ExtUtils/MakeMaker.pm
MM_VERSION  = 6.56
MM_REVISION = 65600

# FULLEXT = Pathname for extension directory (eg Foo/Bar/Oracle).
# BASEEXT = Basename part of FULLEXT. May be just equal FULLEXT. (eg Oracle)
# PARENT_NAME = NAME without BASEEXT and no trailing :: (eg Foo::Bar)
# DLBASE  = Basename part of dynamic library. May be just equal BASEEXT.
MAKE = make
FULLEXT = Bio
BASEEXT = Bio
PARENT_NAME = 
DLBASE = $(BASEEXT)
VERSION_FROM = 
OBJECT = 
LDFROM = $(OBJECT)
LINKTYPE = dynamic
BOOTDEP = 

# Handy lists of source code files:
XS_FILES = 
C_FILES  = 
O_FILES  = 
H_FILES  = 
MAN1PODS = 
MAN3PODS = Bio/Align/AlignI.pm \
	Bio/Align/DNAStatistics.pm \
	Bio/Align/PairwiseStatistics.pm \
	Bio/Align/StatisticsI.pm \
	Bio/Align/Utilities.pm \
	Bio/AlignIO.pm \
	Bio/AlignIO/bl2seq.pm \
	Bio/AlignIO/clustalw.pm \
	Bio/AlignIO/emboss.pm \
	Bio/AlignIO/fasta.pm \
	Bio/AlignIO/mase.pm \
	Bio/AlignIO/mega.pm \
	Bio/AlignIO/meme.pm \
	Bio/AlignIO/msf.pm \
	Bio/AlignIO/nexus.pm \
	Bio/AlignIO/pfam.pm \
	Bio/AlignIO/phylip.pm \
	Bio/AlignIO/prodom.pm \
	Bio/AlignIO/psi.pm \
	Bio/AlignIO/selex.pm \
	Bio/AlignIO/stockholm.pm \
	Bio/AnalysisI.pm \
	Bio/AnalysisParserI.pm \
	Bio/AnalysisResultI.pm \
	Bio/AnnotatableI.pm \
	Bio/Annotation/AnnotationFactory.pm \
	Bio/Annotation/Collection.pm \
	Bio/Annotation/Comment.pm \
	Bio/Annotation/DBLink.pm \
	Bio/Annotation/OntologyTerm.pm \
	Bio/Annotation/Reference.pm \
	Bio/Annotation/SimpleValue.pm \
	Bio/Annotation/StructuredValue.pm \
	Bio/Annotation/TypeManager.pm \
	Bio/AnnotationCollectionI.pm \
	Bio/AnnotationI.pm \
	Bio/Assembly/Contig.pm \
	Bio/Assembly/ContigAnalysis.pm \
	Bio/Assembly/IO.pm \
	Bio/Assembly/IO/ace.pm \
	Bio/Assembly/IO/phrap.pm \
	Bio/Assembly/Scaffold.pm \
	Bio/Assembly/ScaffoldI.pm \
	Bio/Biblio.pm \
	Bio/Biblio/Article.pm \
	Bio/Biblio/BiblioBase.pm \
	Bio/Biblio/Book.pm \
	Bio/Biblio/BookArticle.pm \
	Bio/Biblio/IO.pm \
	Bio/Biblio/IO/medline2ref.pm \
	Bio/Biblio/IO/medlinexml.pm \
	Bio/Biblio/IO/pubmed2ref.pm \
	Bio/Biblio/IO/pubmedxml.pm \
	Bio/Biblio/Journal.pm \
	Bio/Biblio/JournalArticle.pm \
	Bio/Biblio/MedlineArticle.pm \
	Bio/Biblio/MedlineBook.pm \
	Bio/Biblio/MedlineBookArticle.pm \
	Bio/Biblio/MedlineJournal.pm \
	Bio/Biblio/MedlineJournalArticle.pm \
	Bio/Biblio/Organisation.pm \
	Bio/Biblio/Patent.pm \
	Bio/Biblio/Person.pm \
	Bio/Biblio/Proceeding.pm \
	Bio/Biblio/Provider.pm \
	Bio/Biblio/PubmedArticle.pm \
	Bio/Biblio/PubmedBookArticle.pm \
	Bio/Biblio/PubmedJournalArticle.pm \
	Bio/Biblio/Ref.pm \
	Bio/Biblio/Service.pm \
	Bio/Biblio/TechReport.pm \
	Bio/Biblio/Thesis.pm \
	Bio/Biblio/WebResource.pm \
	Bio/Cluster/ClusterFactory.pm \
	Bio/Cluster/FamilyI.pm \
	Bio/Cluster/SequenceFamily.pm \
	Bio/Cluster/UniGene.pm \
	Bio/Cluster/UniGeneI.pm \
	Bio/ClusterI.pm \
	Bio/ClusterIO.pm \
	Bio/ClusterIO/dbsnp.pm \
	Bio/ClusterIO/unigene.pm \
	Bio/Coordinate/Chain.pm \
	Bio/Coordinate/Collection.pm \
	Bio/Coordinate/ExtrapolatingPair.pm \
	Bio/Coordinate/GeneMapper.pm \
	Bio/Coordinate/Graph.pm \
	Bio/Coordinate/MapperI.pm \
	Bio/Coordinate/Pair.pm \
	Bio/Coordinate/Result.pm \
	Bio/Coordinate/Result/Gap.pm \
	Bio/Coordinate/Result/Match.pm \
	Bio/Coordinate/ResultI.pm \
	Bio/Coordinate/Utils.pm \
	Bio/DB/Ace.pm \
	Bio/DB/Biblio/biofetch.pm \
	Bio/DB/Biblio/soap.pm \
	Bio/DB/BiblioI.pm \
	Bio/DB/BioFetch.pm \
	Bio/DB/DBFetch.pm \
	Bio/DB/EMBL.pm \
	Bio/DB/Failover.pm \
	Bio/DB/Fasta.pm \
	Bio/DB/FileCache.pm \
	Bio/DB/Flat.pm \
	Bio/DB/Flat/BDB.pm \
	Bio/DB/Flat/BDB/embl.pm \
	Bio/DB/Flat/BDB/fasta.pm \
	Bio/DB/Flat/OBDAIndex.pm \
	Bio/DB/GDB.pm \
	Bio/DB/GFF.pm \
	Bio/DB/GFF/Adaptor/ace.pm \
	Bio/DB/GFF/Adaptor/biofetch.pm \
	Bio/DB/GFF/Adaptor/dbi.pm \
	Bio/DB/GFF/Adaptor/dbi/caching_handle.pm \
	Bio/DB/GFF/Adaptor/dbi/iterator.pm \
	Bio/DB/GFF/Adaptor/dbi/mysql.pm \
	Bio/DB/GFF/Adaptor/dbi/mysqlace.pm \
	Bio/DB/GFF/Adaptor/dbi/mysqlopt.pm \
	Bio/DB/GFF/Adaptor/dbi/oracle.pm \
	Bio/DB/GFF/Adaptor/dbi/oracleace.pm \
	Bio/DB/GFF/Adaptor/memory.pm \
	Bio/DB/GFF/Adaptor/memory_iterator.pm \
	Bio/DB/GFF/Aggregator.pm \
	Bio/DB/GFF/Aggregator/alignment.pm \
	Bio/DB/GFF/Aggregator/clone.pm \
	Bio/DB/GFF/Aggregator/coding.pm \
	Bio/DB/GFF/Aggregator/match.pm \
	Bio/DB/GFF/Aggregator/none.pm \
	Bio/DB/GFF/Aggregator/processed_transcript.pm \
	Bio/DB/GFF/Aggregator/transcript.pm \
	Bio/DB/GFF/Aggregator/ucsc_acembly.pm \
	Bio/DB/GFF/Aggregator/ucsc_ensgene.pm \
	Bio/DB/GFF/Aggregator/ucsc_genscan.pm \
	Bio/DB/GFF/Aggregator/ucsc_refgene.pm \
	Bio/DB/GFF/Aggregator/ucsc_sanger22.pm \
	Bio/DB/GFF/Aggregator/ucsc_sanger22pseudo.pm \
	Bio/DB/GFF/Aggregator/ucsc_softberry.pm \
	Bio/DB/GFF/Aggregator/ucsc_twinscan.pm \
	Bio/DB/GFF/Aggregator/ucsc_unigene.pm \
	Bio/DB/GFF/Featname.pm \
	Bio/DB/GFF/Feature.pm \
	Bio/DB/GFF/Homol.pm \
	Bio/DB/GFF/RelSegment.pm \
	Bio/DB/GFF/Segment.pm \
	Bio/DB/GFF/Typename.pm \
	Bio/DB/GFF/Util/Binning.pm \
	Bio/DB/GFF/Util/Rearrange.pm \
	Bio/DB/GenBank.pm \
	Bio/DB/GenPept.pm \
	Bio/DB/InMemoryCache.pm \
	Bio/DB/NCBIHelper.pm \
	Bio/DB/Query/GenBank.pm \
	Bio/DB/Query/WebQuery.pm \
	Bio/DB/QueryI.pm \
	Bio/DB/RandomAccessI.pm \
	Bio/DB/RefSeq.pm \
	Bio/DB/Registry.pm \
	Bio/DB/SeqI.pm \
	Bio/DB/SwissProt.pm \
	Bio/DB/Universal.pm \
	Bio/DB/UpdateableSeqI.pm \
	Bio/DB/WebDBSeqI.pm \
	Bio/DB/XEMBL.pm \
	Bio/DB/XEMBLService.pm \
	Bio/DBLinkContainerI.pm \
	Bio/Das/FeatureTypeI.pm \
	Bio/Das/SegmentI.pm \
	Bio/DasI.pm \
	Bio/DescribableI.pm \
	Bio/Event/EventGeneratorI.pm \
	Bio/Event/EventHandlerI.pm \
	Bio/Expression/FeatureGroup.pm \
	Bio/Expression/FeatureI.pm \
	Bio/Expression/ProbeI.pm \
	Bio/Factory/AnalysisI.pm \
	Bio/Factory/ApplicationFactoryI.pm \
	Bio/Factory/BlastHitFactory.pm \
	Bio/Factory/BlastResultFactory.pm \
	Bio/Factory/DriverFactory.pm \
	Bio/Factory/EMBOSS.pm \
	Bio/Factory/FTLocationFactory.pm \
	Bio/Factory/HitFactoryI.pm \
	Bio/Factory/LocationFactoryI.pm \
	Bio/Factory/MapFactoryI.pm \
	Bio/Factory/ObjectBuilderI.pm \
	Bio/Factory/ObjectFactory.pm \
	Bio/Factory/ObjectFactoryI.pm \
	Bio/Factory/ResultFactoryI.pm \
	Bio/Factory/SeqAnalysisParserFactory.pm \
	Bio/Factory/SeqAnalysisParserFactoryI.pm \
	Bio/Factory/SequenceFactoryI.pm \
	Bio/Factory/SequenceProcessorI.pm \
	Bio/Factory/SequenceStreamI.pm \
	Bio/Factory/TreeFactoryI.pm \
	Bio/FeatureHolderI.pm \
	Bio/Graphics.pm \
	Bio/Graphics/Feature.pm \
	Bio/Graphics/FeatureFile.pm \
	Bio/Graphics/Glyph.pm \
	Bio/Graphics/Glyph/Factory.pm \
	Bio/Graphics/Glyph/alignment.pm \
	Bio/Graphics/Glyph/anchored_arrow.pm \
	Bio/Graphics/Glyph/arrow.pm \
	Bio/Graphics/Glyph/box.pm \
	Bio/Graphics/Glyph/cds.pm \
	Bio/Graphics/Glyph/crossbox.pm \
	Bio/Graphics/Glyph/diamond.pm \
	Bio/Graphics/Glyph/dna.pm \
	Bio/Graphics/Glyph/dot.pm \
	Bio/Graphics/Glyph/ellipse.pm \
	Bio/Graphics/Glyph/ex.pm \
	Bio/Graphics/Glyph/extending_arrow.pm \
	Bio/Graphics/Glyph/generic.pm \
	Bio/Graphics/Glyph/graded_segments.pm \
	Bio/Graphics/Glyph/group.pm \
	Bio/Graphics/Glyph/heterogeneous_segments.pm \
	Bio/Graphics/Glyph/line.pm \
	Bio/Graphics/Glyph/oval.pm \
	Bio/Graphics/Glyph/pinsertion.pm \
	Bio/Graphics/Glyph/primers.pm \
	Bio/Graphics/Glyph/processed_transcript.pm \
	Bio/Graphics/Glyph/redgreen_box.pm \
	Bio/Graphics/Glyph/redgreen_segment.pm \
	Bio/Graphics/Glyph/rndrect.pm \
	Bio/Graphics/Glyph/ruler_arrow.pm \
	Bio/Graphics/Glyph/segmented_keyglyph.pm \
	Bio/Graphics/Glyph/segments.pm \
	Bio/Graphics/Glyph/span.pm \
	Bio/Graphics/Glyph/splice_site.pm \
	Bio/Graphics/Glyph/toomany.pm \
	Bio/Graphics/Glyph/track.pm \
	Bio/Graphics/Glyph/transcript.pm \
	Bio/Graphics/Glyph/transcript2.pm \
	Bio/Graphics/Glyph/translation.pm \
	Bio/Graphics/Glyph/triangle.pm \
	Bio/Graphics/Glyph/xyplot.pm \
	Bio/Graphics/Panel.pm \
	Bio/Graphics/Util.pm \
	Bio/IdCollectionI.pm \
	Bio/IdentifiableI.pm \
	Bio/Index/Abstract.pm \
	Bio/Index/AbstractSeq.pm \
	Bio/Index/Blast.pm \
	Bio/Index/EMBL.pm \
	Bio/Index/Fasta.pm \
	Bio/Index/Fastq.pm \
	Bio/Index/GenBank.pm \
	Bio/Index/SwissPfam.pm \
	Bio/Index/Swissprot.pm \
	Bio/LiveSeq/AARange.pm \
	Bio/LiveSeq/Chain.pm \
	Bio/LiveSeq/ChainI.pm \
	Bio/LiveSeq/DNA.pm \
	Bio/LiveSeq/Exon.pm \
	Bio/LiveSeq/Gene.pm \
	Bio/LiveSeq/IO/BioPerl.pm \
	Bio/LiveSeq/IO/Loader.pm \
	Bio/LiveSeq/IO/SRS.pm \
	Bio/LiveSeq/Intron.pm \
	Bio/LiveSeq/Mutation.pm \
	Bio/LiveSeq/Mutator.pm \
	Bio/LiveSeq/Prim_Transcript.pm \
	Bio/LiveSeq/Range.pm \
	Bio/LiveSeq/Repeat_Region.pm \
	Bio/LiveSeq/Repeat_Unit.pm \
	Bio/LiveSeq/SeqI.pm \
	Bio/LiveSeq/Transcript.pm \
	Bio/LiveSeq/Translation.pm \
	Bio/LocatableSeq.pm \
	Bio/Location/Atomic.pm \
	Bio/Location/AvWithinCoordPolicy.pm \
	Bio/Location/CoordinatePolicyI.pm \
	Bio/Location/Fuzzy.pm \
	Bio/Location/FuzzyLocationI.pm \
	Bio/Location/NarrowestCoordPolicy.pm \
	Bio/Location/Simple.pm \
	Bio/Location/Split.pm \
	Bio/Location/SplitLocationI.pm \
	Bio/Location/WidestCoordPolicy.pm \
	Bio/LocationI.pm \
	Bio/Map/CytoMap.pm \
	Bio/Map/CytoMarker.pm \
	Bio/Map/CytoPosition.pm \
	Bio/Map/LinkageMap.pm \
	Bio/Map/LinkagePosition.pm \
	Bio/Map/MapI.pm \
	Bio/Map/MappableI.pm \
	Bio/Map/Marker.pm \
	Bio/Map/MarkerI.pm \
	Bio/Map/Microsatellite.pm \
	Bio/Map/OrderedPosition.pm \
	Bio/Map/OrderedPositionWithDistance.pm \
	Bio/Map/Position.pm \
	Bio/Map/PositionI.pm \
	Bio/Map/SimpleMap.pm \
	Bio/MapIO.pm \
	Bio/MapIO/mapmaker.pm \
	Bio/Matrix/PhylipDist.pm \
	Bio/Ontology/GOterm.pm \
	Bio/Ontology/InterProTerm.pm \
	Bio/Ontology/Ontology.pm \
	Bio/Ontology/OntologyEngineI.pm \
	Bio/Ontology/OntologyI.pm \
	Bio/Ontology/OntologyStore.pm \
	Bio/Ontology/Path.pm \
	Bio/Ontology/PathI.pm \
	Bio/Ontology/Relationship.pm \
	Bio/Ontology/RelationshipFactory.pm \
	Bio/Ontology/RelationshipI.pm \
	Bio/Ontology/RelationshipType.pm \
	Bio/Ontology/SimpleGOEngine.pm \
	Bio/Ontology/SimpleOntologyEngine.pm \
	Bio/Ontology/Term.pm \
	Bio/Ontology/TermFactory.pm \
	Bio/Ontology/TermI.pm \
	Bio/OntologyIO.pm \
	Bio/OntologyIO/Handlers/InterProHandler.pm \
	Bio/OntologyIO/InterProParser.pm \
	Bio/OntologyIO/dagflat.pm \
	Bio/OntologyIO/goflat.pm \
	Bio/OntologyIO/soflat.pm \
	Bio/Perl.pm \
	Bio/Phenotype/Correlate.pm \
	Bio/Phenotype/Measure.pm \
	Bio/Phenotype/OMIM/MiniMIMentry.pm \
	Bio/Phenotype/OMIM/OMIMentry.pm \
	Bio/Phenotype/OMIM/OMIMentryAllelicVariant.pm \
	Bio/Phenotype/OMIM/OMIMparser.pm \
	Bio/Phenotype/Phenotype.pm \
	Bio/Phenotype/PhenotypeI.pm \
	Bio/PrimarySeq.pm \
	Bio/PrimarySeqI.pm \
	Bio/Range.pm \
	Bio/RangeI.pm \
	Bio/Root/Err.pm \
	Bio/Root/Exception.pm \
	Bio/Root/Global.pm \
	Bio/Root/HTTPget.pm \
	Bio/Root/IO.pm \
	Bio/Root/IOManager.pm \
	Bio/Root/Object.pm \
	Bio/Root/Root.pm \
	Bio/Root/RootI.pm \
	Bio/Root/Utilities.pm \
	Bio/Root/Vector.pm \
	Bio/Root/Xref.pm \
	Bio/Search/BlastUtils.pm \
	Bio/Search/DatabaseI.pm \
	Bio/Search/GenericDatabase.pm \
	Bio/Search/HSP/BlastHSP.pm \
	Bio/Search/HSP/FastaHSP.pm \
	Bio/Search/HSP/GenericHSP.pm \
	Bio/Search/HSP/HMMERHSP.pm \
	Bio/Search/HSP/HSPFactory.pm \
	Bio/Search/HSP/HSPI.pm \
	Bio/Search/HSP/WABAHSP.pm \
	Bio/Search/Hit/BlastHit.pm \
	Bio/Search/Hit/Fasta.pm \
	Bio/Search/Hit/GenericHit.pm \
	Bio/Search/Hit/HMMERHit.pm \
	Bio/Search/Hit/HitFactory.pm \
	Bio/Search/Hit/HitI.pm \
	Bio/Search/Processor.pm \
	Bio/Search/Result/BlastResult.pm \
	Bio/Search/Result/GenericResult.pm \
	Bio/Search/Result/HMMERResult.pm \
	Bio/Search/Result/ResultFactory.pm \
	Bio/Search/Result/ResultI.pm \
	Bio/Search/Result/WABAResult.pm \
	Bio/Search/SearchUtils.pm \
	Bio/SearchDist.pm \
	Bio/SearchIO.pm \
	Bio/SearchIO/EventHandlerI.pm \
	Bio/SearchIO/FastHitEventBuilder.pm \
	Bio/SearchIO/SearchResultEventBuilder.pm \
	Bio/SearchIO/SearchWriterI.pm \
	Bio/SearchIO/Writer/HSPTableWriter.pm \
	Bio/SearchIO/Writer/HTMLResultWriter.pm \
	Bio/SearchIO/Writer/HitTableWriter.pm \
	Bio/SearchIO/Writer/ResultTableWriter.pm \
	Bio/SearchIO/Writer/TextResultWriter.pm \
	Bio/SearchIO/blast.pm \
	Bio/SearchIO/blastxml.pm \
	Bio/SearchIO/chado.pm \
	Bio/SearchIO/chadosxpr.pm \
	Bio/SearchIO/exonerate.pm \
	Bio/SearchIO/fasta.pm \
	Bio/SearchIO/hmmer.pm \
	Bio/SearchIO/psiblast.pm \
	Bio/SearchIO/waba.pm \
	Bio/Seq.pm \
	Bio/Seq/BaseSeqProcessor.pm \
	Bio/Seq/EncodedSeq.pm \
	Bio/Seq/LargePrimarySeq.pm \
	Bio/Seq/LargeSeq.pm \
	Bio/Seq/PrimaryQual.pm \
	Bio/Seq/PrimedSeq.pm \
	Bio/Seq/QualI.pm \
	Bio/Seq/RichSeq.pm \
	Bio/Seq/RichSeqI.pm \
	Bio/Seq/SeqBuilder.pm \
	Bio/Seq/SeqFactory.pm \
	Bio/Seq/SeqFastaSpeedFactory.pm \
	Bio/Seq/SeqWithQuality.pm \
	Bio/Seq/SequenceTrace.pm \
	Bio/Seq/TraceI.pm \
	Bio/SeqAnalysisParserI.pm \
	Bio/SeqFeature/AnnotationAdaptor.pm \
	Bio/SeqFeature/Collection.pm \
	Bio/SeqFeature/CollectionI.pm \
	Bio/SeqFeature/Computation.pm \
	Bio/SeqFeature/FeaturePair.pm \
	Bio/SeqFeature/Gene/Exon.pm \
	Bio/SeqFeature/Gene/ExonI.pm \
	Bio/SeqFeature/Gene/GeneStructure.pm \
	Bio/SeqFeature/Gene/GeneStructureI.pm \
	Bio/SeqFeature/Gene/Intron.pm \
	Bio/SeqFeature/Gene/NC_Feature.pm \
	Bio/SeqFeature/Gene/Poly_A_site.pm \
	Bio/SeqFeature/Gene/Promoter.pm \
	Bio/SeqFeature/Gene/Transcript.pm \
	Bio/SeqFeature/Gene/TranscriptI.pm \
	Bio/SeqFeature/Gene/UTR.pm \
	Bio/SeqFeature/Generic.pm \
	Bio/SeqFeature/PositionProxy.pm \
	Bio/SeqFeature/Primer.pm \
	Bio/SeqFeature/Similarity.pm \
	Bio/SeqFeature/SimilarityPair.pm \
	Bio/SeqFeatureI.pm \
	Bio/SeqI.pm \
	Bio/SeqIO.pm \
	Bio/SeqIO/FTHelper.pm \
	Bio/SeqIO/MultiFile.pm \
	Bio/SeqIO/abi.pm \
	Bio/SeqIO/ace.pm \
	Bio/SeqIO/alf.pm \
	Bio/SeqIO/bsml.pm \
	Bio/SeqIO/chado.pm \
	Bio/SeqIO/chadoitext.pm \
	Bio/SeqIO/chadosxpr.pm \
	Bio/SeqIO/chadoxml.pm \
	Bio/SeqIO/ctf.pm \
	Bio/SeqIO/embl.pm \
	Bio/SeqIO/exp.pm \
	Bio/SeqIO/fasta.pm \
	Bio/SeqIO/fastq.pm \
	Bio/SeqIO/game.pm \
	Bio/SeqIO/game/featureHandler.pm \
	Bio/SeqIO/game/idHandler.pm \
	Bio/SeqIO/game/seqHandler.pm \
	Bio/SeqIO/gcg.pm \
	Bio/SeqIO/genbank.pm \
	Bio/SeqIO/largefasta.pm \
	Bio/SeqIO/locuslink.pm \
	Bio/SeqIO/phd.pm \
	Bio/SeqIO/pir.pm \
	Bio/SeqIO/pln.pm \
	Bio/SeqIO/qual.pm \
	Bio/SeqIO/raw.pm \
	Bio/SeqIO/scf.pm \
	Bio/SeqIO/swiss.pm \
	Bio/SeqIO/ztr.pm \
	Bio/SeqUtils.pm \
	Bio/SimpleAlign.pm \
	Bio/Species.pm \
	Bio/Structure/Atom.pm \
	Bio/Structure/Chain.pm \
	Bio/Structure/Entry.pm \
	Bio/Structure/IO.pm \
	Bio/Structure/IO/pdb.pm \
	Bio/Structure/Model.pm \
	Bio/Structure/Residue.pm \
	Bio/Structure/SecStr/DSSP/Res.pm \
	Bio/Structure/SecStr/STRIDE/Res.pm \
	Bio/Structure/StructureI.pm \
	Bio/Symbol/Alphabet.pm \
	Bio/Symbol/AlphabetI.pm \
	Bio/Symbol/DNAAlphabet.pm \
	Bio/Symbol/ProteinAlphabet.pm \
	Bio/Symbol/Symbol.pm \
	Bio/Symbol/SymbolI.pm \
	Bio/Taxonomy.pm \
	Bio/Taxonomy/Taxon.pm \
	Bio/Taxonomy/Tree.pm \
	Bio/Tools/AlignFactory.pm \
	Bio/Tools/Alignment/Consed.pm \
	Bio/Tools/Alignment/Trim.pm \
	Bio/Tools/AnalysisResult.pm \
	Bio/Tools/BPbl2seq.pm \
	Bio/Tools/BPlite.pm \
	Bio/Tools/BPlite/HSP.pm \
	Bio/Tools/BPlite/Iteration.pm \
	Bio/Tools/BPlite/Sbjct.pm \
	Bio/Tools/BPpsilite.pm \
	Bio/Tools/Blast.pm \
	Bio/Tools/Blast/HSP.pm \
	Bio/Tools/Blast/HTML.pm \
	Bio/Tools/Blast/Sbjct.pm \
	Bio/Tools/CodonTable.pm \
	Bio/Tools/Coil.pm \
	Bio/Tools/ECnumber.pm \
	Bio/Tools/EPCR.pm \
	Bio/Tools/ESTScan.pm \
	Bio/Tools/Eponine.pm \
	Bio/Tools/Est2Genome.pm \
	Bio/Tools/FootPrinter.pm \
	Bio/Tools/GFF.pm \
	Bio/Tools/Gel.pm \
	Bio/Tools/Genemark.pm \
	Bio/Tools/Genewise.pm \
	Bio/Tools/Genomewise.pm \
	Bio/Tools/Genscan.pm \
	Bio/Tools/Grail.pm \
	Bio/Tools/HMMER/Domain.pm \
	Bio/Tools/HMMER/Results.pm \
	Bio/Tools/HMMER/Set.pm \
	Bio/Tools/Hmmpfam.pm \
	Bio/Tools/IUPAC.pm \
	Bio/Tools/Lucy.pm \
	Bio/Tools/MZEF.pm \
	Bio/Tools/OddCodes.pm \
	Bio/Tools/Phylo/Molphy.pm \
	Bio/Tools/Phylo/Molphy/Result.pm \
	Bio/Tools/Phylo/PAML.pm \
	Bio/Tools/Phylo/PAML/Result.pm \
	Bio/Tools/Phylo/Phylip/ProtDist.pm \
	Bio/Tools/Prediction/Exon.pm \
	Bio/Tools/Prediction/Gene.pm \
	Bio/Tools/Primer3.pm \
	Bio/Tools/Prints.pm \
	Bio/Tools/Profile.pm \
	Bio/Tools/Promoterwise.pm \
	Bio/Tools/Pseudowise.pm \
	Bio/Tools/RepeatMasker.pm \
	Bio/Tools/RestrictionEnzyme.pm \
	Bio/Tools/Run/RemoteBlast.pm \
	Bio/Tools/Run/StandAloneBlast.pm \
	Bio/Tools/Run/WrapperBase.pm \
	Bio/Tools/Seg.pm \
	Bio/Tools/SeqAnal.pm \
	Bio/Tools/SeqPattern.pm \
	Bio/Tools/SeqStats.pm \
	Bio/Tools/SeqWords.pm \
	Bio/Tools/Sigcleave.pm \
	Bio/Tools/Signalp.pm \
	Bio/Tools/Sim4/Exon.pm \
	Bio/Tools/Sim4/Results.pm \
	Bio/Tools/StateMachine/AbstractStateMachine.pm \
	Bio/Tools/StateMachine/IOStateMachine.pm \
	Bio/Tools/Tmhmm.pm \
	Bio/Tools/WWW.pm \
	Bio/Tools/pSW.pm \
	Bio/Tree/AlleleNode.pm \
	Bio/Tree/Node.pm \
	Bio/Tree/NodeI.pm \
	Bio/Tree/NodeNHX.pm \
	Bio/Tree/RandomFactory.pm \
	Bio/Tree/Statistics.pm \
	Bio/Tree/Tree.pm \
	Bio/Tree/TreeFunctionsI.pm \
	Bio/Tree/TreeI.pm \
	Bio/TreeIO.pm \
	Bio/TreeIO/TreeEventBuilder.pm \
	Bio/TreeIO/newick.pm \
	Bio/TreeIO/nhx.pm \
	Bio/TreeIO/tabtree.pm \
	Bio/UpdateableSeqI.pm \
	Bio/Variation/AAChange.pm \
	Bio/Variation/AAReverseMutate.pm \
	Bio/Variation/Allele.pm \
	Bio/Variation/DNAMutation.pm \
	Bio/Variation/IO.pm \
	Bio/Variation/IO/flat.pm \
	Bio/Variation/IO/xml.pm \
	Bio/Variation/RNAChange.pm \
	Bio/Variation/SNP.pm \
	Bio/Variation/SeqDiff.pm \
	Bio/Variation/VariantI.pm \
	bptutorial.pl

# Where is the Config information that we are using/depend on
CONFIGDEP = $(PERL_ARCHLIB)$(DFSEP)Config.pm $(PERL_INC)$(DFSEP)config.h

# Where to build things
INST_LIBDIR      = $(INST_LIB)
INST_ARCHLIBDIR  = $(INST_ARCHLIB)

INST_AUTODIR     = $(INST_LIB)/auto/$(FULLEXT)
INST_ARCHAUTODIR = $(INST_ARCHLIB)/auto/$(FULLEXT)

INST_STATIC      = 
INST_DYNAMIC     = 
INST_BOOT        = 

# Extra linker info
EXPORT_LIST        = 
PERL_ARCHIVE       = 
PERL_ARCHIVE_AFTER = 


TO_INST_PM = Bio/Align/AlignI.pm \
	Bio/Align/DNAStatistics.pm \
	Bio/Align/PairwiseStatistics.pm \
	Bio/Align/StatisticsI.pm \
	Bio/Align/Utilities.pm \
	Bio/AlignIO.pm \
	Bio/AlignIO/bl2seq.pm \
	Bio/AlignIO/clustalw.pm \
	Bio/AlignIO/emboss.pm \
	Bio/AlignIO/fasta.pm \
	Bio/AlignIO/mase.pm \
	Bio/AlignIO/mega.pm \
	Bio/AlignIO/meme.pm \
	Bio/AlignIO/msf.pm \
	Bio/AlignIO/nexus.pm \
	Bio/AlignIO/pfam.pm \
	Bio/AlignIO/phylip.pm \
	Bio/AlignIO/prodom.pm \
	Bio/AlignIO/psi.pm \
	Bio/AlignIO/selex.pm \
	Bio/AlignIO/stockholm.pm \
	Bio/AnalysisI.pm \
	Bio/AnalysisParserI.pm \
	Bio/AnalysisResultI.pm \
	Bio/AnnotatableI.pm \
	Bio/Annotation/AnnotationFactory.pm \
	Bio/Annotation/Collection.pm \
	Bio/Annotation/Comment.pm \
	Bio/Annotation/DBLink.pm \
	Bio/Annotation/OntologyTerm.pm \
	Bio/Annotation/Reference.pm \
	Bio/Annotation/SimpleValue.pm \
	Bio/Annotation/StructuredValue.pm \
	Bio/Annotation/TypeManager.pm \
	Bio/AnnotationCollectionI.pm \
	Bio/AnnotationI.pm \
	Bio/Assembly/Contig.pm \
	Bio/Assembly/ContigAnalysis.pm \
	Bio/Assembly/IO.pm \
	Bio/Assembly/IO/ace.pm \
	Bio/Assembly/IO/phrap.pm \
	Bio/Assembly/Scaffold.pm \
	Bio/Assembly/ScaffoldI.pm \
	Bio/Biblio.pm \
	Bio/Biblio/Article.pm \
	Bio/Biblio/BiblioBase.pm \
	Bio/Biblio/Book.pm \
	Bio/Biblio/BookArticle.pm \
	Bio/Biblio/IO.pm \
	Bio/Biblio/IO/medline2ref.pm \
	Bio/Biblio/IO/medlinexml.pm \
	Bio/Biblio/IO/pubmed2ref.pm \
	Bio/Biblio/IO/pubmedxml.pm \
	Bio/Biblio/Journal.pm \
	Bio/Biblio/JournalArticle.pm \
	Bio/Biblio/MedlineArticle.pm \
	Bio/Biblio/MedlineBook.pm \
	Bio/Biblio/MedlineBookArticle.pm \
	Bio/Biblio/MedlineJournal.pm \
	Bio/Biblio/MedlineJournalArticle.pm \
	Bio/Biblio/Organisation.pm \
	Bio/Biblio/Patent.pm \
	Bio/Biblio/Person.pm \
	Bio/Biblio/Proceeding.pm \
	Bio/Biblio/Provider.pm \
	Bio/Biblio/PubmedArticle.pm \
	Bio/Biblio/PubmedBookArticle.pm \
	Bio/Biblio/PubmedJournalArticle.pm \
	Bio/Biblio/Ref.pm \
	Bio/Biblio/Service.pm \
	Bio/Biblio/TechReport.pm \
	Bio/Biblio/Thesis.pm \
	Bio/Biblio/WebResource.pm \
	Bio/Cluster/ClusterFactory.pm \
	Bio/Cluster/FamilyI.pm \
	Bio/Cluster/SequenceFamily.pm \
	Bio/Cluster/UniGene.pm \
	Bio/Cluster/UniGeneI.pm \
	Bio/ClusterI.pm \
	Bio/ClusterIO.pm \
	Bio/ClusterIO/dbsnp.pm \
	Bio/ClusterIO/unigene.pm \
	Bio/Coordinate/Chain.pm \
	Bio/Coordinate/Collection.pm \
	Bio/Coordinate/ExtrapolatingPair.pm \
	Bio/Coordinate/GeneMapper.pm \
	Bio/Coordinate/Graph.pm \
	Bio/Coordinate/MapperI.pm \
	Bio/Coordinate/Pair.pm \
	Bio/Coordinate/Result.pm \
	Bio/Coordinate/Result/Gap.pm \
	Bio/Coordinate/Result/Match.pm \
	Bio/Coordinate/ResultI.pm \
	Bio/Coordinate/Utils.pm \
	Bio/DB/Ace.pm \
	Bio/DB/Biblio/biofetch.pm \
	Bio/DB/Biblio/soap.pm \
	Bio/DB/BiblioI.pm \
	Bio/DB/BioFetch.pm \
	Bio/DB/DBFetch.pm \
	Bio/DB/EMBL.pm \
	Bio/DB/Failover.pm \
	Bio/DB/Fasta.pm \
	Bio/DB/FileCache.pm \
	Bio/DB/Flat.pm \
	Bio/DB/Flat/BDB.pm \
	Bio/DB/Flat/BDB/embl.pm \
	Bio/DB/Flat/BDB/fasta.pm \
	Bio/DB/Flat/OBDAIndex.pm \
	Bio/DB/GDB.pm \
	Bio/DB/GFF.pm \
	Bio/DB/GFF/Adaptor/ace.pm \
	Bio/DB/GFF/Adaptor/biofetch.pm \
	Bio/DB/GFF/Adaptor/dbi.pm \
	Bio/DB/GFF/Adaptor/dbi/caching_handle.pm \
	Bio/DB/GFF/Adaptor/dbi/iterator.pm \
	Bio/DB/GFF/Adaptor/dbi/mysql.pm \
	Bio/DB/GFF/Adaptor/dbi/mysqlace.pm \
	Bio/DB/GFF/Adaptor/dbi/mysqlopt.pm \
	Bio/DB/GFF/Adaptor/dbi/oracle.pm \
	Bio/DB/GFF/Adaptor/dbi/oracleace.pm \
	Bio/DB/GFF/Adaptor/memory.pm \
	Bio/DB/GFF/Adaptor/memory_iterator.pm \
	Bio/DB/GFF/Aggregator.pm \
	Bio/DB/GFF/Aggregator/alignment.pm \
	Bio/DB/GFF/Aggregator/clone.pm \
	Bio/DB/GFF/Aggregator/coding.pm \
	Bio/DB/GFF/Aggregator/match.pm \
	Bio/DB/GFF/Aggregator/none.pm \
	Bio/DB/GFF/Aggregator/processed_transcript.pm \
	Bio/DB/GFF/Aggregator/transcript.pm \
	Bio/DB/GFF/Aggregator/ucsc_acembly.pm \
	Bio/DB/GFF/Aggregator/ucsc_ensgene.pm \
	Bio/DB/GFF/Aggregator/ucsc_genscan.pm \
	Bio/DB/GFF/Aggregator/ucsc_refgene.pm \
	Bio/DB/GFF/Aggregator/ucsc_sanger22.pm \
	Bio/DB/GFF/Aggregator/ucsc_sanger22pseudo.pm \
	Bio/DB/GFF/Aggregator/ucsc_softberry.pm \
	Bio/DB/GFF/Aggregator/ucsc_twinscan.pm \
	Bio/DB/GFF/Aggregator/ucsc_unigene.pm \
	Bio/DB/GFF/Featname.pm \
	Bio/DB/GFF/Feature.pm \
	Bio/DB/GFF/Homol.pm \
	Bio/DB/GFF/RelSegment.pm \
	Bio/DB/GFF/Segment.pm \
	Bio/DB/GFF/Typename.pm \
	Bio/DB/GFF/Util/Binning.pm \
	Bio/DB/GFF/Util/Rearrange.pm \
	Bio/DB/GenBank.pm \
	Bio/DB/GenPept.pm \
	Bio/DB/InMemoryCache.pm \
	Bio/DB/MANIFEST \
	Bio/DB/Makefile.PL \
	Bio/DB/NCBIHelper.pm \
	Bio/DB/Query/GenBank.pm \
	Bio/DB/Query/WebQuery.pm \
	Bio/DB/QueryI.pm \
	Bio/DB/RandomAccessI.pm \
	Bio/DB/RefSeq.pm \
	Bio/DB/Registry.pm \
	Bio/DB/SeqI.pm \
	Bio/DB/SwissProt.pm \
	Bio/DB/Universal.pm \
	Bio/DB/UpdateableSeqI.pm \
	Bio/DB/WebDBSeqI.pm \
	Bio/DB/XEMBL.pm \
	Bio/DB/XEMBLService.pm \
	Bio/DBLinkContainerI.pm \
	Bio/Das/FeatureTypeI.pm \
	Bio/Das/SegmentI.pm \
	Bio/DasI.pm \
	Bio/DescribableI.pm \
	Bio/Event/EventGeneratorI.pm \
	Bio/Event/EventHandlerI.pm \
	Bio/Expression/FeatureGroup.pm \
	Bio/Expression/FeatureGroup/FeatureGroupMas50.pm \
	Bio/Expression/FeatureI.pm \
	Bio/Expression/ProbeI.pm \
	Bio/Factory/AnalysisI.pm \
	Bio/Factory/ApplicationFactoryI.pm \
	Bio/Factory/BlastHitFactory.pm \
	Bio/Factory/BlastResultFactory.pm \
	Bio/Factory/DriverFactory.pm \
	Bio/Factory/EMBOSS.pm \
	Bio/Factory/FTLocationFactory.pm \
	Bio/Factory/HitFactoryI.pm \
	Bio/Factory/LocationFactoryI.pm \
	Bio/Factory/MapFactoryI.pm \
	Bio/Factory/ObjectBuilderI.pm \
	Bio/Factory/ObjectFactory.pm \
	Bio/Factory/ObjectFactoryI.pm \
	Bio/Factory/ResultFactoryI.pm \
	Bio/Factory/SeqAnalysisParserFactory.pm \
	Bio/Factory/SeqAnalysisParserFactoryI.pm \
	Bio/Factory/SequenceFactoryI.pm \
	Bio/Factory/SequenceProcessorI.pm \
	Bio/Factory/SequenceStreamI.pm \
	Bio/Factory/TreeFactoryI.pm \
	Bio/FeatureHolderI.pm \
	Bio/Graphics.pm \
	Bio/Graphics/Feature.pm \
	Bio/Graphics/FeatureFile.pm \
	Bio/Graphics/Glyph.pm \
	Bio/Graphics/Glyph/Factory.pm \
	Bio/Graphics/Glyph/alignment.pm \
	Bio/Graphics/Glyph/anchored_arrow.pm \
	Bio/Graphics/Glyph/arrow.pm \
	Bio/Graphics/Glyph/box.pm \
	Bio/Graphics/Glyph/cds.pm \
	Bio/Graphics/Glyph/crossbox.pm \
	Bio/Graphics/Glyph/diamond.pm \
	Bio/Graphics/Glyph/dna.pm \
	Bio/Graphics/Glyph/dot.pm \
	Bio/Graphics/Glyph/ellipse.pm \
	Bio/Graphics/Glyph/ex.pm \
	Bio/Graphics/Glyph/extending_arrow.pm \
	Bio/Graphics/Glyph/generic.pm \
	Bio/Graphics/Glyph/graded_segments.pm \
	Bio/Graphics/Glyph/group.pm \
	Bio/Graphics/Glyph/heterogeneous_segments.pm \
	Bio/Graphics/Glyph/line.pm \
	Bio/Graphics/Glyph/oval.pm \
	Bio/Graphics/Glyph/pinsertion.pm \
	Bio/Graphics/Glyph/primers.pm \
	Bio/Graphics/Glyph/processed_transcript.pm \
	Bio/Graphics/Glyph/redgreen_box.pm \
	Bio/Graphics/Glyph/redgreen_segment.pm \
	Bio/Graphics/Glyph/rndrect.pm \
	Bio/Graphics/Glyph/ruler_arrow.pm \
	Bio/Graphics/Glyph/segmented_keyglyph.pm \
	Bio/Graphics/Glyph/segments.pm \
	Bio/Graphics/Glyph/span.pm \
	Bio/Graphics/Glyph/splice_site.pm \
	Bio/Graphics/Glyph/toomany.pm \
	Bio/Graphics/Glyph/track.pm \
	Bio/Graphics/Glyph/transcript.pm \
	Bio/Graphics/Glyph/transcript2.pm \
	Bio/Graphics/Glyph/translation.pm \
	Bio/Graphics/Glyph/triangle.pm \
	Bio/Graphics/Glyph/xyplot.pm \
	Bio/Graphics/Panel.pm \
	Bio/Graphics/Util.pm \
	Bio/IdCollectionI.pm \
	Bio/IdentifiableI.pm \
	Bio/Index/Abstract.pm \
	Bio/Index/AbstractSeq.pm \
	Bio/Index/Blast.pm \
	Bio/Index/EMBL.pm \
	Bio/Index/Fasta.pm \
	Bio/Index/Fastq.pm \
	Bio/Index/GenBank.pm \
	Bio/Index/SwissPfam.pm \
	Bio/Index/Swissprot.pm \
	Bio/LiveSeq/AARange.pm \
	Bio/LiveSeq/Chain.pm \
	Bio/LiveSeq/ChainI.pm \
	Bio/LiveSeq/DNA.pm \
	Bio/LiveSeq/Exon.pm \
	Bio/LiveSeq/Gene.pm \
	Bio/LiveSeq/IO/BioPerl.pm \
	Bio/LiveSeq/IO/Loader.pm \
	Bio/LiveSeq/IO/README \
	Bio/LiveSeq/IO/SRS.pm \
	Bio/LiveSeq/Intron.pm \
	Bio/LiveSeq/Mutation.pm \
	Bio/LiveSeq/Mutator.pm \
	Bio/LiveSeq/Prim_Transcript.pm \
	Bio/LiveSeq/Range.pm \
	Bio/LiveSeq/Repeat_Region.pm \
	Bio/LiveSeq/Repeat_Unit.pm \
	Bio/LiveSeq/SeqI.pm \
	Bio/LiveSeq/Transcript.pm \
	Bio/LiveSeq/Translation.pm \
	Bio/LocatableSeq.pm \
	Bio/Location/Atomic.pm \
	Bio/Location/AvWithinCoordPolicy.pm \
	Bio/Location/CoordinatePolicyI.pm \
	Bio/Location/Fuzzy.pm \
	Bio/Location/FuzzyLocationI.pm \
	Bio/Location/NarrowestCoordPolicy.pm \
	Bio/Location/Simple.pm \
	Bio/Location/Split.pm \
	Bio/Location/SplitLocationI.pm \
	Bio/Location/WidestCoordPolicy.pm \
	Bio/LocationI.pm \
	Bio/Map/CytoMap.pm \
	Bio/Map/CytoMarker.pm \
	Bio/Map/CytoPosition.pm \
	Bio/Map/LinkageMap.pm \
	Bio/Map/LinkagePosition.pm \
	Bio/Map/MapI.pm \
	Bio/Map/MappableI.pm \
	Bio/Map/Marker.pm \
	Bio/Map/MarkerI.pm \
	Bio/Map/Microsatellite.pm \
	Bio/Map/OrderedPosition.pm \
	Bio/Map/OrderedPositionWithDistance.pm \
	Bio/Map/Position.pm \
	Bio/Map/PositionI.pm \
	Bio/Map/SimpleMap.pm \
	Bio/MapIO.pm \
	Bio/MapIO/mapmaker.pm \
	Bio/Matrix/PhylipDist.pm \
	Bio/Ontology/GOterm.pm \
	Bio/Ontology/InterProTerm.pm \
	Bio/Ontology/Ontology.pm \
	Bio/Ontology/OntologyEngineI.pm \
	Bio/Ontology/OntologyI.pm \
	Bio/Ontology/OntologyStore.pm \
	Bio/Ontology/Path.pm \
	Bio/Ontology/PathI.pm \
	Bio/Ontology/Relationship.pm \
	Bio/Ontology/RelationshipFactory.pm \
	Bio/Ontology/RelationshipI.pm \
	Bio/Ontology/RelationshipType.pm \
	Bio/Ontology/SimpleGOEngine.pm \
	Bio/Ontology/SimpleOntologyEngine.pm \
	Bio/Ontology/Term.pm \
	Bio/Ontology/TermFactory.pm \
	Bio/Ontology/TermI.pm \
	Bio/OntologyIO.pm \
	Bio/OntologyIO/Handlers/InterProHandler.pm \
	Bio/OntologyIO/InterProParser.pm \
	Bio/OntologyIO/dagflat.pm \
	Bio/OntologyIO/goflat.pm \
	Bio/OntologyIO/soflat.pm \
	Bio/Perl.pm \
	Bio/Phenotype/Correlate.pm \
	Bio/Phenotype/Measure.pm \
	Bio/Phenotype/OMIM/MiniMIMentry.pm \
	Bio/Phenotype/OMIM/OMIMentry.pm \
	Bio/Phenotype/OMIM/OMIMentryAllelicVariant.pm \
	Bio/Phenotype/OMIM/OMIMparser.pm \
	Bio/Phenotype/Phenotype.pm \
	Bio/Phenotype/PhenotypeI.pm \
	Bio/PrimarySeq.pm \
	Bio/PrimarySeqI.pm \
	Bio/Range.pm \
	Bio/RangeI.pm \
	Bio/Root/Err.pm \
	Bio/Root/Exception.pm \
	Bio/Root/Global.pm \
	Bio/Root/HTTPget.pm \
	Bio/Root/IO.pm \
	Bio/Root/IOManager.pm \
	Bio/Root/Object.pm \
	Bio/Root/Root.pm \
	Bio/Root/RootI.pm \
	Bio/Root/Utilities.pm \
	Bio/Root/Vector.pm \
	Bio/Root/Xref.pm \
	Bio/Search/BlastUtils.pm \
	Bio/Search/DatabaseI.pm \
	Bio/Search/GenericDatabase.pm \
	Bio/Search/HSP/BlastHSP.pm \
	Bio/Search/HSP/FastaHSP.pm \
	Bio/Search/HSP/GenericHSP.pm \
	Bio/Search/HSP/HMMERHSP.pm \
	Bio/Search/HSP/HSPFactory.pm \
	Bio/Search/HSP/HSPI.pm \
	Bio/Search/HSP/WABAHSP.pm \
	Bio/Search/Hit/BlastHit.pm \
	Bio/Search/Hit/Fasta.pm \
	Bio/Search/Hit/GenericHit.pm \
	Bio/Search/Hit/HMMERHit.pm \
	Bio/Search/Hit/HitFactory.pm \
	Bio/Search/Hit/HitI.pm \
	Bio/Search/Processor.pm \
	Bio/Search/Result/BlastResult.pm \
	Bio/Search/Result/GenericResult.pm \
	Bio/Search/Result/HMMERResult.pm \
	Bio/Search/Result/ResultFactory.pm \
	Bio/Search/Result/ResultI.pm \
	Bio/Search/Result/WABAResult.pm \
	Bio/Search/SearchUtils.pm \
	Bio/SearchDist.pm \
	Bio/SearchIO.pm \
	Bio/SearchIO/EventHandlerI.pm \
	Bio/SearchIO/FastHitEventBuilder.pm \
	Bio/SearchIO/SearchResultEventBuilder.pm \
	Bio/SearchIO/SearchWriterI.pm \
	Bio/SearchIO/Writer/HSPTableWriter.pm \
	Bio/SearchIO/Writer/HTMLResultWriter.pm \
	Bio/SearchIO/Writer/HitTableWriter.pm \
	Bio/SearchIO/Writer/ResultTableWriter.pm \
	Bio/SearchIO/Writer/TextResultWriter.pm \
	Bio/SearchIO/blast.pm \
	Bio/SearchIO/blastxml.pm \
	Bio/SearchIO/chado.pm \
	Bio/SearchIO/chadosxpr.pm \
	Bio/SearchIO/exonerate.pm \
	Bio/SearchIO/fasta.pm \
	Bio/SearchIO/hmmer.pm \
	Bio/SearchIO/psiblast.pm \
	Bio/SearchIO/waba.pm \
	Bio/Seq.pm \
	Bio/Seq/BaseSeqProcessor.pm \
	Bio/Seq/EncodedSeq.pm \
	Bio/Seq/LargePrimarySeq.pm \
	Bio/Seq/LargeSeq.pm \
	Bio/Seq/PrimaryQual.pm \
	Bio/Seq/PrimedSeq.pm \
	Bio/Seq/QualI.pm \
	Bio/Seq/RichSeq.pm \
	Bio/Seq/RichSeqI.pm \
	Bio/Seq/SeqBuilder.pm \
	Bio/Seq/SeqFactory.pm \
	Bio/Seq/SeqFastaSpeedFactory.pm \
	Bio/Seq/SeqWithQuality.pm \
	Bio/Seq/SequenceTrace.pm \
	Bio/Seq/TraceI.pm \
	Bio/SeqAnalysisParserI.pm \
	Bio/SeqFeature/AnnotationAdaptor.pm \
	Bio/SeqFeature/Collection.pm \
	Bio/SeqFeature/CollectionI.pm \
	Bio/SeqFeature/Computation.pm \
	Bio/SeqFeature/FeaturePair.pm \
	Bio/SeqFeature/Gene/Exon.pm \
	Bio/SeqFeature/Gene/ExonI.pm \
	Bio/SeqFeature/Gene/GeneStructure.pm \
	Bio/SeqFeature/Gene/GeneStructureI.pm \
	Bio/SeqFeature/Gene/Intron.pm \
	Bio/SeqFeature/Gene/NC_Feature.pm \
	Bio/SeqFeature/Gene/Poly_A_site.pm \
	Bio/SeqFeature/Gene/Promoter.pm \
	Bio/SeqFeature/Gene/Transcript.pm \
	Bio/SeqFeature/Gene/TranscriptI.pm \
	Bio/SeqFeature/Gene/UTR.pm \
	Bio/SeqFeature/Generic.pm \
	Bio/SeqFeature/PositionProxy.pm \
	Bio/SeqFeature/Primer.pm \
	Bio/SeqFeature/Similarity.pm \
	Bio/SeqFeature/SimilarityPair.pm \
	Bio/SeqFeatureI.pm \
	Bio/SeqI.pm \
	Bio/SeqIO.pm \
	Bio/SeqIO/FTHelper.pm \
	Bio/SeqIO/MultiFile.pm \
	Bio/SeqIO/abi.pm \
	Bio/SeqIO/ace.pm \
	Bio/SeqIO/alf.pm \
	Bio/SeqIO/bsml.pm \
	Bio/SeqIO/chado.pm \
	Bio/SeqIO/chadoitext.pm \
	Bio/SeqIO/chadosxpr.pm \
	Bio/SeqIO/chadoxml.pm \
	Bio/SeqIO/ctf.pm \
	Bio/SeqIO/embl.pm \
	Bio/SeqIO/exp.pm \
	Bio/SeqIO/fasta.pm \
	Bio/SeqIO/fastq.pm \
	Bio/SeqIO/game.pm \
	Bio/SeqIO/game/featureHandler.pm \
	Bio/SeqIO/game/idHandler.pm \
	Bio/SeqIO/game/seqHandler.pm \
	Bio/SeqIO/gcg.pm \
	Bio/SeqIO/genbank.pm \
	Bio/SeqIO/largefasta.pm \
	Bio/SeqIO/locuslink.pm \
	Bio/SeqIO/phd.pm \
	Bio/SeqIO/pir.pm \
	Bio/SeqIO/pln.pm \
	Bio/SeqIO/qual.pm \
	Bio/SeqIO/raw.pm \
	Bio/SeqIO/scf.pm \
	Bio/SeqIO/swiss.pm \
	Bio/SeqIO/ztr.pm \
	Bio/SeqUtils.pm \
	Bio/SimpleAlign.pm \
	Bio/Species.pm \
	Bio/Structure/Atom.pm \
	Bio/Structure/Chain.pm \
	Bio/Structure/Entry.pm \
	Bio/Structure/IO.pm \
	Bio/Structure/IO/pdb.pm \
	Bio/Structure/Model.pm \
	Bio/Structure/Residue.pm \
	Bio/Structure/SecStr/DSSP/Res.pm \
	Bio/Structure/SecStr/STRIDE/Res.pm \
	Bio/Structure/StructureI.pm \
	Bio/Symbol/Alphabet.pm \
	Bio/Symbol/AlphabetI.pm \
	Bio/Symbol/DNAAlphabet.pm \
	Bio/Symbol/ProteinAlphabet.pm \
	Bio/Symbol/Symbol.pm \
	Bio/Symbol/SymbolI.pm \
	Bio/Taxonomy.pm \
	Bio/Taxonomy/Taxon.pm \
	Bio/Taxonomy/Tree.pm \
	Bio/Tools/AlignFactory.pm \
	Bio/Tools/Alignment/Consed.pm \
	Bio/Tools/Alignment/Trim.pm \
	Bio/Tools/AnalysisResult.pm \
	Bio/Tools/BPbl2seq.pm \
	Bio/Tools/BPlite.pm \
	Bio/Tools/BPlite/HSP.pm \
	Bio/Tools/BPlite/Iteration.pm \
	Bio/Tools/BPlite/Sbjct.pm \
	Bio/Tools/BPpsilite.pm \
	Bio/Tools/Blast.pm \
	Bio/Tools/Blast/CHANGES \
	Bio/Tools/Blast/HSP.pm \
	Bio/Tools/Blast/HTML.pm \
	Bio/Tools/Blast/README \
	Bio/Tools/Blast/Sbjct.pm \
	Bio/Tools/CodonTable.pm \
	Bio/Tools/Coil.pm \
	Bio/Tools/ECnumber.pm \
	Bio/Tools/EPCR.pm \
	Bio/Tools/ESTScan.pm \
	Bio/Tools/Eponine.pm \
	Bio/Tools/Est2Genome.pm \
	Bio/Tools/FootPrinter.pm \
	Bio/Tools/GFF.pm \
	Bio/Tools/Gel.pm \
	Bio/Tools/Genemark.pm \
	Bio/Tools/Genewise.pm \
	Bio/Tools/Genomewise.pm \
	Bio/Tools/Genscan.pm \
	Bio/Tools/Grail.pm \
	Bio/Tools/HMMER/Domain.pm \
	Bio/Tools/HMMER/Results.pm \
	Bio/Tools/HMMER/Set.pm \
	Bio/Tools/Hmmpfam.pm \
	Bio/Tools/IUPAC.pm \
	Bio/Tools/Lucy.pm \
	Bio/Tools/MZEF.pm \
	Bio/Tools/OddCodes.pm \
	Bio/Tools/Phylo/Molphy.pm \
	Bio/Tools/Phylo/Molphy/Result.pm \
	Bio/Tools/Phylo/PAML.pm \
	Bio/Tools/Phylo/PAML/Result.pm \
	Bio/Tools/Phylo/Phylip/ProtDist.pm \
	Bio/Tools/Prediction/Exon.pm \
	Bio/Tools/Prediction/Gene.pm \
	Bio/Tools/Primer3.pm \
	Bio/Tools/Prints.pm \
	Bio/Tools/Profile.pm \
	Bio/Tools/Promoterwise.pm \
	Bio/Tools/Pseudowise.pm \
	Bio/Tools/RepeatMasker.pm \
	Bio/Tools/RestrictionEnzyme.pm \
	Bio/Tools/Run/README \
	Bio/Tools/Run/RemoteBlast.pm \
	Bio/Tools/Run/StandAloneBlast.pm \
	Bio/Tools/Run/WrapperBase.pm \
	Bio/Tools/Seg.pm \
	Bio/Tools/SeqAnal.pm \
	Bio/Tools/SeqPattern.pm \
	Bio/Tools/SeqStats.pm \
	Bio/Tools/SeqWords.pm \
	Bio/Tools/Sigcleave.pm \
	Bio/Tools/Signalp.pm \
	Bio/Tools/Sim4/Exon.pm \
	Bio/Tools/Sim4/Results.pm \
	Bio/Tools/StateMachine/AbstractStateMachine.pm \
	Bio/Tools/StateMachine/IOStateMachine.pm \
	Bio/Tools/Tmhmm.pm \
	Bio/Tools/WWW.pm \
	Bio/Tools/pSW.pm \
	Bio/Tree/AlleleNode.pm \
	Bio/Tree/Node.pm \
	Bio/Tree/NodeI.pm \
	Bio/Tree/NodeNHX.pm \
	Bio/Tree/RandomFactory.pm \
	Bio/Tree/Statistics.pm \
	Bio/Tree/Tree.pm \
	Bio/Tree/TreeFunctionsI.pm \
	Bio/Tree/TreeI.pm \
	Bio/TreeIO.pm \
	Bio/TreeIO/TreeEventBuilder.pm \
	Bio/TreeIO/newick.pm \
	Bio/TreeIO/nhx.pm \
	Bio/TreeIO/tabtree.pm \
	Bio/UpdateableSeqI.pm \
	Bio/Variation/AAChange.pm \
	Bio/Variation/AAReverseMutate.pm \
	Bio/Variation/Allele.pm \
	Bio/Variation/DNAMutation.pm \
	Bio/Variation/IO.pm \
	Bio/Variation/IO/flat.pm \
	Bio/Variation/IO/xml.pm \
	Bio/Variation/README \
	Bio/Variation/RNAChange.pm \
	Bio/Variation/SNP.pm \
	Bio/Variation/SeqDiff.pm \
	Bio/Variation/VariantI.pm \
	LocalConfig.pm \
	bptutorial.pl

PM_TO_BLIB = Bio/Tools/Blast/CHANGES \
	$(INST_LIB)/Bio/Tools/Blast/CHANGES \
	Bio/LiveSeq/ChainI.pm \
	$(INST_LIB)/Bio/LiveSeq/ChainI.pm \
	Bio/AlignIO/mega.pm \
	$(INST_LIB)/Bio/AlignIO/mega.pm \
	Bio/LiveSeq/Mutation.pm \
	$(INST_LIB)/Bio/LiveSeq/Mutation.pm \
	Bio/Tools/HMMER/Set.pm \
	$(INST_LIB)/Bio/Tools/HMMER/Set.pm \
	Bio/Factory/ObjectFactory.pm \
	$(INST_LIB)/Bio/Factory/ObjectFactory.pm \
	Bio/Factory/BlastResultFactory.pm \
	$(INST_LIB)/Bio/Factory/BlastResultFactory.pm \
	Bio/Factory/SeqAnalysisParserFactoryI.pm \
	$(INST_LIB)/Bio/Factory/SeqAnalysisParserFactoryI.pm \
	Bio/DB/GFF/Aggregator/ucsc_unigene.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_unigene.pm \
	Bio/TreeIO.pm \
	$(INST_LIB)/Bio/TreeIO.pm \
	Bio/DB/GFF/Aggregator/ucsc_acembly.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_acembly.pm \
	Bio/Cluster/UniGeneI.pm \
	$(INST_LIB)/Bio/Cluster/UniGeneI.pm \
	Bio/Biblio/PubmedBookArticle.pm \
	$(INST_LIB)/Bio/Biblio/PubmedBookArticle.pm \
	Bio/Align/Utilities.pm \
	$(INST_LIB)/Bio/Align/Utilities.pm \
	Bio/Search/HSP/WABAHSP.pm \
	$(INST_LIB)/Bio/Search/HSP/WABAHSP.pm \
	Bio/Expression/ProbeI.pm \
	$(INST_LIB)/Bio/Expression/ProbeI.pm \
	Bio/DB/Flat.pm \
	$(INST_LIB)/Bio/DB/Flat.pm \
	Bio/Tools/Run/README \
	$(INST_LIB)/Bio/Tools/Run/README \
	Bio/DB/SwissProt.pm \
	$(INST_LIB)/Bio/DB/SwissProt.pm \
	Bio/SearchIO/fasta.pm \
	$(INST_LIB)/Bio/SearchIO/fasta.pm \
	Bio/Biblio/Provider.pm \
	$(INST_LIB)/Bio/Biblio/Provider.pm \
	Bio/Coordinate/Graph.pm \
	$(INST_LIB)/Bio/Coordinate/Graph.pm \
	Bio/DB/GFF/Aggregator/ucsc_sanger22.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_sanger22.pm \
	Bio/Root/RootI.pm \
	$(INST_LIB)/Bio/Root/RootI.pm \
	Bio/LocationI.pm \
	$(INST_LIB)/Bio/LocationI.pm \
	Bio/TreeIO/TreeEventBuilder.pm \
	$(INST_LIB)/Bio/TreeIO/TreeEventBuilder.pm \
	Bio/Taxonomy.pm \
	$(INST_LIB)/Bio/Taxonomy.pm \
	Bio/Location/Fuzzy.pm \
	$(INST_LIB)/Bio/Location/Fuzzy.pm \
	Bio/DBLinkContainerI.pm \
	$(INST_LIB)/Bio/DBLinkContainerI.pm \
	Bio/SeqIO.pm \
	$(INST_LIB)/Bio/SeqIO.pm \
	Bio/AnalysisI.pm \
	$(INST_LIB)/Bio/AnalysisI.pm \
	Bio/LiveSeq/AARange.pm \
	$(INST_LIB)/Bio/LiveSeq/AARange.pm \
	Bio/DB/Biblio/biofetch.pm \
	$(INST_LIB)/Bio/DB/Biblio/biofetch.pm \
	Bio/Index/Swissprot.pm \
	$(INST_LIB)/Bio/Index/Swissprot.pm \
	Bio/Annotation/OntologyTerm.pm \
	$(INST_LIB)/Bio/Annotation/OntologyTerm.pm \
	Bio/Map/SimpleMap.pm \
	$(INST_LIB)/Bio/Map/SimpleMap.pm \
	Bio/Coordinate/Result/Gap.pm \
	$(INST_LIB)/Bio/Coordinate/Result/Gap.pm \
	Bio/Tools/Run/RemoteBlast.pm \
	$(INST_LIB)/Bio/Tools/Run/RemoteBlast.pm \
	Bio/DB/BiblioI.pm \
	$(INST_LIB)/Bio/DB/BiblioI.pm \
	Bio/SearchIO/Writer/HSPTableWriter.pm \
	$(INST_LIB)/Bio/SearchIO/Writer/HSPTableWriter.pm \
	Bio/MapIO.pm \
	$(INST_LIB)/Bio/MapIO.pm \
	Bio/Graphics/Glyph/segmented_keyglyph.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/segmented_keyglyph.pm \
	Bio/SeqFeature/Collection.pm \
	$(INST_LIB)/Bio/SeqFeature/Collection.pm \
	Bio/Graphics/Glyph/box.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/box.pm \
	Bio/Graphics/Glyph/cds.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/cds.pm \
	Bio/Tree/TreeI.pm \
	$(INST_LIB)/Bio/Tree/TreeI.pm \
	Bio/Graphics/Glyph/extending_arrow.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/extending_arrow.pm \
	Bio/Map/MappableI.pm \
	$(INST_LIB)/Bio/Map/MappableI.pm \
	Bio/Seq/BaseSeqProcessor.pm \
	$(INST_LIB)/Bio/Seq/BaseSeqProcessor.pm \
	Bio/SeqIO/fastq.pm \
	$(INST_LIB)/Bio/SeqIO/fastq.pm \
	Bio/LiveSeq/SeqI.pm \
	$(INST_LIB)/Bio/LiveSeq/SeqI.pm \
	Bio/Seq/TraceI.pm \
	$(INST_LIB)/Bio/Seq/TraceI.pm \
	Bio/Phenotype/Phenotype.pm \
	$(INST_LIB)/Bio/Phenotype/Phenotype.pm \
	Bio/Seq/PrimaryQual.pm \
	$(INST_LIB)/Bio/Seq/PrimaryQual.pm \
	Bio/Variation/Allele.pm \
	$(INST_LIB)/Bio/Variation/Allele.pm \
	Bio/Map/CytoMap.pm \
	$(INST_LIB)/Bio/Map/CytoMap.pm \
	Bio/DB/Failover.pm \
	$(INST_LIB)/Bio/DB/Failover.pm \
	Bio/LiveSeq/IO/Loader.pm \
	$(INST_LIB)/Bio/LiveSeq/IO/Loader.pm \
	Bio/LiveSeq/Repeat_Unit.pm \
	$(INST_LIB)/Bio/LiveSeq/Repeat_Unit.pm \
	Bio/DB/GFF/Adaptor/memory.pm \
	$(INST_LIB)/Bio/DB/GFF/Adaptor/memory.pm \
	Bio/DB/UpdateableSeqI.pm \
	$(INST_LIB)/Bio/DB/UpdateableSeqI.pm \
	Bio/DB/RefSeq.pm \
	$(INST_LIB)/Bio/DB/RefSeq.pm \
	Bio/Tools/RepeatMasker.pm \
	$(INST_LIB)/Bio/Tools/RepeatMasker.pm \
	Bio/Coordinate/Result/Match.pm \
	$(INST_LIB)/Bio/Coordinate/Result/Match.pm \
	Bio/Search/Result/HMMERResult.pm \
	$(INST_LIB)/Bio/Search/Result/HMMERResult.pm \
	Bio/Seq/PrimedSeq.pm \
	$(INST_LIB)/Bio/Seq/PrimedSeq.pm \
	Bio/SearchIO/chado.pm \
	$(INST_LIB)/Bio/SearchIO/chado.pm \
	Bio/Structure/Chain.pm \
	$(INST_LIB)/Bio/Structure/Chain.pm \
	Bio/AlignIO/prodom.pm \
	$(INST_LIB)/Bio/AlignIO/prodom.pm \
	Bio/Assembly/ContigAnalysis.pm \
	$(INST_LIB)/Bio/Assembly/ContigAnalysis.pm \
	Bio/Coordinate/ExtrapolatingPair.pm \
	$(INST_LIB)/Bio/Coordinate/ExtrapolatingPair.pm \
	Bio/Graphics/Glyph/group.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/group.pm \
	Bio/IdCollectionI.pm \
	$(INST_LIB)/Bio/IdCollectionI.pm \
	Bio/Tree/AlleleNode.pm \
	$(INST_LIB)/Bio/Tree/AlleleNode.pm \
	Bio/Tools/Gel.pm \
	$(INST_LIB)/Bio/Tools/Gel.pm \
	Bio/DB/NCBIHelper.pm \
	$(INST_LIB)/Bio/DB/NCBIHelper.pm \
	Bio/Biblio/BiblioBase.pm \
	$(INST_LIB)/Bio/Biblio/BiblioBase.pm \
	Bio/Biblio/IO.pm \
	$(INST_LIB)/Bio/Biblio/IO.pm \
	Bio/DB/GFF/Feature.pm \
	$(INST_LIB)/Bio/DB/GFF/Feature.pm \
	Bio/Phenotype/OMIM/MiniMIMentry.pm \
	$(INST_LIB)/Bio/Phenotype/OMIM/MiniMIMentry.pm \
	Bio/UpdateableSeqI.pm \
	$(INST_LIB)/Bio/UpdateableSeqI.pm \
	Bio/SeqIO/chadosxpr.pm \
	$(INST_LIB)/Bio/SeqIO/chadosxpr.pm \
	Bio/Event/EventHandlerI.pm \
	$(INST_LIB)/Bio/Event/EventHandlerI.pm \
	Bio/Biblio/Thesis.pm \
	$(INST_LIB)/Bio/Biblio/Thesis.pm \
	Bio/Graphics/Glyph/processed_transcript.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/processed_transcript.pm \
	Bio/DB/Fasta.pm \
	$(INST_LIB)/Bio/DB/Fasta.pm \
	Bio/Biblio.pm \
	$(INST_LIB)/Bio/Biblio.pm \
	Bio/Phenotype/OMIM/OMIMparser.pm \
	$(INST_LIB)/Bio/Phenotype/OMIM/OMIMparser.pm \
	Bio/DB/GFF/Adaptor/memory_iterator.pm \
	$(INST_LIB)/Bio/DB/GFF/Adaptor/memory_iterator.pm \
	Bio/SeqIO/game/seqHandler.pm \
	$(INST_LIB)/Bio/SeqIO/game/seqHandler.pm \
	Bio/Tree/TreeFunctionsI.pm \
	$(INST_LIB)/Bio/Tree/TreeFunctionsI.pm \
	Bio/Tools/StateMachine/IOStateMachine.pm \
	$(INST_LIB)/Bio/Tools/StateMachine/IOStateMachine.pm \
	Bio/SeqIO/largefasta.pm \
	$(INST_LIB)/Bio/SeqIO/largefasta.pm \
	Bio/Annotation/SimpleValue.pm \
	$(INST_LIB)/Bio/Annotation/SimpleValue.pm \
	Bio/AlignIO/emboss.pm \
	$(INST_LIB)/Bio/AlignIO/emboss.pm \
	Bio/Align/AlignI.pm \
	$(INST_LIB)/Bio/Align/AlignI.pm \
	Bio/DB/Query/WebQuery.pm \
	$(INST_LIB)/Bio/DB/Query/WebQuery.pm \
	Bio/Graphics/Glyph/generic.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/generic.pm \
	Bio/Ontology/OntologyI.pm \
	$(INST_LIB)/Bio/Ontology/OntologyI.pm \
	Bio/SearchIO/hmmer.pm \
	$(INST_LIB)/Bio/SearchIO/hmmer.pm \
	Bio/SeqIO/ctf.pm \
	$(INST_LIB)/Bio/SeqIO/ctf.pm \
	Bio/LiveSeq/IO/SRS.pm \
	$(INST_LIB)/Bio/LiveSeq/IO/SRS.pm \
	Bio/Tools/SeqPattern.pm \
	$(INST_LIB)/Bio/Tools/SeqPattern.pm \
	Bio/LiveSeq/DNA.pm \
	$(INST_LIB)/Bio/LiveSeq/DNA.pm \
	Bio/Index/AbstractSeq.pm \
	$(INST_LIB)/Bio/Index/AbstractSeq.pm \
	Bio/AlignIO.pm \
	$(INST_LIB)/Bio/AlignIO.pm \
	Bio/Tree/NodeI.pm \
	$(INST_LIB)/Bio/Tree/NodeI.pm \
	Bio/SeqFeature/Gene/Exon.pm \
	$(INST_LIB)/Bio/SeqFeature/Gene/Exon.pm \
	Bio/Tools/Prints.pm \
	$(INST_LIB)/Bio/Tools/Prints.pm \
	Bio/Taxonomy/Tree.pm \
	$(INST_LIB)/Bio/Taxonomy/Tree.pm \
	Bio/Ontology/RelationshipType.pm \
	$(INST_LIB)/Bio/Ontology/RelationshipType.pm \
	Bio/Cluster/UniGene.pm \
	$(INST_LIB)/Bio/Cluster/UniGene.pm \
	Bio/SeqFeature/Generic.pm \
	$(INST_LIB)/Bio/SeqFeature/Generic.pm \
	Bio/Root/Global.pm \
	$(INST_LIB)/Bio/Root/Global.pm \
	Bio/SeqFeature/SimilarityPair.pm \
	$(INST_LIB)/Bio/SeqFeature/SimilarityPair.pm \
	Bio/SeqIO/raw.pm \
	$(INST_LIB)/Bio/SeqIO/raw.pm \
	Bio/Search/Processor.pm \
	$(INST_LIB)/Bio/Search/Processor.pm \
	Bio/Seq.pm \
	$(INST_LIB)/Bio/Seq.pm \
	Bio/Biblio/Article.pm \
	$(INST_LIB)/Bio/Biblio/Article.pm \
	Bio/Tools/HMMER/Results.pm \
	$(INST_LIB)/Bio/Tools/HMMER/Results.pm \
	Bio/Search/BlastUtils.pm \
	$(INST_LIB)/Bio/Search/BlastUtils.pm \
	Bio/Biblio/Organisation.pm \
	$(INST_LIB)/Bio/Biblio/Organisation.pm \
	Bio/Tree/RandomFactory.pm \
	$(INST_LIB)/Bio/Tree/RandomFactory.pm \
	Bio/Structure/IO/pdb.pm \
	$(INST_LIB)/Bio/Structure/IO/pdb.pm \
	Bio/Tools/Phylo/PAML/Result.pm \
	$(INST_LIB)/Bio/Tools/Phylo/PAML/Result.pm \
	Bio/Structure/StructureI.pm \
	$(INST_LIB)/Bio/Structure/StructureI.pm \
	Bio/DescribableI.pm \
	$(INST_LIB)/Bio/DescribableI.pm \
	Bio/Symbol/AlphabetI.pm \
	$(INST_LIB)/Bio/Symbol/AlphabetI.pm \
	Bio/Search/Result/ResultI.pm \
	$(INST_LIB)/Bio/Search/Result/ResultI.pm \
	Bio/Graphics/Glyph/rndrect.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/rndrect.pm \
	Bio/DB/Flat/OBDAIndex.pm \
	$(INST_LIB)/Bio/DB/Flat/OBDAIndex.pm \
	Bio/RangeI.pm \
	$(INST_LIB)/Bio/RangeI.pm \
	Bio/Location/SplitLocationI.pm \
	$(INST_LIB)/Bio/Location/SplitLocationI.pm \
	Bio/SeqFeature/Gene/NC_Feature.pm \
	$(INST_LIB)/Bio/SeqFeature/Gene/NC_Feature.pm \
	Bio/Ontology/RelationshipI.pm \
	$(INST_LIB)/Bio/Ontology/RelationshipI.pm \
	Bio/OntologyIO.pm \
	$(INST_LIB)/Bio/OntologyIO.pm \
	Bio/Graphics/Feature.pm \
	$(INST_LIB)/Bio/Graphics/Feature.pm \
	Bio/Index/GenBank.pm \
	$(INST_LIB)/Bio/Index/GenBank.pm \
	Bio/Tools/StateMachine/AbstractStateMachine.pm \
	$(INST_LIB)/Bio/Tools/StateMachine/AbstractStateMachine.pm \
	Bio/Biblio/MedlineJournalArticle.pm \
	$(INST_LIB)/Bio/Biblio/MedlineJournalArticle.pm \
	Bio/DB/GFF/Aggregator/ucsc_sanger22pseudo.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_sanger22pseudo.pm \
	Bio/Variation/IO/xml.pm \
	$(INST_LIB)/Bio/Variation/IO/xml.pm \
	Bio/DB/GFF/Aggregator/alignment.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/alignment.pm \
	Bio/FeatureHolderI.pm \
	$(INST_LIB)/Bio/FeatureHolderI.pm \
	Bio/SeqFeature/Gene/TranscriptI.pm \
	$(INST_LIB)/Bio/SeqFeature/Gene/TranscriptI.pm \
	Bio/DB/RandomAccessI.pm \
	$(INST_LIB)/Bio/DB/RandomAccessI.pm \
	Bio/Cluster/FamilyI.pm \
	$(INST_LIB)/Bio/Cluster/FamilyI.pm \
	Bio/SeqIO/qual.pm \
	$(INST_LIB)/Bio/SeqIO/qual.pm \
	Bio/Ontology/TermI.pm \
	$(INST_LIB)/Bio/Ontology/TermI.pm \
	Bio/SeqIO/pln.pm \
	$(INST_LIB)/Bio/SeqIO/pln.pm \
	Bio/AnalysisResultI.pm \
	$(INST_LIB)/Bio/AnalysisResultI.pm \
	Bio/Search/Result/BlastResult.pm \
	$(INST_LIB)/Bio/Search/Result/BlastResult.pm \
	Bio/Assembly/ScaffoldI.pm \
	$(INST_LIB)/Bio/Assembly/ScaffoldI.pm \
	Bio/Annotation/AnnotationFactory.pm \
	$(INST_LIB)/Bio/Annotation/AnnotationFactory.pm \
	Bio/SeqIO/locuslink.pm \
	$(INST_LIB)/Bio/SeqIO/locuslink.pm \
	Bio/Structure/Atom.pm \
	$(INST_LIB)/Bio/Structure/Atom.pm \
	Bio/Tools/Sim4/Exon.pm \
	$(INST_LIB)/Bio/Tools/Sim4/Exon.pm \
	Bio/Factory/BlastHitFactory.pm \
	$(INST_LIB)/Bio/Factory/BlastHitFactory.pm \
	Bio/Graphics/Glyph/ex.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/ex.pm \
	Bio/Tools/Phylo/Molphy/Result.pm \
	$(INST_LIB)/Bio/Tools/Phylo/Molphy/Result.pm \
	Bio/DB/GFF/Aggregator/match.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/match.pm \
	Bio/DB/GFF/Aggregator/ucsc_softberry.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_softberry.pm \
	Bio/Map/PositionI.pm \
	$(INST_LIB)/Bio/Map/PositionI.pm \
	Bio/Biblio/BookArticle.pm \
	$(INST_LIB)/Bio/Biblio/BookArticle.pm \
	Bio/Tools/SeqStats.pm \
	$(INST_LIB)/Bio/Tools/SeqStats.pm \
	Bio/LiveSeq/Transcript.pm \
	$(INST_LIB)/Bio/LiveSeq/Transcript.pm \
	Bio/Ontology/OntologyEngineI.pm \
	$(INST_LIB)/Bio/Ontology/OntologyEngineI.pm \
	Bio/Ontology/Term.pm \
	$(INST_LIB)/Bio/Ontology/Term.pm \
	Bio/Search/HSP/FastaHSP.pm \
	$(INST_LIB)/Bio/Search/HSP/FastaHSP.pm \
	Bio/Graphics/Util.pm \
	$(INST_LIB)/Bio/Graphics/Util.pm \
	Bio/DB/Flat/BDB.pm \
	$(INST_LIB)/Bio/DB/Flat/BDB.pm \
	Bio/AlignIO/mase.pm \
	$(INST_LIB)/Bio/AlignIO/mase.pm \
	Bio/Graphics/Glyph/diamond.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/diamond.pm \
	Bio/SeqFeature/Gene/Transcript.pm \
	$(INST_LIB)/Bio/SeqFeature/Gene/Transcript.pm \
	Bio/Graphics/Glyph/ellipse.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/ellipse.pm \
	Bio/SeqIO/scf.pm \
	$(INST_LIB)/Bio/SeqIO/scf.pm \
	Bio/MapIO/mapmaker.pm \
	$(INST_LIB)/Bio/MapIO/mapmaker.pm \
	Bio/OntologyIO/InterProParser.pm \
	$(INST_LIB)/Bio/OntologyIO/InterProParser.pm \
	Bio/Ontology/OntologyStore.pm \
	$(INST_LIB)/Bio/Ontology/OntologyStore.pm \
	Bio/DB/GFF/Adaptor/dbi/caching_handle.pm \
	$(INST_LIB)/Bio/DB/GFF/Adaptor/dbi/caching_handle.pm \
	Bio/Location/Atomic.pm \
	$(INST_LIB)/Bio/Location/Atomic.pm \
	Bio/Ontology/Ontology.pm \
	$(INST_LIB)/Bio/Ontology/Ontology.pm \
	Bio/SeqIO/fasta.pm \
	$(INST_LIB)/Bio/SeqIO/fasta.pm \
	Bio/Root/Root.pm \
	$(INST_LIB)/Bio/Root/Root.pm \
	Bio/Graphics/Glyph/oval.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/oval.pm \
	Bio/DB/GFF/Aggregator/ucsc_twinscan.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_twinscan.pm \
	Bio/Graphics/Glyph/anchored_arrow.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/anchored_arrow.pm \
	Bio/SeqFeature/Similarity.pm \
	$(INST_LIB)/Bio/SeqFeature/Similarity.pm \
	Bio/Tools/Sigcleave.pm \
	$(INST_LIB)/Bio/Tools/Sigcleave.pm \
	Bio/Tools/Alignment/Consed.pm \
	$(INST_LIB)/Bio/Tools/Alignment/Consed.pm \
	Bio/Tools/Alignment/Trim.pm \
	$(INST_LIB)/Bio/Tools/Alignment/Trim.pm \
	Bio/Phenotype/OMIM/OMIMentry.pm \
	$(INST_LIB)/Bio/Phenotype/OMIM/OMIMentry.pm \
	Bio/Map/MarkerI.pm \
	$(INST_LIB)/Bio/Map/MarkerI.pm \
	Bio/SearchIO/waba.pm \
	$(INST_LIB)/Bio/SearchIO/waba.pm \
	Bio/AlignIO/phylip.pm \
	$(INST_LIB)/Bio/AlignIO/phylip.pm \
	Bio/Tools/ESTScan.pm \
	$(INST_LIB)/Bio/Tools/ESTScan.pm \
	Bio/Variation/SeqDiff.pm \
	$(INST_LIB)/Bio/Variation/SeqDiff.pm \
	Bio/DB/BioFetch.pm \
	$(INST_LIB)/Bio/DB/BioFetch.pm \
	Bio/SeqIO/phd.pm \
	$(INST_LIB)/Bio/SeqIO/phd.pm \
	Bio/Factory/LocationFactoryI.pm \
	$(INST_LIB)/Bio/Factory/LocationFactoryI.pm \
	Bio/SeqFeature/AnnotationAdaptor.pm \
	$(INST_LIB)/Bio/SeqFeature/AnnotationAdaptor.pm \
	Bio/DB/MANIFEST \
	$(INST_LIB)/Bio/DB/MANIFEST \
	Bio/SeqIO/embl.pm \
	$(INST_LIB)/Bio/SeqIO/embl.pm \
	Bio/Biblio/IO/medline2ref.pm \
	$(INST_LIB)/Bio/Biblio/IO/medline2ref.pm \
	Bio/SeqIO/chadoitext.pm \
	$(INST_LIB)/Bio/SeqIO/chadoitext.pm \
	Bio/Graphics/Glyph/pinsertion.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/pinsertion.pm \
	Bio/Tools/AnalysisResult.pm \
	$(INST_LIB)/Bio/Tools/AnalysisResult.pm \
	Bio/Graphics/Glyph/toomany.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/toomany.pm \
	Bio/Biblio/Journal.pm \
	$(INST_LIB)/Bio/Biblio/Journal.pm \
	Bio/DB/GFF/Aggregator/transcript.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/transcript.pm \
	Bio/Search/HSP/GenericHSP.pm \
	$(INST_LIB)/Bio/Search/HSP/GenericHSP.pm \
	Bio/Coordinate/Result.pm \
	$(INST_LIB)/Bio/Coordinate/Result.pm \
	Bio/Factory/DriverFactory.pm \
	$(INST_LIB)/Bio/Factory/DriverFactory.pm \
	Bio/Biblio/PubmedJournalArticle.pm \
	$(INST_LIB)/Bio/Biblio/PubmedJournalArticle.pm \
	Bio/SeqAnalysisParserI.pm \
	$(INST_LIB)/Bio/SeqAnalysisParserI.pm \
	Bio/Variation/RNAChange.pm \
	$(INST_LIB)/Bio/Variation/RNAChange.pm \
	Bio/Factory/ObjectFactoryI.pm \
	$(INST_LIB)/Bio/Factory/ObjectFactoryI.pm \
	Bio/Search/Result/WABAResult.pm \
	$(INST_LIB)/Bio/Search/Result/WABAResult.pm \
	Bio/SearchIO.pm \
	$(INST_LIB)/Bio/SearchIO.pm \
	Bio/AnalysisParserI.pm \
	$(INST_LIB)/Bio/AnalysisParserI.pm \
	Bio/SearchDist.pm \
	$(INST_LIB)/Bio/SearchDist.pm \
	Bio/Expression/FeatureGroup/FeatureGroupMas50.pm \
	$(INST_LIB)/Bio/Expression/FeatureGroup/FeatureGroupMas50.pm \
	Bio/Tools/ECnumber.pm \
	$(INST_LIB)/Bio/Tools/ECnumber.pm \
	Bio/LiveSeq/Range.pm \
	$(INST_LIB)/Bio/LiveSeq/Range.pm \
	Bio/Tools/Pseudowise.pm \
	$(INST_LIB)/Bio/Tools/Pseudowise.pm \
	Bio/Tools/Genscan.pm \
	$(INST_LIB)/Bio/Tools/Genscan.pm \
	Bio/OntologyIO/soflat.pm \
	$(INST_LIB)/Bio/OntologyIO/soflat.pm \
	Bio/Tools/GFF.pm \
	$(INST_LIB)/Bio/Tools/GFF.pm \
	Bio/Tools/Blast.pm \
	$(INST_LIB)/Bio/Tools/Blast.pm \
	Bio/DB/GFF/Util/Rearrange.pm \
	$(INST_LIB)/Bio/DB/GFF/Util/Rearrange.pm \
	Bio/AnnotationCollectionI.pm \
	$(INST_LIB)/Bio/AnnotationCollectionI.pm \
	Bio/Das/SegmentI.pm \
	$(INST_LIB)/Bio/Das/SegmentI.pm \
	Bio/SeqFeatureI.pm \
	$(INST_LIB)/Bio/SeqFeatureI.pm \
	Bio/Das/FeatureTypeI.pm \
	$(INST_LIB)/Bio/Das/FeatureTypeI.pm \
	Bio/Graphics/Glyph/graded_segments.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/graded_segments.pm \
	bptutorial.pl \
	$(INST_LIB)/bptutorial.pl \
	Bio/Graphics/Glyph/dna.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/dna.pm \
	Bio/Structure/Model.pm \
	$(INST_LIB)/Bio/Structure/Model.pm \
	Bio/Seq/SeqFastaSpeedFactory.pm \
	$(INST_LIB)/Bio/Seq/SeqFastaSpeedFactory.pm \
	Bio/Tree/Node.pm \
	$(INST_LIB)/Bio/Tree/Node.pm \
	Bio/LiveSeq/Repeat_Region.pm \
	$(INST_LIB)/Bio/LiveSeq/Repeat_Region.pm \
	Bio/Annotation/Reference.pm \
	$(INST_LIB)/Bio/Annotation/Reference.pm \
	Bio/Coordinate/MapperI.pm \
	$(INST_LIB)/Bio/Coordinate/MapperI.pm \
	Bio/DB/WebDBSeqI.pm \
	$(INST_LIB)/Bio/DB/WebDBSeqI.pm \
	Bio/TreeIO/tabtree.pm \
	$(INST_LIB)/Bio/TreeIO/tabtree.pm \
	Bio/Tools/Primer3.pm \
	$(INST_LIB)/Bio/Tools/Primer3.pm \
	Bio/Seq/SeqFactory.pm \
	$(INST_LIB)/Bio/Seq/SeqFactory.pm \
	Bio/Tools/Seg.pm \
	$(INST_LIB)/Bio/Tools/Seg.pm \
	Bio/Search/Result/ResultFactory.pm \
	$(INST_LIB)/Bio/Search/Result/ResultFactory.pm \
	Bio/Phenotype/OMIM/OMIMentryAllelicVariant.pm \
	$(INST_LIB)/Bio/Phenotype/OMIM/OMIMentryAllelicVariant.pm \
	Bio/Tools/Phylo/Phylip/ProtDist.pm \
	$(INST_LIB)/Bio/Tools/Phylo/Phylip/ProtDist.pm \
	Bio/SeqIO/genbank.pm \
	$(INST_LIB)/Bio/SeqIO/genbank.pm \
	Bio/PrimarySeq.pm \
	$(INST_LIB)/Bio/PrimarySeq.pm \
	Bio/AlignIO/pfam.pm \
	$(INST_LIB)/Bio/AlignIO/pfam.pm \
	Bio/AlignIO/nexus.pm \
	$(INST_LIB)/Bio/AlignIO/nexus.pm \
	Bio/DB/GFF/Adaptor/dbi.pm \
	$(INST_LIB)/Bio/DB/GFF/Adaptor/dbi.pm \
	Bio/DB/GFF/Aggregator/none.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/none.pm \
	Bio/Tools/Signalp.pm \
	$(INST_LIB)/Bio/Tools/Signalp.pm \
	Bio/Location/CoordinatePolicyI.pm \
	$(INST_LIB)/Bio/Location/CoordinatePolicyI.pm \
	LocalConfig.pm \
	$(INST_LIB)/LocalConfig.pm \
	Bio/AlignIO/bl2seq.pm \
	$(INST_LIB)/Bio/AlignIO/bl2seq.pm \
	Bio/Root/HTTPget.pm \
	$(INST_LIB)/Bio/Root/HTTPget.pm \
	Bio/Ontology/SimpleOntologyEngine.pm \
	$(INST_LIB)/Bio/Ontology/SimpleOntologyEngine.pm \
	Bio/Tools/BPbl2seq.pm \
	$(INST_LIB)/Bio/Tools/BPbl2seq.pm \
	Bio/Variation/IO.pm \
	$(INST_LIB)/Bio/Variation/IO.pm \
	Bio/Ontology/PathI.pm \
	$(INST_LIB)/Bio/Ontology/PathI.pm \
	Bio/AnnotationI.pm \
	$(INST_LIB)/Bio/AnnotationI.pm \
	Bio/Tools/WWW.pm \
	$(INST_LIB)/Bio/Tools/WWW.pm \
	Bio/Seq/LargePrimarySeq.pm \
	$(INST_LIB)/Bio/Seq/LargePrimarySeq.pm \
	Bio/Annotation/Collection.pm \
	$(INST_LIB)/Bio/Annotation/Collection.pm \
	Bio/Tools/IUPAC.pm \
	$(INST_LIB)/Bio/Tools/IUPAC.pm \
	Bio/Graphics/Glyph/triangle.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/triangle.pm \
	Bio/Graphics.pm \
	$(INST_LIB)/Bio/Graphics.pm \
	Bio/SeqIO/abi.pm \
	$(INST_LIB)/Bio/SeqIO/abi.pm \
	Bio/DB/GFF/Aggregator/ucsc_refgene.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_refgene.pm \
	Bio/AlignIO/msf.pm \
	$(INST_LIB)/Bio/AlignIO/msf.pm \
	Bio/LiveSeq/Intron.pm \
	$(INST_LIB)/Bio/LiveSeq/Intron.pm \
	Bio/DB/GFF/Aggregator/clone.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/clone.pm \
	Bio/Map/Position.pm \
	$(INST_LIB)/Bio/Map/Position.pm \
	Bio/PrimarySeqI.pm \
	$(INST_LIB)/Bio/PrimarySeqI.pm \
	Bio/Graphics/Glyph/alignment.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/alignment.pm \
	Bio/Root/IOManager.pm \
	$(INST_LIB)/Bio/Root/IOManager.pm \
	Bio/LiveSeq/Gene.pm \
	$(INST_LIB)/Bio/LiveSeq/Gene.pm \
	Bio/Ontology/Relationship.pm \
	$(INST_LIB)/Bio/Ontology/Relationship.pm \
	Bio/Search/Hit/HitFactory.pm \
	$(INST_LIB)/Bio/Search/Hit/HitFactory.pm \
	Bio/SearchIO/exonerate.pm \
	$(INST_LIB)/Bio/SearchIO/exonerate.pm \
	Bio/Factory/HitFactoryI.pm \
	$(INST_LIB)/Bio/Factory/HitFactoryI.pm \
	Bio/SimpleAlign.pm \
	$(INST_LIB)/Bio/SimpleAlign.pm \
	Bio/Tools/Blast/Sbjct.pm \
	$(INST_LIB)/Bio/Tools/Blast/Sbjct.pm \
	Bio/Tools/Phylo/PAML.pm \
	$(INST_LIB)/Bio/Tools/Phylo/PAML.pm \
	Bio/Factory/ApplicationFactoryI.pm \
	$(INST_LIB)/Bio/Factory/ApplicationFactoryI.pm \
	Bio/DB/GFF/Typename.pm \
	$(INST_LIB)/Bio/DB/GFF/Typename.pm \
	Bio/Search/HSP/HSPFactory.pm \
	$(INST_LIB)/Bio/Search/HSP/HSPFactory.pm \
	Bio/DB/GFF/Homol.pm \
	$(INST_LIB)/Bio/DB/GFF/Homol.pm \
	Bio/Event/EventGeneratorI.pm \
	$(INST_LIB)/Bio/Event/EventGeneratorI.pm \
	Bio/AlignIO/psi.pm \
	$(INST_LIB)/Bio/AlignIO/psi.pm \
	Bio/Graphics/Glyph/redgreen_box.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/redgreen_box.pm \
	Bio/Tools/BPlite/Sbjct.pm \
	$(INST_LIB)/Bio/Tools/BPlite/Sbjct.pm \
	Bio/Graphics/FeatureFile.pm \
	$(INST_LIB)/Bio/Graphics/FeatureFile.pm \
	Bio/Map/Microsatellite.pm \
	$(INST_LIB)/Bio/Map/Microsatellite.pm \
	Bio/LiveSeq/Translation.pm \
	$(INST_LIB)/Bio/LiveSeq/Translation.pm \
	Bio/Search/Hit/HitI.pm \
	$(INST_LIB)/Bio/Search/Hit/HitI.pm \
	Bio/DB/SeqI.pm \
	$(INST_LIB)/Bio/DB/SeqI.pm \
	Bio/Index/SwissPfam.pm \
	$(INST_LIB)/Bio/Index/SwissPfam.pm \
	Bio/DB/GenBank.pm \
	$(INST_LIB)/Bio/DB/GenBank.pm \
	Bio/DB/Flat/BDB/fasta.pm \
	$(INST_LIB)/Bio/DB/Flat/BDB/fasta.pm \
	Bio/Seq/SequenceTrace.pm \
	$(INST_LIB)/Bio/Seq/SequenceTrace.pm \
	Bio/Tools/BPlite/HSP.pm \
	$(INST_LIB)/Bio/Tools/BPlite/HSP.pm \
	Bio/SeqIO/chado.pm \
	$(INST_LIB)/Bio/SeqIO/chado.pm \
	Bio/Index/EMBL.pm \
	$(INST_LIB)/Bio/Index/EMBL.pm \
	Bio/DB/GFF/Aggregator/coding.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/coding.pm \
	Bio/SeqFeature/Gene/GeneStructureI.pm \
	$(INST_LIB)/Bio/SeqFeature/Gene/GeneStructureI.pm \
	Bio/ClusterIO.pm \
	$(INST_LIB)/Bio/ClusterIO.pm \
	Bio/DB/EMBL.pm \
	$(INST_LIB)/Bio/DB/EMBL.pm \
	Bio/DB/DBFetch.pm \
	$(INST_LIB)/Bio/DB/DBFetch.pm \
	Bio/Tools/Blast/README \
	$(INST_LIB)/Bio/Tools/Blast/README \
	Bio/Map/CytoPosition.pm \
	$(INST_LIB)/Bio/Map/CytoPosition.pm \
	Bio/Phenotype/Correlate.pm \
	$(INST_LIB)/Bio/Phenotype/Correlate.pm \
	Bio/Graphics/Glyph/Factory.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/Factory.pm \
	Bio/Assembly/IO/phrap.pm \
	$(INST_LIB)/Bio/Assembly/IO/phrap.pm \
	Bio/Ontology/Path.pm \
	$(INST_LIB)/Bio/Ontology/Path.pm \
	Bio/LiveSeq/Prim_Transcript.pm \
	$(INST_LIB)/Bio/LiveSeq/Prim_Transcript.pm \
	Bio/Tools/Promoterwise.pm \
	$(INST_LIB)/Bio/Tools/Promoterwise.pm \
	Bio/Tree/NodeNHX.pm \
	$(INST_LIB)/Bio/Tree/NodeNHX.pm \
	Bio/AlignIO/clustalw.pm \
	$(INST_LIB)/Bio/AlignIO/clustalw.pm \
	Bio/DB/Ace.pm \
	$(INST_LIB)/Bio/DB/Ace.pm \
	Bio/DB/GFF/Adaptor/dbi/mysql.pm \
	$(INST_LIB)/Bio/DB/GFF/Adaptor/dbi/mysql.pm \
	Bio/Factory/FTLocationFactory.pm \
	$(INST_LIB)/Bio/Factory/FTLocationFactory.pm \
	Bio/Align/DNAStatistics.pm \
	$(INST_LIB)/Bio/Align/DNAStatistics.pm \
	Bio/Coordinate/Utils.pm \
	$(INST_LIB)/Bio/Coordinate/Utils.pm \
	Bio/Root/Vector.pm \
	$(INST_LIB)/Bio/Root/Vector.pm \
	Bio/LiveSeq/IO/BioPerl.pm \
	$(INST_LIB)/Bio/LiveSeq/IO/BioPerl.pm \
	Bio/Variation/IO/flat.pm \
	$(INST_LIB)/Bio/Variation/IO/flat.pm \
	Bio/SeqIO/chadoxml.pm \
	$(INST_LIB)/Bio/SeqIO/chadoxml.pm \
	Bio/Biblio/Ref.pm \
	$(INST_LIB)/Bio/Biblio/Ref.pm \
	Bio/DB/Query/GenBank.pm \
	$(INST_LIB)/Bio/DB/Query/GenBank.pm \
	Bio/Search/GenericDatabase.pm \
	$(INST_LIB)/Bio/Search/GenericDatabase.pm \
	Bio/Factory/SequenceProcessorI.pm \
	$(INST_LIB)/Bio/Factory/SequenceProcessorI.pm \
	Bio/Location/AvWithinCoordPolicy.pm \
	$(INST_LIB)/Bio/Location/AvWithinCoordPolicy.pm \
	Bio/Biblio/TechReport.pm \
	$(INST_LIB)/Bio/Biblio/TechReport.pm \
	Bio/Structure/SecStr/DSSP/Res.pm \
	$(INST_LIB)/Bio/Structure/SecStr/DSSP/Res.pm \
	Bio/Graphics/Glyph/translation.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/translation.pm \
	Bio/SeqI.pm \
	$(INST_LIB)/Bio/SeqI.pm \
	Bio/Location/Simple.pm \
	$(INST_LIB)/Bio/Location/Simple.pm \
	Bio/SeqIO/gcg.pm \
	$(INST_LIB)/Bio/SeqIO/gcg.pm \
	Bio/SeqIO/alf.pm \
	$(INST_LIB)/Bio/SeqIO/alf.pm \
	Bio/SearchIO/SearchResultEventBuilder.pm \
	$(INST_LIB)/Bio/SearchIO/SearchResultEventBuilder.pm \
	Bio/Seq/QualI.pm \
	$(INST_LIB)/Bio/Seq/QualI.pm \
	Bio/Map/LinkagePosition.pm \
	$(INST_LIB)/Bio/Map/LinkagePosition.pm \
	Bio/Ontology/SimpleGOEngine.pm \
	$(INST_LIB)/Bio/Ontology/SimpleGOEngine.pm \
	Bio/SearchIO/psiblast.pm \
	$(INST_LIB)/Bio/SearchIO/psiblast.pm \
	Bio/Cluster/ClusterFactory.pm \
	$(INST_LIB)/Bio/Cluster/ClusterFactory.pm \
	Bio/SeqFeature/Gene/Poly_A_site.pm \
	$(INST_LIB)/Bio/SeqFeature/Gene/Poly_A_site.pm \
	Bio/SeqFeature/Gene/ExonI.pm \
	$(INST_LIB)/Bio/SeqFeature/Gene/ExonI.pm \
	Bio/Tools/Hmmpfam.pm \
	$(INST_LIB)/Bio/Tools/Hmmpfam.pm \
	Bio/Expression/FeatureGroup.pm \
	$(INST_LIB)/Bio/Expression/FeatureGroup.pm \
	Bio/Graphics/Glyph/primers.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/primers.pm \
	Bio/DB/GFF/Aggregator/ucsc_genscan.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_genscan.pm \
	Bio/IdentifiableI.pm \
	$(INST_LIB)/Bio/IdentifiableI.pm \
	Bio/Graphics/Glyph/heterogeneous_segments.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/heterogeneous_segments.pm \
	Bio/Ontology/InterProTerm.pm \
	$(INST_LIB)/Bio/Ontology/InterProTerm.pm \
	Bio/SeqIO/game/featureHandler.pm \
	$(INST_LIB)/Bio/SeqIO/game/featureHandler.pm \
	Bio/Graphics/Glyph/crossbox.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/crossbox.pm \
	Bio/Coordinate/Chain.pm \
	$(INST_LIB)/Bio/Coordinate/Chain.pm \
	Bio/Tools/Grail.pm \
	$(INST_LIB)/Bio/Tools/Grail.pm \
	Bio/Tools/Blast/HTML.pm \
	$(INST_LIB)/Bio/Tools/Blast/HTML.pm \
	Bio/DB/InMemoryCache.pm \
	$(INST_LIB)/Bio/DB/InMemoryCache.pm \
	Bio/DB/GFF/Segment.pm \
	$(INST_LIB)/Bio/DB/GFF/Segment.pm \
	Bio/Symbol/DNAAlphabet.pm \
	$(INST_LIB)/Bio/Symbol/DNAAlphabet.pm \
	Bio/OntologyIO/goflat.pm \
	$(INST_LIB)/Bio/OntologyIO/goflat.pm \
	Bio/Tools/Blast/HSP.pm \
	$(INST_LIB)/Bio/Tools/Blast/HSP.pm \
	Bio/DB/FileCache.pm \
	$(INST_LIB)/Bio/DB/FileCache.pm \
	Bio/DB/GFF/Util/Binning.pm \
	$(INST_LIB)/Bio/DB/GFF/Util/Binning.pm \
	Bio/SearchIO/blast.pm \
	$(INST_LIB)/Bio/SearchIO/blast.pm \
	Bio/Species.pm \
	$(INST_LIB)/Bio/Species.pm \
	Bio/Biblio/Person.pm \
	$(INST_LIB)/Bio/Biblio/Person.pm \
	Bio/Map/Marker.pm \
	$(INST_LIB)/Bio/Map/Marker.pm \
	Bio/Tools/CodonTable.pm \
	$(INST_LIB)/Bio/Tools/CodonTable.pm \
	Bio/SeqIO/bsml.pm \
	$(INST_LIB)/Bio/SeqIO/bsml.pm \
	Bio/Tools/AlignFactory.pm \
	$(INST_LIB)/Bio/Tools/AlignFactory.pm \
	Bio/DB/QueryI.pm \
	$(INST_LIB)/Bio/DB/QueryI.pm \
	Bio/Graphics/Glyph/segments.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/segments.pm \
	Bio/Seq/SeqWithQuality.pm \
	$(INST_LIB)/Bio/Seq/SeqWithQuality.pm \
	Bio/Annotation/DBLink.pm \
	$(INST_LIB)/Bio/Annotation/DBLink.pm \
	Bio/SearchIO/FastHitEventBuilder.pm \
	$(INST_LIB)/Bio/SearchIO/FastHitEventBuilder.pm \
	Bio/DB/XEMBL.pm \
	$(INST_LIB)/Bio/DB/XEMBL.pm \
	Bio/DB/GFF/Adaptor/dbi/mysqlopt.pm \
	$(INST_LIB)/Bio/DB/GFF/Adaptor/dbi/mysqlopt.pm \
	Bio/Search/Hit/BlastHit.pm \
	$(INST_LIB)/Bio/Search/Hit/BlastHit.pm \
	Bio/Graphics/Glyph/redgreen_segment.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/redgreen_segment.pm \
	Bio/Variation/AAReverseMutate.pm \
	$(INST_LIB)/Bio/Variation/AAReverseMutate.pm \
	Bio/Tools/Prediction/Gene.pm \
	$(INST_LIB)/Bio/Tools/Prediction/Gene.pm \
	Bio/SeqFeature/Primer.pm \
	$(INST_LIB)/Bio/SeqFeature/Primer.pm \
	Bio/Search/SearchUtils.pm \
	$(INST_LIB)/Bio/Search/SearchUtils.pm \
	Bio/Biblio/WebResource.pm \
	$(INST_LIB)/Bio/Biblio/WebResource.pm \
	Bio/AlignIO/fasta.pm \
	$(INST_LIB)/Bio/AlignIO/fasta.pm \
	Bio/Tools/Phylo/Molphy.pm \
	$(INST_LIB)/Bio/Tools/Phylo/Molphy.pm \
	Bio/Search/Hit/HMMERHit.pm \
	$(INST_LIB)/Bio/Search/Hit/HMMERHit.pm \
	Bio/Tools/Run/StandAloneBlast.pm \
	$(INST_LIB)/Bio/Tools/Run/StandAloneBlast.pm \
	Bio/Root/Err.pm \
	$(INST_LIB)/Bio/Root/Err.pm \
	Bio/Tools/MZEF.pm \
	$(INST_LIB)/Bio/Tools/MZEF.pm \
	Bio/SearchIO/Writer/ResultTableWriter.pm \
	$(INST_LIB)/Bio/SearchIO/Writer/ResultTableWriter.pm \
	Bio/Ontology/RelationshipFactory.pm \
	$(INST_LIB)/Bio/Ontology/RelationshipFactory.pm \
	Bio/Seq/RichSeqI.pm \
	$(INST_LIB)/Bio/Seq/RichSeqI.pm \
	Bio/Tools/Prediction/Exon.pm \
	$(INST_LIB)/Bio/Tools/Prediction/Exon.pm \
	Bio/SearchIO/EventHandlerI.pm \
	$(INST_LIB)/Bio/SearchIO/EventHandlerI.pm \
	Bio/SearchIO/Writer/HitTableWriter.pm \
	$(INST_LIB)/Bio/SearchIO/Writer/HitTableWriter.pm \
	Bio/DB/GFF/Adaptor/biofetch.pm \
	$(INST_LIB)/Bio/DB/GFF/Adaptor/biofetch.pm \
	Bio/Tools/Est2Genome.pm \
	$(INST_LIB)/Bio/Tools/Est2Genome.pm \
	Bio/SearchIO/chadosxpr.pm \
	$(INST_LIB)/Bio/SearchIO/chadosxpr.pm \
	Bio/Search/HSP/HMMERHSP.pm \
	$(INST_LIB)/Bio/Search/HSP/HMMERHSP.pm \
	Bio/Variation/DNAMutation.pm \
	$(INST_LIB)/Bio/Variation/DNAMutation.pm \
	Bio/Factory/MapFactoryI.pm \
	$(INST_LIB)/Bio/Factory/MapFactoryI.pm \
	Bio/Structure/IO.pm \
	$(INST_LIB)/Bio/Structure/IO.pm \
	Bio/Tree/Tree.pm \
	$(INST_LIB)/Bio/Tree/Tree.pm \
	Bio/Variation/SNP.pm \
	$(INST_LIB)/Bio/Variation/SNP.pm \
	Bio/SearchIO/SearchWriterI.pm \
	$(INST_LIB)/Bio/SearchIO/SearchWriterI.pm \
	Bio/Factory/AnalysisI.pm \
	$(INST_LIB)/Bio/Factory/AnalysisI.pm \
	Bio/SeqFeature/Gene/Promoter.pm \
	$(INST_LIB)/Bio/SeqFeature/Gene/Promoter.pm \
	Bio/SeqFeature/FeaturePair.pm \
	$(INST_LIB)/Bio/SeqFeature/FeaturePair.pm \
	Bio/Graphics/Glyph/transcript2.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/transcript2.pm \
	Bio/Location/Split.pm \
	$(INST_LIB)/Bio/Location/Split.pm \
	Bio/SeqIO/exp.pm \
	$(INST_LIB)/Bio/SeqIO/exp.pm \
	Bio/Graphics/Glyph/transcript.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/transcript.pm \
	Bio/Biblio/IO/pubmedxml.pm \
	$(INST_LIB)/Bio/Biblio/IO/pubmedxml.pm \
	Bio/Search/Result/GenericResult.pm \
	$(INST_LIB)/Bio/Search/Result/GenericResult.pm \
	Bio/Cluster/SequenceFamily.pm \
	$(INST_LIB)/Bio/Cluster/SequenceFamily.pm \
	Bio/SeqIO/swiss.pm \
	$(INST_LIB)/Bio/SeqIO/swiss.pm \
	Bio/DB/Makefile.PL \
	$(INST_LIB)/Bio/DB/Makefile.PL \
	Bio/ClusterIO/unigene.pm \
	$(INST_LIB)/Bio/ClusterIO/unigene.pm \
	Bio/DB/XEMBLService.pm \
	$(INST_LIB)/Bio/DB/XEMBLService.pm \
	Bio/Annotation/StructuredValue.pm \
	$(INST_LIB)/Bio/Annotation/StructuredValue.pm \
	Bio/Tools/FootPrinter.pm \
	$(INST_LIB)/Bio/Tools/FootPrinter.pm \
	Bio/SeqIO/FTHelper.pm \
	$(INST_LIB)/Bio/SeqIO/FTHelper.pm \
	Bio/Graphics/Glyph/dot.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/dot.pm \
	Bio/Tools/Coil.pm \
	$(INST_LIB)/Bio/Tools/Coil.pm \
	Bio/Coordinate/Collection.pm \
	$(INST_LIB)/Bio/Coordinate/Collection.pm \
	Bio/Root/IO.pm \
	$(INST_LIB)/Bio/Root/IO.pm \
	Bio/Tools/Lucy.pm \
	$(INST_LIB)/Bio/Tools/Lucy.pm \
	Bio/Biblio/MedlineBook.pm \
	$(INST_LIB)/Bio/Biblio/MedlineBook.pm \
	Bio/Symbol/Symbol.pm \
	$(INST_LIB)/Bio/Symbol/Symbol.pm \
	Bio/Ontology/GOterm.pm \
	$(INST_LIB)/Bio/Ontology/GOterm.pm \
	Bio/DB/GFF/Featname.pm \
	$(INST_LIB)/Bio/DB/GFF/Featname.pm \
	Bio/Symbol/Alphabet.pm \
	$(INST_LIB)/Bio/Symbol/Alphabet.pm \
	Bio/Graphics/Glyph/xyplot.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/xyplot.pm \
	Bio/LocatableSeq.pm \
	$(INST_LIB)/Bio/LocatableSeq.pm \
	Bio/Factory/ObjectBuilderI.pm \
	$(INST_LIB)/Bio/Factory/ObjectBuilderI.pm \
	Bio/SeqIO/ztr.pm \
	$(INST_LIB)/Bio/SeqIO/ztr.pm \
	Bio/DB/Universal.pm \
	$(INST_LIB)/Bio/DB/Universal.pm \
	Bio/Tools/OddCodes.pm \
	$(INST_LIB)/Bio/Tools/OddCodes.pm \
	Bio/Structure/Entry.pm \
	$(INST_LIB)/Bio/Structure/Entry.pm \
	Bio/Biblio/JournalArticle.pm \
	$(INST_LIB)/Bio/Biblio/JournalArticle.pm \
	Bio/Coordinate/Pair.pm \
	$(INST_LIB)/Bio/Coordinate/Pair.pm \
	Bio/DB/GFF/Aggregator/processed_transcript.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/processed_transcript.pm \
	Bio/Coordinate/ResultI.pm \
	$(INST_LIB)/Bio/Coordinate/ResultI.pm \
	Bio/Graphics/Glyph/arrow.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/arrow.pm \
	Bio/Graphics/Panel.pm \
	$(INST_LIB)/Bio/Graphics/Panel.pm \
	Bio/ClusterI.pm \
	$(INST_LIB)/Bio/ClusterI.pm \
	Bio/Index/Fasta.pm \
	$(INST_LIB)/Bio/Index/Fasta.pm \
	Bio/Align/StatisticsI.pm \
	$(INST_LIB)/Bio/Align/StatisticsI.pm \
	Bio/SeqFeature/PositionProxy.pm \
	$(INST_LIB)/Bio/SeqFeature/PositionProxy.pm \
	Bio/Tools/SeqAnal.pm \
	$(INST_LIB)/Bio/Tools/SeqAnal.pm \
	Bio/LiveSeq/IO/README \
	$(INST_LIB)/Bio/LiveSeq/IO/README \
	Bio/DB/GFF/Adaptor/ace.pm \
	$(INST_LIB)/Bio/DB/GFF/Adaptor/ace.pm \
	Bio/Structure/SecStr/STRIDE/Res.pm \
	$(INST_LIB)/Bio/Structure/SecStr/STRIDE/Res.pm \
	Bio/Biblio/MedlineBookArticle.pm \
	$(INST_LIB)/Bio/Biblio/MedlineBookArticle.pm \
	Bio/Graphics/Glyph/ruler_arrow.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/ruler_arrow.pm \
	Bio/Tools/Genemark.pm \
	$(INST_LIB)/Bio/Tools/Genemark.pm \
	Bio/Biblio/Patent.pm \
	$(INST_LIB)/Bio/Biblio/Patent.pm \
	Bio/Seq/RichSeq.pm \
	$(INST_LIB)/Bio/Seq/RichSeq.pm \
	Bio/DB/GFF/Aggregator.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator.pm \
	Bio/DB/GFF.pm \
	$(INST_LIB)/Bio/DB/GFF.pm \
	Bio/Tools/BPpsilite.pm \
	$(INST_LIB)/Bio/Tools/BPpsilite.pm \
	Bio/Graphics/Glyph/track.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/track.pm \
	Bio/Factory/ResultFactoryI.pm \
	$(INST_LIB)/Bio/Factory/ResultFactoryI.pm \
	Bio/SeqFeature/CollectionI.pm \
	$(INST_LIB)/Bio/SeqFeature/CollectionI.pm \
	Bio/Biblio/MedlineArticle.pm \
	$(INST_LIB)/Bio/Biblio/MedlineArticle.pm \
	Bio/Phenotype/PhenotypeI.pm \
	$(INST_LIB)/Bio/Phenotype/PhenotypeI.pm \
	Bio/Taxonomy/Taxon.pm \
	$(INST_LIB)/Bio/Taxonomy/Taxon.pm \
	Bio/Seq/LargeSeq.pm \
	$(INST_LIB)/Bio/Seq/LargeSeq.pm \
	Bio/DB/GDB.pm \
	$(INST_LIB)/Bio/DB/GDB.pm \
	Bio/Tools/Profile.pm \
	$(INST_LIB)/Bio/Tools/Profile.pm \
	Bio/Seq/EncodedSeq.pm \
	$(INST_LIB)/Bio/Seq/EncodedSeq.pm \
	Bio/Factory/EMBOSS.pm \
	$(INST_LIB)/Bio/Factory/EMBOSS.pm \
	Bio/Variation/README \
	$(INST_LIB)/Bio/Variation/README \
	Bio/DB/Registry.pm \
	$(INST_LIB)/Bio/DB/Registry.pm \
	Bio/Variation/VariantI.pm \
	$(INST_LIB)/Bio/Variation/VariantI.pm \
	Bio/Root/Utilities.pm \
	$(INST_LIB)/Bio/Root/Utilities.pm \
	Bio/Graphics/Glyph/line.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/line.pm \
	Bio/Factory/SequenceFactoryI.pm \
	$(INST_LIB)/Bio/Factory/SequenceFactoryI.pm \
	Bio/Search/Hit/GenericHit.pm \
	$(INST_LIB)/Bio/Search/Hit/GenericHit.pm \
	Bio/Tools/RestrictionEnzyme.pm \
	$(INST_LIB)/Bio/Tools/RestrictionEnzyme.pm \
	Bio/Map/LinkageMap.pm \
	$(INST_LIB)/Bio/Map/LinkageMap.pm \
	Bio/AlignIO/stockholm.pm \
	$(INST_LIB)/Bio/AlignIO/stockholm.pm \
	Bio/OntologyIO/Handlers/InterProHandler.pm \
	$(INST_LIB)/Bio/OntologyIO/Handlers/InterProHandler.pm \
	Bio/DB/GFF/Adaptor/dbi/mysqlace.pm \
	$(INST_LIB)/Bio/DB/GFF/Adaptor/dbi/mysqlace.pm \
	Bio/SeqFeature/Computation.pm \
	$(INST_LIB)/Bio/SeqFeature/Computation.pm \
	Bio/DasI.pm \
	$(INST_LIB)/Bio/DasI.pm \
	Bio/Graphics/Glyph/splice_site.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/splice_site.pm \
	Bio/TreeIO/nhx.pm \
	$(INST_LIB)/Bio/TreeIO/nhx.pm \
	Bio/SeqUtils.pm \
	$(INST_LIB)/Bio/SeqUtils.pm \
	Bio/Assembly/Scaffold.pm \
	$(INST_LIB)/Bio/Assembly/Scaffold.pm \
	Bio/Assembly/Contig.pm \
	$(INST_LIB)/Bio/Assembly/Contig.pm \
	Bio/Structure/Residue.pm \
	$(INST_LIB)/Bio/Structure/Residue.pm \
	Bio/Tools/Sim4/Results.pm \
	$(INST_LIB)/Bio/Tools/Sim4/Results.pm \
	Bio/DB/GFF/Adaptor/dbi/iterator.pm \
	$(INST_LIB)/Bio/DB/GFF/Adaptor/dbi/iterator.pm \
	Bio/Variation/AAChange.pm \
	$(INST_LIB)/Bio/Variation/AAChange.pm \
	Bio/Index/Fastq.pm \
	$(INST_LIB)/Bio/Index/Fastq.pm \
	Bio/SearchIO/Writer/TextResultWriter.pm \
	$(INST_LIB)/Bio/SearchIO/Writer/TextResultWriter.pm \
	Bio/Range.pm \
	$(INST_LIB)/Bio/Range.pm \
	Bio/SeqIO/game/idHandler.pm \
	$(INST_LIB)/Bio/SeqIO/game/idHandler.pm \
	Bio/OntologyIO/dagflat.pm \
	$(INST_LIB)/Bio/OntologyIO/dagflat.pm \
	Bio/SeqIO/pir.pm \
	$(INST_LIB)/Bio/SeqIO/pir.pm \
	Bio/Factory/SequenceStreamI.pm \
	$(INST_LIB)/Bio/Factory/SequenceStreamI.pm \
	Bio/Matrix/PhylipDist.pm \
	$(INST_LIB)/Bio/Matrix/PhylipDist.pm \
	Bio/Annotation/Comment.pm \
	$(INST_LIB)/Bio/Annotation/Comment.pm \
	Bio/Tools/BPlite.pm \
	$(INST_LIB)/Bio/Tools/BPlite.pm \
	Bio/Symbol/ProteinAlphabet.pm \
	$(INST_LIB)/Bio/Symbol/ProteinAlphabet.pm \
	Bio/Tools/Genewise.pm \
	$(INST_LIB)/Bio/Tools/Genewise.pm \
	Bio/Biblio/PubmedArticle.pm \
	$(INST_LIB)/Bio/Biblio/PubmedArticle.pm \
	Bio/Perl.pm \
	$(INST_LIB)/Bio/Perl.pm \
	Bio/Search/HSP/HSPI.pm \
	$(INST_LIB)/Bio/Search/HSP/HSPI.pm \
	Bio/LiveSeq/Exon.pm \
	$(INST_LIB)/Bio/LiveSeq/Exon.pm \
	Bio/LiveSeq/Chain.pm \
	$(INST_LIB)/Bio/LiveSeq/Chain.pm \
	Bio/ClusterIO/dbsnp.pm \
	$(INST_LIB)/Bio/ClusterIO/dbsnp.pm \
	Bio/Root/Xref.pm \
	$(INST_LIB)/Bio/Root/Xref.pm \
	Bio/Tools/Run/WrapperBase.pm \
	$(INST_LIB)/Bio/Tools/Run/WrapperBase.pm \
	Bio/AlignIO/meme.pm \
	$(INST_LIB)/Bio/AlignIO/meme.pm \
	Bio/Map/MapI.pm \
	$(INST_LIB)/Bio/Map/MapI.pm \
	Bio/Location/WidestCoordPolicy.pm \
	$(INST_LIB)/Bio/Location/WidestCoordPolicy.pm \
	Bio/Tools/Genomewise.pm \
	$(INST_LIB)/Bio/Tools/Genomewise.pm \
	Bio/DB/Flat/BDB/embl.pm \
	$(INST_LIB)/Bio/DB/Flat/BDB/embl.pm \
	Bio/Search/HSP/BlastHSP.pm \
	$(INST_LIB)/Bio/Search/HSP/BlastHSP.pm \
	Bio/LiveSeq/Mutator.pm \
	$(INST_LIB)/Bio/LiveSeq/Mutator.pm \
	Bio/Phenotype/Measure.pm \
	$(INST_LIB)/Bio/Phenotype/Measure.pm \
	Bio/SeqFeature/Gene/Intron.pm \
	$(INST_LIB)/Bio/SeqFeature/Gene/Intron.pm \
	Bio/Tools/Eponine.pm \
	$(INST_LIB)/Bio/Tools/Eponine.pm \
	Bio/DB/GFF/RelSegment.pm \
	$(INST_LIB)/Bio/DB/GFF/RelSegment.pm \
	Bio/Annotation/TypeManager.pm \
	$(INST_LIB)/Bio/Annotation/TypeManager.pm \
	Bio/DB/GFF/Aggregator/ucsc_ensgene.pm \
	$(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_ensgene.pm \
	Bio/Map/OrderedPosition.pm \
	$(INST_LIB)/Bio/Map/OrderedPosition.pm \
	Bio/SeqIO/MultiFile.pm \
	$(INST_LIB)/Bio/SeqIO/MultiFile.pm \
	Bio/Tools/Tmhmm.pm \
	$(INST_LIB)/Bio/Tools/Tmhmm.pm \
	Bio/Factory/SeqAnalysisParserFactory.pm \
	$(INST_LIB)/Bio/Factory/SeqAnalysisParserFactory.pm \
	Bio/SeqFeature/Gene/UTR.pm \
	$(INST_LIB)/Bio/SeqFeature/Gene/UTR.pm \
	Bio/Map/CytoMarker.pm \
	$(INST_LIB)/Bio/Map/CytoMarker.pm \
	Bio/AlignIO/selex.pm \
	$(INST_LIB)/Bio/AlignIO/selex.pm \
	Bio/Seq/SeqBuilder.pm \
	$(INST_LIB)/Bio/Seq/SeqBuilder.pm \
	Bio/Biblio/Service.pm \
	$(INST_LIB)/Bio/Biblio/Service.pm \
	Bio/Biblio/MedlineJournal.pm \
	$(INST_LIB)/Bio/Biblio/MedlineJournal.pm \
	Bio/DB/GenPept.pm \
	$(INST_LIB)/Bio/DB/GenPept.pm \
	Bio/Index/Blast.pm \
	$(INST_LIB)/Bio/Index/Blast.pm \
	Bio/Search/DatabaseI.pm \
	$(INST_LIB)/Bio/Search/DatabaseI.pm \
	Bio/Graphics/Glyph.pm \
	$(INST_LIB)/Bio/Graphics/Glyph.pm \
	Bio/Symbol/SymbolI.pm \
	$(INST_LIB)/Bio/Symbol/SymbolI.pm \
	Bio/Tools/EPCR.pm \
	$(INST_LIB)/Bio/Tools/EPCR.pm \
	Bio/Location/NarrowestCoordPolicy.pm \
	$(INST_LIB)/Bio/Location/NarrowestCoordPolicy.pm \
	Bio/Tools/pSW.pm \
	$(INST_LIB)/Bio/Tools/pSW.pm \
	Bio/AnnotatableI.pm \
	$(INST_LIB)/Bio/AnnotatableI.pm \
	Bio/Biblio/Proceeding.pm \
	$(INST_LIB)/Bio/Biblio/Proceeding.pm \
	Bio/TreeIO/newick.pm \
	$(INST_LIB)/Bio/TreeIO/newick.pm \
	Bio/SeqFeature/Gene/GeneStructure.pm \
	$(INST_LIB)/Bio/SeqFeature/Gene/GeneStructure.pm \
	Bio/Biblio/Book.pm \
	$(INST_LIB)/Bio/Biblio/Book.pm \
	Bio/Tools/BPlite/Iteration.pm \
	$(INST_LIB)/Bio/Tools/BPlite/Iteration.pm \
	Bio/SearchIO/Writer/HTMLResultWriter.pm \
	$(INST_LIB)/Bio/SearchIO/Writer/HTMLResultWriter.pm \
	Bio/Tree/Statistics.pm \
	$(INST_LIB)/Bio/Tree/Statistics.pm \
	Bio/Align/PairwiseStatistics.pm \
	$(INST_LIB)/Bio/Align/PairwiseStatistics.pm \
	Bio/Assembly/IO.pm \
	$(INST_LIB)/Bio/Assembly/IO.pm \
	Bio/Graphics/Glyph/span.pm \
	$(INST_LIB)/Bio/Graphics/Glyph/span.pm \
	Bio/Ontology/TermFactory.pm \
	$(INST_LIB)/Bio/Ontology/TermFactory.pm \
	Bio/Tools/SeqWords.pm \
	$(INST_LIB)/Bio/Tools/SeqWords.pm \
	Bio/DB/GFF/Adaptor/dbi/oracleace.pm \
	$(INST_LIB)/Bio/DB/GFF/Adaptor/dbi/oracleace.pm \
	Bio/Expression/FeatureI.pm \
	$(INST_LIB)/Bio/Expression/FeatureI.pm \
	Bio/Map/OrderedPositionWithDistance.pm \
	$(INST_LIB)/Bio/Map/OrderedPositionWithDistance.pm \
	Bio/SeqIO/game.pm \
	$(INST_LIB)/Bio/SeqIO/game.pm \
	Bio/Biblio/IO/pubmed2ref.pm \
	$(INST_LIB)/Bio/Biblio/IO/pubmed2ref.pm \
	Bio/Index/Abstract.pm \
	$(INST_LIB)/Bio/Index/Abstract.pm \
	Bio/Tools/HMMER/Domain.pm \
	$(INST_LIB)/Bio/Tools/HMMER/Domain.pm \
	Bio/SearchIO/blastxml.pm \
	$(INST_LIB)/Bio/SearchIO/blastxml.pm \
	Bio/DB/GFF/Adaptor/dbi/oracle.pm \
	$(INST_LIB)/Bio/DB/GFF/Adaptor/dbi/oracle.pm \
	Bio/Coordinate/GeneMapper.pm \
	$(INST_LIB)/Bio/Coordinate/GeneMapper.pm \
	Bio/Biblio/IO/medlinexml.pm \
	$(INST_LIB)/Bio/Biblio/IO/medlinexml.pm \
	Bio/SeqIO/ace.pm \
	$(INST_LIB)/Bio/SeqIO/ace.pm \
	Bio/Location/FuzzyLocationI.pm \
	$(INST_LIB)/Bio/Location/FuzzyLocationI.pm \
	Bio/Search/Hit/Fasta.pm \
	$(INST_LIB)/Bio/Search/Hit/Fasta.pm \
	Bio/Root/Exception.pm \
	$(INST_LIB)/Bio/Root/Exception.pm \
	Bio/Factory/TreeFactoryI.pm \
	$(INST_LIB)/Bio/Factory/TreeFactoryI.pm \
	Bio/Assembly/IO/ace.pm \
	$(INST_LIB)/Bio/Assembly/IO/ace.pm \
	Bio/DB/Biblio/soap.pm \
	$(INST_LIB)/Bio/DB/Biblio/soap.pm \
	Bio/Root/Object.pm \
	$(INST_LIB)/Bio/Root/Object.pm


# --- MakeMaker platform_constants section:
MM_Unix_VERSION = 6.56
PERL_MALLOC_DEF = -DPERL_EXTMALLOC_DEF -Dmalloc=Perl_malloc -Dfree=Perl_mfree -Drealloc=Perl_realloc -Dcalloc=Perl_calloc


# --- MakeMaker tool_autosplit section:
# Usage: $(AUTOSPLITFILE) FileToSplit AutoDirToSplitInto
AUTOSPLITFILE = $(ABSPERLRUN)  -e 'use AutoSplit;  autosplit($$ARGV[0], $$ARGV[1], 0, 1, 1)' --



# --- MakeMaker tool_xsubpp section:


# --- MakeMaker tools_other section:
SHELL = /bin/sh
CHMOD = chmod
CP = cp
MV = mv
NOOP = $(TRUE)
NOECHO = @
RM_F = rm -f
RM_RF = rm -rf
TEST_F = test -f
TOUCH = touch
UMASK_NULL = umask 0
DEV_NULL = > /dev/null 2>&1
MKPATH = $(ABSPERLRUN) -MExtUtils::Command -e 'mkpath' --
EQUALIZE_TIMESTAMP = $(ABSPERLRUN) -MExtUtils::Command -e 'eqtime' --
FALSE = false
TRUE = true
ECHO = echo
ECHO_N = echo -n
UNINST = 0
VERBINST = 0
MOD_INSTALL = $(ABSPERLRUN) -MExtUtils::Install -e 'install([ from_to => {@ARGV}, verbose => '\''$(VERBINST)'\'', uninstall_shadows => '\''$(UNINST)'\'', dir_mode => '\''$(PERM_DIR)'\'' ]);' --
DOC_INSTALL = $(ABSPERLRUN) -MExtUtils::Command::MM -e 'perllocal_install' --
UNINSTALL = $(ABSPERLRUN) -MExtUtils::Command::MM -e 'uninstall' --
WARN_IF_OLD_PACKLIST = $(ABSPERLRUN) -MExtUtils::Command::MM -e 'warn_if_old_packlist' --
MACROSTART = 
MACROEND = 
USEMAKEFILE = -f
FIXIN = $(ABSPERLRUN) -MExtUtils::MY -e 'MY->fixin(shift)' --


# --- MakeMaker makemakerdflt section:
makemakerdflt : all
	$(NOECHO) $(NOOP)


# --- MakeMaker dist section:
TAR = tar
TARFLAGS = cvf
ZIP = zip
ZIPFLAGS = -r
COMPRESS = gzip -9f
SUFFIX = .gz
SHAR = shar
PREOP = $(NOECHO) $(NOOP)
POSTOP = $(NOECHO) $(NOOP)
TO_UNIX = $(NOECHO) $(NOOP)
CI = ci -u
RCS_LABEL = rcs -Nv$(VERSION_SYM): -q
DIST_CP = best
DIST_DEFAULT = all tardist
DISTNAME = bioperl
DISTVNAME = bioperl-1.2.3


# --- MakeMaker macro section:


# --- MakeMaker depend section:


# --- MakeMaker cflags section:


# --- MakeMaker const_loadlibs section:


# --- MakeMaker const_cccmd section:


# --- MakeMaker post_constants section:


# --- MakeMaker pasthru section:

PASTHRU = LIBPERL_A="$(LIBPERL_A)"\
	LINKTYPE="$(LINKTYPE)"\
	PREFIX="$(PREFIX)"


# --- MakeMaker special_targets section:
.SUFFIXES : .xs .c .C .cpp .i .s .cxx .cc $(OBJ_EXT)

.PHONY: all config static dynamic test linkext manifest blibdirs clean realclean disttest distdir



# --- MakeMaker c_o section:


# --- MakeMaker xs_c section:


# --- MakeMaker xs_o section:


# --- MakeMaker top_targets section:
all :: pure_all manifypods
	$(NOECHO) $(NOOP)


pure_all :: config pm_to_blib subdirs linkext
	$(NOECHO) $(NOOP)

subdirs :: $(MYEXTLIB)
	$(NOECHO) $(NOOP)

config :: $(FIRST_MAKEFILE) blibdirs
	$(NOECHO) $(NOOP)

help :
	perldoc ExtUtils::MakeMaker


# --- MakeMaker blibdirs section:
blibdirs : $(INST_LIBDIR)$(DFSEP).exists $(INST_ARCHLIB)$(DFSEP).exists $(INST_AUTODIR)$(DFSEP).exists $(INST_ARCHAUTODIR)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists $(INST_SCRIPT)$(DFSEP).exists $(INST_MAN1DIR)$(DFSEP).exists $(INST_MAN3DIR)$(DFSEP).exists
	$(NOECHO) $(NOOP)

# Backwards compat with 6.18 through 6.25
blibdirs.ts : blibdirs
	$(NOECHO) $(NOOP)

$(INST_LIBDIR)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_LIBDIR)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_LIBDIR)
	$(NOECHO) $(TOUCH) $(INST_LIBDIR)$(DFSEP).exists

$(INST_ARCHLIB)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_ARCHLIB)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_ARCHLIB)
	$(NOECHO) $(TOUCH) $(INST_ARCHLIB)$(DFSEP).exists

$(INST_AUTODIR)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_AUTODIR)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_AUTODIR)
	$(NOECHO) $(TOUCH) $(INST_AUTODIR)$(DFSEP).exists

$(INST_ARCHAUTODIR)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_ARCHAUTODIR)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_ARCHAUTODIR)
	$(NOECHO) $(TOUCH) $(INST_ARCHAUTODIR)$(DFSEP).exists

$(INST_BIN)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_BIN)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_BIN)
	$(NOECHO) $(TOUCH) $(INST_BIN)$(DFSEP).exists

$(INST_SCRIPT)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_SCRIPT)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_SCRIPT)
	$(NOECHO) $(TOUCH) $(INST_SCRIPT)$(DFSEP).exists

$(INST_MAN1DIR)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_MAN1DIR)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_MAN1DIR)
	$(NOECHO) $(TOUCH) $(INST_MAN1DIR)$(DFSEP).exists

$(INST_MAN3DIR)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_MAN3DIR)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_MAN3DIR)
	$(NOECHO) $(TOUCH) $(INST_MAN3DIR)$(DFSEP).exists



# --- MakeMaker linkext section:

linkext :: $(LINKTYPE)
	$(NOECHO) $(NOOP)


# --- MakeMaker dlsyms section:


# --- MakeMaker dynamic section:

dynamic :: $(FIRST_MAKEFILE) $(INST_DYNAMIC) $(INST_BOOT)
	$(NOECHO) $(NOOP)


# --- MakeMaker dynamic_bs section:

BOOTSTRAP =


# --- MakeMaker dynamic_lib section:


# --- MakeMaker static section:

## $(INST_PM) has been moved to the all: target.
## It remains here for awhile to allow for old usage: "make static"
static :: $(FIRST_MAKEFILE) $(INST_STATIC)
	$(NOECHO) $(NOOP)


# --- MakeMaker static_lib section:


# --- MakeMaker manifypods section:

manifypods : pure_all
	@$(NOOP)


# --- MakeMaker processPL section:

all :: biodatabases.pod
	$(NOECHO) $(NOOP)

biodatabases.pod :: biodatabases.PL pm_to_blib
	$(PERLRUNINST) biodatabases.PL biodatabases.pod

all :: biodesign.pod
	$(NOECHO) $(NOOP)

biodesign.pod :: biodesign.PL pm_to_blib
	$(PERLRUNINST) biodesign.PL biodesign.pod

all :: bioperl.pod
	$(NOECHO) $(NOOP)

bioperl.pod :: bioperl.PL pm_to_blib
	$(PERLRUNINST) bioperl.PL bioperl.pod

all :: bioscripts.pod
	$(NOECHO) $(NOOP)

bioscripts.pod :: bioscripts.PL pm_to_blib
	$(PERLRUNINST) bioscripts.PL bioscripts.pod


# --- MakeMaker installbin section:


# --- MakeMaker subdirs section:

# none

# --- MakeMaker clean_subdirs section:
clean_subdirs :
	$(NOECHO) $(NOOP)


# --- MakeMaker clean section:

# Delete temporary files but do not touch installed files. We don't delete
# the Makefile here so a later make realclean still has a makefile to use.

clean :: clean_subdirs
	- $(RM_F) \
	  *$(LIB_EXT) core \
	  core.[0-9] $(INST_ARCHAUTODIR)/extralibs.all \
	  core.[0-9][0-9] $(BASEEXT).bso \
	  pm_to_blib.ts core.[0-9][0-9][0-9][0-9] \
	  $(BASEEXT).x $(BOOTSTRAP) \
	  perl$(EXE_EXT) tmon.out \
	  *$(OBJ_EXT) pm_to_blib \
	  $(INST_ARCHAUTODIR)/extralibs.ld blibdirs.ts \
	  core.[0-9][0-9][0-9][0-9][0-9] *perl.core \
	  core.*perl.*.? $(MAKE_APERL_FILE) \
	  perl $(BASEEXT).def \
	  core.[0-9][0-9][0-9] mon.out \
	  lib$(BASEEXT).def perlmain.c \
	  perl.exe so_locations \
	  $(BASEEXT).exp 
	- $(RM_RF) \
	  blib 
	- $(MV) $(FIRST_MAKEFILE) $(MAKEFILE_OLD) $(DEV_NULL)


# --- MakeMaker realclean_subdirs section:
realclean_subdirs :
	$(NOECHO) $(NOOP)


# --- MakeMaker realclean section:
# Delete temporary files (via clean) and also delete dist files
realclean purge ::  clean realclean_subdirs
	- $(RM_F) \
	  $(MAKEFILE_OLD) $(FIRST_MAKEFILE) 
	- $(RM_RF) \
	  ./Seq.pm.bak $(DISTVNAME) 


# --- MakeMaker metafile section:
metafile : create_distdir
	$(NOECHO) $(ECHO) Generating META.yml
	$(NOECHO) $(ECHO) '--- #YAML:1.0' > META_new.yml
	$(NOECHO) $(ECHO) 'name:               bioperl' >> META_new.yml
	$(NOECHO) $(ECHO) 'version:            1.2.3' >> META_new.yml
	$(NOECHO) $(ECHO) 'abstract:           Bioinformatics Toolkit' >> META_new.yml
	$(NOECHO) $(ECHO) 'author:' >> META_new.yml
	$(NOECHO) $(ECHO) '    - Bioperl Team (bioperl-l@bioperl.org)' >> META_new.yml
	$(NOECHO) $(ECHO) 'license:            unknown' >> META_new.yml
	$(NOECHO) $(ECHO) 'distribution_type:  module' >> META_new.yml
	$(NOECHO) $(ECHO) 'configure_requires:' >> META_new.yml
	$(NOECHO) $(ECHO) '    ExtUtils::MakeMaker:  0' >> META_new.yml
	$(NOECHO) $(ECHO) 'build_requires:' >> META_new.yml
	$(NOECHO) $(ECHO) '    ExtUtils::MakeMaker:  0' >> META_new.yml
	$(NOECHO) $(ECHO) 'requires:  {}' >> META_new.yml
	$(NOECHO) $(ECHO) 'no_index:' >> META_new.yml
	$(NOECHO) $(ECHO) '    directory:' >> META_new.yml
	$(NOECHO) $(ECHO) '        - t' >> META_new.yml
	$(NOECHO) $(ECHO) '        - inc' >> META_new.yml
	$(NOECHO) $(ECHO) 'generated_by:       ExtUtils::MakeMaker version 6.56' >> META_new.yml
	$(NOECHO) $(ECHO) 'meta-spec:' >> META_new.yml
	$(NOECHO) $(ECHO) '    url:      http://module-build.sourceforge.net/META-spec-v1.4.html' >> META_new.yml
	$(NOECHO) $(ECHO) '    version:  1.4' >> META_new.yml
	-$(NOECHO) $(MV) META_new.yml $(DISTVNAME)/META.yml


# --- MakeMaker signature section:
signature :
	cpansign -s


# --- MakeMaker dist_basics section:
distclean :: realclean distcheck
	$(NOECHO) $(NOOP)

distcheck :
	$(PERLRUN) "-MExtUtils::Manifest=fullcheck" -e fullcheck

skipcheck :
	$(PERLRUN) "-MExtUtils::Manifest=skipcheck" -e skipcheck

manifest :
	$(PERLRUN) "-MExtUtils::Manifest=mkmanifest" -e mkmanifest

veryclean : realclean
	$(RM_F) *~ */*~ *.orig */*.orig *.bak */*.bak *.old */*.old 



# --- MakeMaker dist_core section:

dist : $(DIST_DEFAULT) $(FIRST_MAKEFILE)
	$(NOECHO) $(ABSPERLRUN) -l -e 'print '\''Warning: Makefile possibly out of date with $(VERSION_FROM)'\''' \
	  -e '    if -e '\''$(VERSION_FROM)'\'' and -M '\''$(VERSION_FROM)'\'' < -M '\''$(FIRST_MAKEFILE)'\'';' --

tardist : $(DISTVNAME).tar$(SUFFIX)
	$(NOECHO) $(NOOP)

uutardist : $(DISTVNAME).tar$(SUFFIX)
	uuencode $(DISTVNAME).tar$(SUFFIX) $(DISTVNAME).tar$(SUFFIX) > $(DISTVNAME).tar$(SUFFIX)_uu

$(DISTVNAME).tar$(SUFFIX) : distdir
	$(PREOP)
	$(TO_UNIX)
	$(TAR) $(TARFLAGS) $(DISTVNAME).tar $(DISTVNAME)
	$(RM_RF) $(DISTVNAME)
	$(COMPRESS) $(DISTVNAME).tar
	$(POSTOP)

zipdist : $(DISTVNAME).zip
	$(NOECHO) $(NOOP)

$(DISTVNAME).zip : distdir
	$(PREOP)
	$(ZIP) $(ZIPFLAGS) $(DISTVNAME).zip $(DISTVNAME)
	$(RM_RF) $(DISTVNAME)
	$(POSTOP)

shdist : distdir
	$(PREOP)
	$(SHAR) $(DISTVNAME) > $(DISTVNAME).shar
	$(RM_RF) $(DISTVNAME)
	$(POSTOP)


# --- MakeMaker distdir section:
create_distdir :
	$(RM_RF) $(DISTVNAME)
	$(PERLRUN) "-MExtUtils::Manifest=manicopy,maniread" \
		-e "manicopy(maniread(),'$(DISTVNAME)', '$(DIST_CP)');"

distdir : create_distdir distmeta 
	$(NOECHO) $(NOOP)



# --- MakeMaker dist_test section:
disttest : distdir
	cd $(DISTVNAME) && $(ABSPERLRUN) Makefile.PL 
	cd $(DISTVNAME) && $(MAKE) $(PASTHRU)
	cd $(DISTVNAME) && $(MAKE) test $(PASTHRU)



# --- MakeMaker dist_ci section:

ci :
	$(PERLRUN) "-MExtUtils::Manifest=maniread" \
	  -e "@all = keys %{ maniread() };" \
	  -e "print(qq{Executing $(CI) @all\n}); system(qq{$(CI) @all});" \
	  -e "print(qq{Executing $(RCS_LABEL) ...\n}); system(qq{$(RCS_LABEL) @all});"


# --- MakeMaker distmeta section:
distmeta : create_distdir metafile
	$(NOECHO) cd $(DISTVNAME) && $(ABSPERLRUN) -MExtUtils::Manifest=maniadd -e 'eval { maniadd({q{META.yml} => q{Module meta-data (added by MakeMaker)}}) } ' \
	  -e '    or print "Could not add META.yml to MANIFEST: $${'\''@'\''}\n"' --



# --- MakeMaker distsignature section:
distsignature : create_distdir
	$(NOECHO) cd $(DISTVNAME) && $(ABSPERLRUN) -MExtUtils::Manifest=maniadd -e 'eval { maniadd({q{SIGNATURE} => q{Public-key signature (added by MakeMaker)}}) } ' \
	  -e '    or print "Could not add SIGNATURE to MANIFEST: $${'\''@'\''}\n"' --
	$(NOECHO) cd $(DISTVNAME) && $(TOUCH) SIGNATURE
	cd $(DISTVNAME) && cpansign -s



# --- MakeMaker install section:

install :: pure_install doc_install
	$(NOECHO) $(NOOP)

install_perl :: pure_perl_install doc_perl_install
	$(NOECHO) $(NOOP)

install_site :: pure_site_install doc_site_install
	$(NOECHO) $(NOOP)

install_vendor :: pure_vendor_install doc_vendor_install
	$(NOECHO) $(NOOP)

pure_install :: pure_$(INSTALLDIRS)_install
	$(NOECHO) $(NOOP)

doc_install :: doc_$(INSTALLDIRS)_install
	$(NOECHO) $(NOOP)

pure__install : pure_site_install
	$(NOECHO) $(ECHO) INSTALLDIRS not defined, defaulting to INSTALLDIRS=site

doc__install : doc_site_install
	$(NOECHO) $(ECHO) INSTALLDIRS not defined, defaulting to INSTALLDIRS=site

pure_perl_install :: all
	$(NOECHO) umask 022; $(MOD_INSTALL) \
		$(INST_LIB) $(DESTINSTALLPRIVLIB) \
		$(INST_ARCHLIB) $(DESTINSTALLARCHLIB) \
		$(INST_BIN) $(DESTINSTALLBIN) \
		$(INST_SCRIPT) $(DESTINSTALLSCRIPT) \
		$(INST_MAN1DIR) $(DESTINSTALLMAN1DIR) \
		$(INST_MAN3DIR) $(DESTINSTALLMAN3DIR)
	$(NOECHO) $(WARN_IF_OLD_PACKLIST) \
		$(SITEARCHEXP)/auto/$(FULLEXT)


pure_site_install :: all
	$(NOECHO) umask 02; $(MOD_INSTALL) \
		read $(SITEARCHEXP)/auto/$(FULLEXT)/.packlist \
		write $(DESTINSTALLSITEARCH)/auto/$(FULLEXT)/.packlist \
		$(INST_LIB) $(DESTINSTALLSITELIB) \
		$(INST_ARCHLIB) $(DESTINSTALLSITEARCH) \
		$(INST_BIN) $(DESTINSTALLSITEBIN) \
		$(INST_SCRIPT) $(DESTINSTALLSITESCRIPT) \
		$(INST_MAN1DIR) $(DESTINSTALLSITEMAN1DIR) \
		$(INST_MAN3DIR) $(DESTINSTALLSITEMAN3DIR)
	$(NOECHO) $(WARN_IF_OLD_PACKLIST) \
		$(PERL_ARCHLIB)/auto/$(FULLEXT)

pure_vendor_install :: all
	$(NOECHO) umask 022; $(MOD_INSTALL) \
		$(INST_LIB) $(DESTINSTALLVENDORLIB) \
		$(INST_ARCHLIB) $(DESTINSTALLVENDORARCH) \
		$(INST_BIN) $(DESTINSTALLVENDORBIN) \
		$(INST_SCRIPT) $(DESTINSTALLVENDORSCRIPT) \
		$(INST_MAN1DIR) $(DESTINSTALLVENDORMAN1DIR) \
		$(INST_MAN3DIR) $(DESTINSTALLVENDORMAN3DIR)

doc_perl_install :: all

doc_site_install :: all
	$(NOECHO) $(ECHO) Appending installation info to $(DESTINSTALLSITEARCH)/perllocal.pod
	-$(NOECHO) umask 02; $(MKPATH) $(DESTINSTALLSITEARCH)
	-$(NOECHO) umask 02; $(DOC_INSTALL) \
		"Module" "$(NAME)" \
		"installed into" "$(INSTALLSITELIB)" \
		LINKTYPE "$(LINKTYPE)" \
		VERSION "$(VERSION)" \
		EXE_FILES "$(EXE_FILES)" \
		>> $(DESTINSTALLSITEARCH)/perllocal.pod

doc_vendor_install :: all


uninstall :: uninstall_from_$(INSTALLDIRS)dirs
	$(NOECHO) $(NOOP)

uninstall_from_perldirs ::

uninstall_from_sitedirs ::
	$(NOECHO) $(UNINSTALL) $(SITEARCHEXP)/auto/$(FULLEXT)/.packlist

uninstall_from_vendordirs ::



# --- MakeMaker force section:
# Phony target to force checking subdirectories.
FORCE :
	$(NOECHO) $(NOOP)


# --- MakeMaker perldepend section:


# --- MakeMaker makefile section:
# We take a very conservative approach here, but it's worth it.
# We move Makefile to Makefile.old here to avoid gnu make looping.
$(FIRST_MAKEFILE) : Makefile.PL $(CONFIGDEP)
	$(NOECHO) $(ECHO) "Makefile out-of-date with respect to $?"
	$(NOECHO) $(ECHO) "Cleaning current config before rebuilding Makefile..."
	-$(NOECHO) $(RM_F) $(MAKEFILE_OLD)
	-$(NOECHO) $(MV)   $(FIRST_MAKEFILE) $(MAKEFILE_OLD)
	- $(MAKE) $(USEMAKEFILE) $(MAKEFILE_OLD) clean $(DEV_NULL)
	$(PERLRUN) Makefile.PL 
	$(NOECHO) $(ECHO) "==> Your Makefile has been rebuilt. <=="
	$(NOECHO) $(ECHO) "==> Please rerun the $(MAKE) command.  <=="
	$(FALSE)



# --- MakeMaker staticmake section:

# --- MakeMaker makeaperl section ---
MAP_TARGET    = perl
FULLPERL      = /usr/bin/perl

$(MAP_TARGET) :: static $(MAKE_APERL_FILE)
	$(MAKE) $(USEMAKEFILE) $(MAKE_APERL_FILE) $@

$(MAKE_APERL_FILE) : $(FIRST_MAKEFILE) pm_to_blib
	$(NOECHO) $(ECHO) Writing \"$(MAKE_APERL_FILE)\" for this $(MAP_TARGET)
	$(NOECHO) $(PERLRUNINST) \
		Makefile.PL DIR= \
		MAKEFILE=$(MAKE_APERL_FILE) LINKTYPE=static \
		MAKEAPERL=1 NORECURS=1 CCCDLFLAGS=


# --- MakeMaker test section:

TEST_VERBOSE=0
TEST_TYPE=test_$(LINKTYPE)
TEST_FILE = test.pl
TEST_FILES = t/*.t
TESTDB_SW = -d

testdb :: testdb_$(LINKTYPE)

test :: $(TEST_TYPE) subdirs-test

subdirs-test ::
	$(NOECHO) $(NOOP)


test_dynamic :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERLRUN) "-MExtUtils::Command::MM" "-e" "test_harness($(TEST_VERBOSE), '$(INST_LIB)', '$(INST_ARCHLIB)')" $(TEST_FILES)

testdb_dynamic :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERLRUN) $(TESTDB_SW) "-I$(INST_LIB)" "-I$(INST_ARCHLIB)" $(TEST_FILE)

test_ : test_dynamic

test_static :: test_dynamic
testdb_static :: testdb_dynamic


# --- MakeMaker ppd section:
# Creates a PPD (Perl Package Description) for a binary distribution.
ppd :
	$(NOECHO) $(ECHO) '<SOFTPKG NAME="$(DISTNAME)" VERSION="1.2.3">' > $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '    <ABSTRACT>Bioinformatics Toolkit</ABSTRACT>' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '    <AUTHOR>Bioperl Team (bioperl-l@bioperl.org)</AUTHOR>' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '    <IMPLEMENTATION>' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <ARCHITECTURE NAME="x86_64-linux-gnu-thread-multi-5.12" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <CODEBASE HREF="" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '    </IMPLEMENTATION>' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '</SOFTPKG>' >> $(DISTNAME).ppd


# --- MakeMaker pm_to_blib section:

pm_to_blib : $(FIRST_MAKEFILE) $(TO_INST_PM)
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  Bio/Tools/Blast/CHANGES $(INST_LIB)/Bio/Tools/Blast/CHANGES \
	  Bio/LiveSeq/ChainI.pm $(INST_LIB)/Bio/LiveSeq/ChainI.pm \
	  Bio/AlignIO/mega.pm $(INST_LIB)/Bio/AlignIO/mega.pm \
	  Bio/LiveSeq/Mutation.pm $(INST_LIB)/Bio/LiveSeq/Mutation.pm \
	  Bio/Tools/HMMER/Set.pm $(INST_LIB)/Bio/Tools/HMMER/Set.pm \
	  Bio/Factory/ObjectFactory.pm $(INST_LIB)/Bio/Factory/ObjectFactory.pm \
	  Bio/Factory/BlastResultFactory.pm $(INST_LIB)/Bio/Factory/BlastResultFactory.pm \
	  Bio/Factory/SeqAnalysisParserFactoryI.pm $(INST_LIB)/Bio/Factory/SeqAnalysisParserFactoryI.pm \
	  Bio/DB/GFF/Aggregator/ucsc_unigene.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_unigene.pm \
	  Bio/TreeIO.pm $(INST_LIB)/Bio/TreeIO.pm \
	  Bio/DB/GFF/Aggregator/ucsc_acembly.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_acembly.pm \
	  Bio/Cluster/UniGeneI.pm $(INST_LIB)/Bio/Cluster/UniGeneI.pm \
	  Bio/Biblio/PubmedBookArticle.pm $(INST_LIB)/Bio/Biblio/PubmedBookArticle.pm \
	  Bio/Align/Utilities.pm $(INST_LIB)/Bio/Align/Utilities.pm \
	  Bio/Search/HSP/WABAHSP.pm $(INST_LIB)/Bio/Search/HSP/WABAHSP.pm \
	  Bio/Expression/ProbeI.pm $(INST_LIB)/Bio/Expression/ProbeI.pm \
	  Bio/DB/Flat.pm $(INST_LIB)/Bio/DB/Flat.pm \
	  Bio/Tools/Run/README $(INST_LIB)/Bio/Tools/Run/README \
	  Bio/DB/SwissProt.pm $(INST_LIB)/Bio/DB/SwissProt.pm \
	  Bio/SearchIO/fasta.pm $(INST_LIB)/Bio/SearchIO/fasta.pm \
	  Bio/Biblio/Provider.pm $(INST_LIB)/Bio/Biblio/Provider.pm \
	  Bio/Coordinate/Graph.pm $(INST_LIB)/Bio/Coordinate/Graph.pm \
	  Bio/DB/GFF/Aggregator/ucsc_sanger22.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_sanger22.pm \
	  Bio/Root/RootI.pm $(INST_LIB)/Bio/Root/RootI.pm \
	  Bio/LocationI.pm $(INST_LIB)/Bio/LocationI.pm \
	  Bio/TreeIO/TreeEventBuilder.pm $(INST_LIB)/Bio/TreeIO/TreeEventBuilder.pm \
	  Bio/Taxonomy.pm $(INST_LIB)/Bio/Taxonomy.pm \
	  Bio/Location/Fuzzy.pm $(INST_LIB)/Bio/Location/Fuzzy.pm \
	  Bio/DBLinkContainerI.pm $(INST_LIB)/Bio/DBLinkContainerI.pm \
	  Bio/SeqIO.pm $(INST_LIB)/Bio/SeqIO.pm \
	  Bio/AnalysisI.pm $(INST_LIB)/Bio/AnalysisI.pm \
	  Bio/LiveSeq/AARange.pm $(INST_LIB)/Bio/LiveSeq/AARange.pm \
	  Bio/DB/Biblio/biofetch.pm $(INST_LIB)/Bio/DB/Biblio/biofetch.pm \
	  Bio/Index/Swissprot.pm $(INST_LIB)/Bio/Index/Swissprot.pm \
	  Bio/Annotation/OntologyTerm.pm $(INST_LIB)/Bio/Annotation/OntologyTerm.pm \
	  Bio/Map/SimpleMap.pm $(INST_LIB)/Bio/Map/SimpleMap.pm \
	  Bio/Coordinate/Result/Gap.pm $(INST_LIB)/Bio/Coordinate/Result/Gap.pm \
	  Bio/Tools/Run/RemoteBlast.pm $(INST_LIB)/Bio/Tools/Run/RemoteBlast.pm \
	  Bio/DB/BiblioI.pm $(INST_LIB)/Bio/DB/BiblioI.pm \
	  Bio/SearchIO/Writer/HSPTableWriter.pm $(INST_LIB)/Bio/SearchIO/Writer/HSPTableWriter.pm \
	  Bio/MapIO.pm $(INST_LIB)/Bio/MapIO.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  Bio/Graphics/Glyph/segmented_keyglyph.pm $(INST_LIB)/Bio/Graphics/Glyph/segmented_keyglyph.pm \
	  Bio/SeqFeature/Collection.pm $(INST_LIB)/Bio/SeqFeature/Collection.pm \
	  Bio/Graphics/Glyph/box.pm $(INST_LIB)/Bio/Graphics/Glyph/box.pm \
	  Bio/Graphics/Glyph/cds.pm $(INST_LIB)/Bio/Graphics/Glyph/cds.pm \
	  Bio/Tree/TreeI.pm $(INST_LIB)/Bio/Tree/TreeI.pm \
	  Bio/Graphics/Glyph/extending_arrow.pm $(INST_LIB)/Bio/Graphics/Glyph/extending_arrow.pm \
	  Bio/Map/MappableI.pm $(INST_LIB)/Bio/Map/MappableI.pm \
	  Bio/Seq/BaseSeqProcessor.pm $(INST_LIB)/Bio/Seq/BaseSeqProcessor.pm \
	  Bio/SeqIO/fastq.pm $(INST_LIB)/Bio/SeqIO/fastq.pm \
	  Bio/LiveSeq/SeqI.pm $(INST_LIB)/Bio/LiveSeq/SeqI.pm \
	  Bio/Seq/TraceI.pm $(INST_LIB)/Bio/Seq/TraceI.pm \
	  Bio/Phenotype/Phenotype.pm $(INST_LIB)/Bio/Phenotype/Phenotype.pm \
	  Bio/Seq/PrimaryQual.pm $(INST_LIB)/Bio/Seq/PrimaryQual.pm \
	  Bio/Variation/Allele.pm $(INST_LIB)/Bio/Variation/Allele.pm \
	  Bio/Map/CytoMap.pm $(INST_LIB)/Bio/Map/CytoMap.pm \
	  Bio/DB/Failover.pm $(INST_LIB)/Bio/DB/Failover.pm \
	  Bio/LiveSeq/IO/Loader.pm $(INST_LIB)/Bio/LiveSeq/IO/Loader.pm \
	  Bio/LiveSeq/Repeat_Unit.pm $(INST_LIB)/Bio/LiveSeq/Repeat_Unit.pm \
	  Bio/DB/GFF/Adaptor/memory.pm $(INST_LIB)/Bio/DB/GFF/Adaptor/memory.pm \
	  Bio/DB/UpdateableSeqI.pm $(INST_LIB)/Bio/DB/UpdateableSeqI.pm \
	  Bio/DB/RefSeq.pm $(INST_LIB)/Bio/DB/RefSeq.pm \
	  Bio/Tools/RepeatMasker.pm $(INST_LIB)/Bio/Tools/RepeatMasker.pm \
	  Bio/Coordinate/Result/Match.pm $(INST_LIB)/Bio/Coordinate/Result/Match.pm \
	  Bio/Search/Result/HMMERResult.pm $(INST_LIB)/Bio/Search/Result/HMMERResult.pm \
	  Bio/Seq/PrimedSeq.pm $(INST_LIB)/Bio/Seq/PrimedSeq.pm \
	  Bio/SearchIO/chado.pm $(INST_LIB)/Bio/SearchIO/chado.pm \
	  Bio/Structure/Chain.pm $(INST_LIB)/Bio/Structure/Chain.pm \
	  Bio/AlignIO/prodom.pm $(INST_LIB)/Bio/AlignIO/prodom.pm \
	  Bio/Assembly/ContigAnalysis.pm $(INST_LIB)/Bio/Assembly/ContigAnalysis.pm \
	  Bio/Coordinate/ExtrapolatingPair.pm $(INST_LIB)/Bio/Coordinate/ExtrapolatingPair.pm \
	  Bio/Graphics/Glyph/group.pm $(INST_LIB)/Bio/Graphics/Glyph/group.pm \
	  Bio/IdCollectionI.pm $(INST_LIB)/Bio/IdCollectionI.pm \
	  Bio/Tree/AlleleNode.pm $(INST_LIB)/Bio/Tree/AlleleNode.pm \
	  Bio/Tools/Gel.pm $(INST_LIB)/Bio/Tools/Gel.pm \
	  Bio/DB/NCBIHelper.pm $(INST_LIB)/Bio/DB/NCBIHelper.pm \
	  Bio/Biblio/BiblioBase.pm $(INST_LIB)/Bio/Biblio/BiblioBase.pm \
	  Bio/Biblio/IO.pm $(INST_LIB)/Bio/Biblio/IO.pm \
	  Bio/DB/GFF/Feature.pm $(INST_LIB)/Bio/DB/GFF/Feature.pm \
	  Bio/Phenotype/OMIM/MiniMIMentry.pm $(INST_LIB)/Bio/Phenotype/OMIM/MiniMIMentry.pm \
	  Bio/UpdateableSeqI.pm $(INST_LIB)/Bio/UpdateableSeqI.pm \
	  Bio/SeqIO/chadosxpr.pm $(INST_LIB)/Bio/SeqIO/chadosxpr.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  Bio/Event/EventHandlerI.pm $(INST_LIB)/Bio/Event/EventHandlerI.pm \
	  Bio/Biblio/Thesis.pm $(INST_LIB)/Bio/Biblio/Thesis.pm \
	  Bio/Graphics/Glyph/processed_transcript.pm $(INST_LIB)/Bio/Graphics/Glyph/processed_transcript.pm \
	  Bio/DB/Fasta.pm $(INST_LIB)/Bio/DB/Fasta.pm \
	  Bio/Biblio.pm $(INST_LIB)/Bio/Biblio.pm \
	  Bio/Phenotype/OMIM/OMIMparser.pm $(INST_LIB)/Bio/Phenotype/OMIM/OMIMparser.pm \
	  Bio/DB/GFF/Adaptor/memory_iterator.pm $(INST_LIB)/Bio/DB/GFF/Adaptor/memory_iterator.pm \
	  Bio/SeqIO/game/seqHandler.pm $(INST_LIB)/Bio/SeqIO/game/seqHandler.pm \
	  Bio/Tree/TreeFunctionsI.pm $(INST_LIB)/Bio/Tree/TreeFunctionsI.pm \
	  Bio/Tools/StateMachine/IOStateMachine.pm $(INST_LIB)/Bio/Tools/StateMachine/IOStateMachine.pm \
	  Bio/SeqIO/largefasta.pm $(INST_LIB)/Bio/SeqIO/largefasta.pm \
	  Bio/Annotation/SimpleValue.pm $(INST_LIB)/Bio/Annotation/SimpleValue.pm \
	  Bio/AlignIO/emboss.pm $(INST_LIB)/Bio/AlignIO/emboss.pm \
	  Bio/Align/AlignI.pm $(INST_LIB)/Bio/Align/AlignI.pm \
	  Bio/DB/Query/WebQuery.pm $(INST_LIB)/Bio/DB/Query/WebQuery.pm \
	  Bio/Graphics/Glyph/generic.pm $(INST_LIB)/Bio/Graphics/Glyph/generic.pm \
	  Bio/Ontology/OntologyI.pm $(INST_LIB)/Bio/Ontology/OntologyI.pm \
	  Bio/SearchIO/hmmer.pm $(INST_LIB)/Bio/SearchIO/hmmer.pm \
	  Bio/SeqIO/ctf.pm $(INST_LIB)/Bio/SeqIO/ctf.pm \
	  Bio/LiveSeq/IO/SRS.pm $(INST_LIB)/Bio/LiveSeq/IO/SRS.pm \
	  Bio/Tools/SeqPattern.pm $(INST_LIB)/Bio/Tools/SeqPattern.pm \
	  Bio/LiveSeq/DNA.pm $(INST_LIB)/Bio/LiveSeq/DNA.pm \
	  Bio/Index/AbstractSeq.pm $(INST_LIB)/Bio/Index/AbstractSeq.pm \
	  Bio/AlignIO.pm $(INST_LIB)/Bio/AlignIO.pm \
	  Bio/Tree/NodeI.pm $(INST_LIB)/Bio/Tree/NodeI.pm \
	  Bio/SeqFeature/Gene/Exon.pm $(INST_LIB)/Bio/SeqFeature/Gene/Exon.pm \
	  Bio/Tools/Prints.pm $(INST_LIB)/Bio/Tools/Prints.pm \
	  Bio/Taxonomy/Tree.pm $(INST_LIB)/Bio/Taxonomy/Tree.pm \
	  Bio/Ontology/RelationshipType.pm $(INST_LIB)/Bio/Ontology/RelationshipType.pm \
	  Bio/Cluster/UniGene.pm $(INST_LIB)/Bio/Cluster/UniGene.pm \
	  Bio/SeqFeature/Generic.pm $(INST_LIB)/Bio/SeqFeature/Generic.pm \
	  Bio/Root/Global.pm $(INST_LIB)/Bio/Root/Global.pm \
	  Bio/SeqFeature/SimilarityPair.pm $(INST_LIB)/Bio/SeqFeature/SimilarityPair.pm \
	  Bio/SeqIO/raw.pm $(INST_LIB)/Bio/SeqIO/raw.pm \
	  Bio/Search/Processor.pm $(INST_LIB)/Bio/Search/Processor.pm \
	  Bio/Seq.pm $(INST_LIB)/Bio/Seq.pm \
	  Bio/Biblio/Article.pm $(INST_LIB)/Bio/Biblio/Article.pm \
	  Bio/Tools/HMMER/Results.pm $(INST_LIB)/Bio/Tools/HMMER/Results.pm \
	  Bio/Search/BlastUtils.pm $(INST_LIB)/Bio/Search/BlastUtils.pm \
	  Bio/Biblio/Organisation.pm $(INST_LIB)/Bio/Biblio/Organisation.pm \
	  Bio/Tree/RandomFactory.pm $(INST_LIB)/Bio/Tree/RandomFactory.pm \
	  Bio/Structure/IO/pdb.pm $(INST_LIB)/Bio/Structure/IO/pdb.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  Bio/Tools/Phylo/PAML/Result.pm $(INST_LIB)/Bio/Tools/Phylo/PAML/Result.pm \
	  Bio/Structure/StructureI.pm $(INST_LIB)/Bio/Structure/StructureI.pm \
	  Bio/DescribableI.pm $(INST_LIB)/Bio/DescribableI.pm \
	  Bio/Symbol/AlphabetI.pm $(INST_LIB)/Bio/Symbol/AlphabetI.pm \
	  Bio/Search/Result/ResultI.pm $(INST_LIB)/Bio/Search/Result/ResultI.pm \
	  Bio/Graphics/Glyph/rndrect.pm $(INST_LIB)/Bio/Graphics/Glyph/rndrect.pm \
	  Bio/DB/Flat/OBDAIndex.pm $(INST_LIB)/Bio/DB/Flat/OBDAIndex.pm \
	  Bio/RangeI.pm $(INST_LIB)/Bio/RangeI.pm \
	  Bio/Location/SplitLocationI.pm $(INST_LIB)/Bio/Location/SplitLocationI.pm \
	  Bio/SeqFeature/Gene/NC_Feature.pm $(INST_LIB)/Bio/SeqFeature/Gene/NC_Feature.pm \
	  Bio/Ontology/RelationshipI.pm $(INST_LIB)/Bio/Ontology/RelationshipI.pm \
	  Bio/OntologyIO.pm $(INST_LIB)/Bio/OntologyIO.pm \
	  Bio/Graphics/Feature.pm $(INST_LIB)/Bio/Graphics/Feature.pm \
	  Bio/Index/GenBank.pm $(INST_LIB)/Bio/Index/GenBank.pm \
	  Bio/Tools/StateMachine/AbstractStateMachine.pm $(INST_LIB)/Bio/Tools/StateMachine/AbstractStateMachine.pm \
	  Bio/Biblio/MedlineJournalArticle.pm $(INST_LIB)/Bio/Biblio/MedlineJournalArticle.pm \
	  Bio/DB/GFF/Aggregator/ucsc_sanger22pseudo.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_sanger22pseudo.pm \
	  Bio/Variation/IO/xml.pm $(INST_LIB)/Bio/Variation/IO/xml.pm \
	  Bio/DB/GFF/Aggregator/alignment.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/alignment.pm \
	  Bio/FeatureHolderI.pm $(INST_LIB)/Bio/FeatureHolderI.pm \
	  Bio/SeqFeature/Gene/TranscriptI.pm $(INST_LIB)/Bio/SeqFeature/Gene/TranscriptI.pm \
	  Bio/DB/RandomAccessI.pm $(INST_LIB)/Bio/DB/RandomAccessI.pm \
	  Bio/Cluster/FamilyI.pm $(INST_LIB)/Bio/Cluster/FamilyI.pm \
	  Bio/SeqIO/qual.pm $(INST_LIB)/Bio/SeqIO/qual.pm \
	  Bio/Ontology/TermI.pm $(INST_LIB)/Bio/Ontology/TermI.pm \
	  Bio/SeqIO/pln.pm $(INST_LIB)/Bio/SeqIO/pln.pm \
	  Bio/AnalysisResultI.pm $(INST_LIB)/Bio/AnalysisResultI.pm \
	  Bio/Search/Result/BlastResult.pm $(INST_LIB)/Bio/Search/Result/BlastResult.pm \
	  Bio/Assembly/ScaffoldI.pm $(INST_LIB)/Bio/Assembly/ScaffoldI.pm \
	  Bio/Annotation/AnnotationFactory.pm $(INST_LIB)/Bio/Annotation/AnnotationFactory.pm \
	  Bio/SeqIO/locuslink.pm $(INST_LIB)/Bio/SeqIO/locuslink.pm \
	  Bio/Structure/Atom.pm $(INST_LIB)/Bio/Structure/Atom.pm \
	  Bio/Tools/Sim4/Exon.pm $(INST_LIB)/Bio/Tools/Sim4/Exon.pm \
	  Bio/Factory/BlastHitFactory.pm $(INST_LIB)/Bio/Factory/BlastHitFactory.pm \
	  Bio/Graphics/Glyph/ex.pm $(INST_LIB)/Bio/Graphics/Glyph/ex.pm \
	  Bio/Tools/Phylo/Molphy/Result.pm $(INST_LIB)/Bio/Tools/Phylo/Molphy/Result.pm \
	  Bio/DB/GFF/Aggregator/match.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/match.pm \
	  Bio/DB/GFF/Aggregator/ucsc_softberry.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_softberry.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  Bio/Map/PositionI.pm $(INST_LIB)/Bio/Map/PositionI.pm \
	  Bio/Biblio/BookArticle.pm $(INST_LIB)/Bio/Biblio/BookArticle.pm \
	  Bio/Tools/SeqStats.pm $(INST_LIB)/Bio/Tools/SeqStats.pm \
	  Bio/LiveSeq/Transcript.pm $(INST_LIB)/Bio/LiveSeq/Transcript.pm \
	  Bio/Ontology/OntologyEngineI.pm $(INST_LIB)/Bio/Ontology/OntologyEngineI.pm \
	  Bio/Ontology/Term.pm $(INST_LIB)/Bio/Ontology/Term.pm \
	  Bio/Search/HSP/FastaHSP.pm $(INST_LIB)/Bio/Search/HSP/FastaHSP.pm \
	  Bio/Graphics/Util.pm $(INST_LIB)/Bio/Graphics/Util.pm \
	  Bio/DB/Flat/BDB.pm $(INST_LIB)/Bio/DB/Flat/BDB.pm \
	  Bio/AlignIO/mase.pm $(INST_LIB)/Bio/AlignIO/mase.pm \
	  Bio/Graphics/Glyph/diamond.pm $(INST_LIB)/Bio/Graphics/Glyph/diamond.pm \
	  Bio/SeqFeature/Gene/Transcript.pm $(INST_LIB)/Bio/SeqFeature/Gene/Transcript.pm \
	  Bio/Graphics/Glyph/ellipse.pm $(INST_LIB)/Bio/Graphics/Glyph/ellipse.pm \
	  Bio/SeqIO/scf.pm $(INST_LIB)/Bio/SeqIO/scf.pm \
	  Bio/MapIO/mapmaker.pm $(INST_LIB)/Bio/MapIO/mapmaker.pm \
	  Bio/OntologyIO/InterProParser.pm $(INST_LIB)/Bio/OntologyIO/InterProParser.pm \
	  Bio/Ontology/OntologyStore.pm $(INST_LIB)/Bio/Ontology/OntologyStore.pm \
	  Bio/DB/GFF/Adaptor/dbi/caching_handle.pm $(INST_LIB)/Bio/DB/GFF/Adaptor/dbi/caching_handle.pm \
	  Bio/Location/Atomic.pm $(INST_LIB)/Bio/Location/Atomic.pm \
	  Bio/Ontology/Ontology.pm $(INST_LIB)/Bio/Ontology/Ontology.pm \
	  Bio/SeqIO/fasta.pm $(INST_LIB)/Bio/SeqIO/fasta.pm \
	  Bio/Root/Root.pm $(INST_LIB)/Bio/Root/Root.pm \
	  Bio/Graphics/Glyph/oval.pm $(INST_LIB)/Bio/Graphics/Glyph/oval.pm \
	  Bio/DB/GFF/Aggregator/ucsc_twinscan.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_twinscan.pm \
	  Bio/Graphics/Glyph/anchored_arrow.pm $(INST_LIB)/Bio/Graphics/Glyph/anchored_arrow.pm \
	  Bio/SeqFeature/Similarity.pm $(INST_LIB)/Bio/SeqFeature/Similarity.pm \
	  Bio/Tools/Sigcleave.pm $(INST_LIB)/Bio/Tools/Sigcleave.pm \
	  Bio/Tools/Alignment/Consed.pm $(INST_LIB)/Bio/Tools/Alignment/Consed.pm \
	  Bio/Tools/Alignment/Trim.pm $(INST_LIB)/Bio/Tools/Alignment/Trim.pm \
	  Bio/Phenotype/OMIM/OMIMentry.pm $(INST_LIB)/Bio/Phenotype/OMIM/OMIMentry.pm \
	  Bio/Map/MarkerI.pm $(INST_LIB)/Bio/Map/MarkerI.pm \
	  Bio/SearchIO/waba.pm $(INST_LIB)/Bio/SearchIO/waba.pm \
	  Bio/AlignIO/phylip.pm $(INST_LIB)/Bio/AlignIO/phylip.pm \
	  Bio/Tools/ESTScan.pm $(INST_LIB)/Bio/Tools/ESTScan.pm \
	  Bio/Variation/SeqDiff.pm $(INST_LIB)/Bio/Variation/SeqDiff.pm \
	  Bio/DB/BioFetch.pm $(INST_LIB)/Bio/DB/BioFetch.pm \
	  Bio/SeqIO/phd.pm $(INST_LIB)/Bio/SeqIO/phd.pm \
	  Bio/Factory/LocationFactoryI.pm $(INST_LIB)/Bio/Factory/LocationFactoryI.pm \
	  Bio/SeqFeature/AnnotationAdaptor.pm $(INST_LIB)/Bio/SeqFeature/AnnotationAdaptor.pm \
	  Bio/DB/MANIFEST $(INST_LIB)/Bio/DB/MANIFEST 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  Bio/SeqIO/embl.pm $(INST_LIB)/Bio/SeqIO/embl.pm \
	  Bio/Biblio/IO/medline2ref.pm $(INST_LIB)/Bio/Biblio/IO/medline2ref.pm \
	  Bio/SeqIO/chadoitext.pm $(INST_LIB)/Bio/SeqIO/chadoitext.pm \
	  Bio/Graphics/Glyph/pinsertion.pm $(INST_LIB)/Bio/Graphics/Glyph/pinsertion.pm \
	  Bio/Tools/AnalysisResult.pm $(INST_LIB)/Bio/Tools/AnalysisResult.pm \
	  Bio/Graphics/Glyph/toomany.pm $(INST_LIB)/Bio/Graphics/Glyph/toomany.pm \
	  Bio/Biblio/Journal.pm $(INST_LIB)/Bio/Biblio/Journal.pm \
	  Bio/DB/GFF/Aggregator/transcript.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/transcript.pm \
	  Bio/Search/HSP/GenericHSP.pm $(INST_LIB)/Bio/Search/HSP/GenericHSP.pm \
	  Bio/Coordinate/Result.pm $(INST_LIB)/Bio/Coordinate/Result.pm \
	  Bio/Factory/DriverFactory.pm $(INST_LIB)/Bio/Factory/DriverFactory.pm \
	  Bio/Biblio/PubmedJournalArticle.pm $(INST_LIB)/Bio/Biblio/PubmedJournalArticle.pm \
	  Bio/SeqAnalysisParserI.pm $(INST_LIB)/Bio/SeqAnalysisParserI.pm \
	  Bio/Variation/RNAChange.pm $(INST_LIB)/Bio/Variation/RNAChange.pm \
	  Bio/Factory/ObjectFactoryI.pm $(INST_LIB)/Bio/Factory/ObjectFactoryI.pm \
	  Bio/Search/Result/WABAResult.pm $(INST_LIB)/Bio/Search/Result/WABAResult.pm \
	  Bio/SearchIO.pm $(INST_LIB)/Bio/SearchIO.pm \
	  Bio/AnalysisParserI.pm $(INST_LIB)/Bio/AnalysisParserI.pm \
	  Bio/SearchDist.pm $(INST_LIB)/Bio/SearchDist.pm \
	  Bio/Expression/FeatureGroup/FeatureGroupMas50.pm $(INST_LIB)/Bio/Expression/FeatureGroup/FeatureGroupMas50.pm \
	  Bio/Tools/ECnumber.pm $(INST_LIB)/Bio/Tools/ECnumber.pm \
	  Bio/LiveSeq/Range.pm $(INST_LIB)/Bio/LiveSeq/Range.pm \
	  Bio/Tools/Pseudowise.pm $(INST_LIB)/Bio/Tools/Pseudowise.pm \
	  Bio/Tools/Genscan.pm $(INST_LIB)/Bio/Tools/Genscan.pm \
	  Bio/OntologyIO/soflat.pm $(INST_LIB)/Bio/OntologyIO/soflat.pm \
	  Bio/Tools/GFF.pm $(INST_LIB)/Bio/Tools/GFF.pm \
	  Bio/Tools/Blast.pm $(INST_LIB)/Bio/Tools/Blast.pm \
	  Bio/DB/GFF/Util/Rearrange.pm $(INST_LIB)/Bio/DB/GFF/Util/Rearrange.pm \
	  Bio/AnnotationCollectionI.pm $(INST_LIB)/Bio/AnnotationCollectionI.pm \
	  Bio/Das/SegmentI.pm $(INST_LIB)/Bio/Das/SegmentI.pm \
	  Bio/SeqFeatureI.pm $(INST_LIB)/Bio/SeqFeatureI.pm \
	  Bio/Das/FeatureTypeI.pm $(INST_LIB)/Bio/Das/FeatureTypeI.pm \
	  Bio/Graphics/Glyph/graded_segments.pm $(INST_LIB)/Bio/Graphics/Glyph/graded_segments.pm \
	  bptutorial.pl $(INST_LIB)/bptutorial.pl \
	  Bio/Graphics/Glyph/dna.pm $(INST_LIB)/Bio/Graphics/Glyph/dna.pm \
	  Bio/Structure/Model.pm $(INST_LIB)/Bio/Structure/Model.pm \
	  Bio/Seq/SeqFastaSpeedFactory.pm $(INST_LIB)/Bio/Seq/SeqFastaSpeedFactory.pm \
	  Bio/Tree/Node.pm $(INST_LIB)/Bio/Tree/Node.pm \
	  Bio/LiveSeq/Repeat_Region.pm $(INST_LIB)/Bio/LiveSeq/Repeat_Region.pm \
	  Bio/Annotation/Reference.pm $(INST_LIB)/Bio/Annotation/Reference.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  Bio/Coordinate/MapperI.pm $(INST_LIB)/Bio/Coordinate/MapperI.pm \
	  Bio/DB/WebDBSeqI.pm $(INST_LIB)/Bio/DB/WebDBSeqI.pm \
	  Bio/TreeIO/tabtree.pm $(INST_LIB)/Bio/TreeIO/tabtree.pm \
	  Bio/Tools/Primer3.pm $(INST_LIB)/Bio/Tools/Primer3.pm \
	  Bio/Seq/SeqFactory.pm $(INST_LIB)/Bio/Seq/SeqFactory.pm \
	  Bio/Tools/Seg.pm $(INST_LIB)/Bio/Tools/Seg.pm \
	  Bio/Search/Result/ResultFactory.pm $(INST_LIB)/Bio/Search/Result/ResultFactory.pm \
	  Bio/Phenotype/OMIM/OMIMentryAllelicVariant.pm $(INST_LIB)/Bio/Phenotype/OMIM/OMIMentryAllelicVariant.pm \
	  Bio/Tools/Phylo/Phylip/ProtDist.pm $(INST_LIB)/Bio/Tools/Phylo/Phylip/ProtDist.pm \
	  Bio/SeqIO/genbank.pm $(INST_LIB)/Bio/SeqIO/genbank.pm \
	  Bio/PrimarySeq.pm $(INST_LIB)/Bio/PrimarySeq.pm \
	  Bio/AlignIO/pfam.pm $(INST_LIB)/Bio/AlignIO/pfam.pm \
	  Bio/AlignIO/nexus.pm $(INST_LIB)/Bio/AlignIO/nexus.pm \
	  Bio/DB/GFF/Adaptor/dbi.pm $(INST_LIB)/Bio/DB/GFF/Adaptor/dbi.pm \
	  Bio/DB/GFF/Aggregator/none.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/none.pm \
	  Bio/Tools/Signalp.pm $(INST_LIB)/Bio/Tools/Signalp.pm \
	  Bio/Location/CoordinatePolicyI.pm $(INST_LIB)/Bio/Location/CoordinatePolicyI.pm \
	  LocalConfig.pm $(INST_LIB)/LocalConfig.pm \
	  Bio/AlignIO/bl2seq.pm $(INST_LIB)/Bio/AlignIO/bl2seq.pm \
	  Bio/Root/HTTPget.pm $(INST_LIB)/Bio/Root/HTTPget.pm \
	  Bio/Ontology/SimpleOntologyEngine.pm $(INST_LIB)/Bio/Ontology/SimpleOntologyEngine.pm \
	  Bio/Tools/BPbl2seq.pm $(INST_LIB)/Bio/Tools/BPbl2seq.pm \
	  Bio/Variation/IO.pm $(INST_LIB)/Bio/Variation/IO.pm \
	  Bio/Ontology/PathI.pm $(INST_LIB)/Bio/Ontology/PathI.pm \
	  Bio/AnnotationI.pm $(INST_LIB)/Bio/AnnotationI.pm \
	  Bio/Tools/WWW.pm $(INST_LIB)/Bio/Tools/WWW.pm \
	  Bio/Seq/LargePrimarySeq.pm $(INST_LIB)/Bio/Seq/LargePrimarySeq.pm \
	  Bio/Annotation/Collection.pm $(INST_LIB)/Bio/Annotation/Collection.pm \
	  Bio/Tools/IUPAC.pm $(INST_LIB)/Bio/Tools/IUPAC.pm \
	  Bio/Graphics/Glyph/triangle.pm $(INST_LIB)/Bio/Graphics/Glyph/triangle.pm \
	  Bio/Graphics.pm $(INST_LIB)/Bio/Graphics.pm \
	  Bio/SeqIO/abi.pm $(INST_LIB)/Bio/SeqIO/abi.pm \
	  Bio/DB/GFF/Aggregator/ucsc_refgene.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_refgene.pm \
	  Bio/AlignIO/msf.pm $(INST_LIB)/Bio/AlignIO/msf.pm \
	  Bio/LiveSeq/Intron.pm $(INST_LIB)/Bio/LiveSeq/Intron.pm \
	  Bio/DB/GFF/Aggregator/clone.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/clone.pm \
	  Bio/Map/Position.pm $(INST_LIB)/Bio/Map/Position.pm \
	  Bio/PrimarySeqI.pm $(INST_LIB)/Bio/PrimarySeqI.pm \
	  Bio/Graphics/Glyph/alignment.pm $(INST_LIB)/Bio/Graphics/Glyph/alignment.pm \
	  Bio/Root/IOManager.pm $(INST_LIB)/Bio/Root/IOManager.pm \
	  Bio/LiveSeq/Gene.pm $(INST_LIB)/Bio/LiveSeq/Gene.pm \
	  Bio/Ontology/Relationship.pm $(INST_LIB)/Bio/Ontology/Relationship.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  Bio/Search/Hit/HitFactory.pm $(INST_LIB)/Bio/Search/Hit/HitFactory.pm \
	  Bio/SearchIO/exonerate.pm $(INST_LIB)/Bio/SearchIO/exonerate.pm \
	  Bio/Factory/HitFactoryI.pm $(INST_LIB)/Bio/Factory/HitFactoryI.pm \
	  Bio/SimpleAlign.pm $(INST_LIB)/Bio/SimpleAlign.pm \
	  Bio/Tools/Blast/Sbjct.pm $(INST_LIB)/Bio/Tools/Blast/Sbjct.pm \
	  Bio/Tools/Phylo/PAML.pm $(INST_LIB)/Bio/Tools/Phylo/PAML.pm \
	  Bio/Factory/ApplicationFactoryI.pm $(INST_LIB)/Bio/Factory/ApplicationFactoryI.pm \
	  Bio/DB/GFF/Typename.pm $(INST_LIB)/Bio/DB/GFF/Typename.pm \
	  Bio/Search/HSP/HSPFactory.pm $(INST_LIB)/Bio/Search/HSP/HSPFactory.pm \
	  Bio/DB/GFF/Homol.pm $(INST_LIB)/Bio/DB/GFF/Homol.pm \
	  Bio/Event/EventGeneratorI.pm $(INST_LIB)/Bio/Event/EventGeneratorI.pm \
	  Bio/AlignIO/psi.pm $(INST_LIB)/Bio/AlignIO/psi.pm \
	  Bio/Graphics/Glyph/redgreen_box.pm $(INST_LIB)/Bio/Graphics/Glyph/redgreen_box.pm \
	  Bio/Tools/BPlite/Sbjct.pm $(INST_LIB)/Bio/Tools/BPlite/Sbjct.pm \
	  Bio/Graphics/FeatureFile.pm $(INST_LIB)/Bio/Graphics/FeatureFile.pm \
	  Bio/Map/Microsatellite.pm $(INST_LIB)/Bio/Map/Microsatellite.pm \
	  Bio/LiveSeq/Translation.pm $(INST_LIB)/Bio/LiveSeq/Translation.pm \
	  Bio/Search/Hit/HitI.pm $(INST_LIB)/Bio/Search/Hit/HitI.pm \
	  Bio/DB/SeqI.pm $(INST_LIB)/Bio/DB/SeqI.pm \
	  Bio/Index/SwissPfam.pm $(INST_LIB)/Bio/Index/SwissPfam.pm \
	  Bio/DB/GenBank.pm $(INST_LIB)/Bio/DB/GenBank.pm \
	  Bio/DB/Flat/BDB/fasta.pm $(INST_LIB)/Bio/DB/Flat/BDB/fasta.pm \
	  Bio/Seq/SequenceTrace.pm $(INST_LIB)/Bio/Seq/SequenceTrace.pm \
	  Bio/Tools/BPlite/HSP.pm $(INST_LIB)/Bio/Tools/BPlite/HSP.pm \
	  Bio/SeqIO/chado.pm $(INST_LIB)/Bio/SeqIO/chado.pm \
	  Bio/Index/EMBL.pm $(INST_LIB)/Bio/Index/EMBL.pm \
	  Bio/DB/GFF/Aggregator/coding.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/coding.pm \
	  Bio/SeqFeature/Gene/GeneStructureI.pm $(INST_LIB)/Bio/SeqFeature/Gene/GeneStructureI.pm \
	  Bio/ClusterIO.pm $(INST_LIB)/Bio/ClusterIO.pm \
	  Bio/DB/EMBL.pm $(INST_LIB)/Bio/DB/EMBL.pm \
	  Bio/DB/DBFetch.pm $(INST_LIB)/Bio/DB/DBFetch.pm \
	  Bio/Tools/Blast/README $(INST_LIB)/Bio/Tools/Blast/README \
	  Bio/Map/CytoPosition.pm $(INST_LIB)/Bio/Map/CytoPosition.pm \
	  Bio/Phenotype/Correlate.pm $(INST_LIB)/Bio/Phenotype/Correlate.pm \
	  Bio/Graphics/Glyph/Factory.pm $(INST_LIB)/Bio/Graphics/Glyph/Factory.pm \
	  Bio/Assembly/IO/phrap.pm $(INST_LIB)/Bio/Assembly/IO/phrap.pm \
	  Bio/Ontology/Path.pm $(INST_LIB)/Bio/Ontology/Path.pm \
	  Bio/LiveSeq/Prim_Transcript.pm $(INST_LIB)/Bio/LiveSeq/Prim_Transcript.pm \
	  Bio/Tools/Promoterwise.pm $(INST_LIB)/Bio/Tools/Promoterwise.pm \
	  Bio/Tree/NodeNHX.pm $(INST_LIB)/Bio/Tree/NodeNHX.pm \
	  Bio/AlignIO/clustalw.pm $(INST_LIB)/Bio/AlignIO/clustalw.pm \
	  Bio/DB/Ace.pm $(INST_LIB)/Bio/DB/Ace.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  Bio/DB/GFF/Adaptor/dbi/mysql.pm $(INST_LIB)/Bio/DB/GFF/Adaptor/dbi/mysql.pm \
	  Bio/Factory/FTLocationFactory.pm $(INST_LIB)/Bio/Factory/FTLocationFactory.pm \
	  Bio/Align/DNAStatistics.pm $(INST_LIB)/Bio/Align/DNAStatistics.pm \
	  Bio/Coordinate/Utils.pm $(INST_LIB)/Bio/Coordinate/Utils.pm \
	  Bio/Root/Vector.pm $(INST_LIB)/Bio/Root/Vector.pm \
	  Bio/LiveSeq/IO/BioPerl.pm $(INST_LIB)/Bio/LiveSeq/IO/BioPerl.pm \
	  Bio/Variation/IO/flat.pm $(INST_LIB)/Bio/Variation/IO/flat.pm \
	  Bio/SeqIO/chadoxml.pm $(INST_LIB)/Bio/SeqIO/chadoxml.pm \
	  Bio/Biblio/Ref.pm $(INST_LIB)/Bio/Biblio/Ref.pm \
	  Bio/DB/Query/GenBank.pm $(INST_LIB)/Bio/DB/Query/GenBank.pm \
	  Bio/Search/GenericDatabase.pm $(INST_LIB)/Bio/Search/GenericDatabase.pm \
	  Bio/Factory/SequenceProcessorI.pm $(INST_LIB)/Bio/Factory/SequenceProcessorI.pm \
	  Bio/Location/AvWithinCoordPolicy.pm $(INST_LIB)/Bio/Location/AvWithinCoordPolicy.pm \
	  Bio/Biblio/TechReport.pm $(INST_LIB)/Bio/Biblio/TechReport.pm \
	  Bio/Structure/SecStr/DSSP/Res.pm $(INST_LIB)/Bio/Structure/SecStr/DSSP/Res.pm \
	  Bio/Graphics/Glyph/translation.pm $(INST_LIB)/Bio/Graphics/Glyph/translation.pm \
	  Bio/SeqI.pm $(INST_LIB)/Bio/SeqI.pm \
	  Bio/Location/Simple.pm $(INST_LIB)/Bio/Location/Simple.pm \
	  Bio/SeqIO/gcg.pm $(INST_LIB)/Bio/SeqIO/gcg.pm \
	  Bio/SeqIO/alf.pm $(INST_LIB)/Bio/SeqIO/alf.pm \
	  Bio/SearchIO/SearchResultEventBuilder.pm $(INST_LIB)/Bio/SearchIO/SearchResultEventBuilder.pm \
	  Bio/Seq/QualI.pm $(INST_LIB)/Bio/Seq/QualI.pm \
	  Bio/Map/LinkagePosition.pm $(INST_LIB)/Bio/Map/LinkagePosition.pm \
	  Bio/Ontology/SimpleGOEngine.pm $(INST_LIB)/Bio/Ontology/SimpleGOEngine.pm \
	  Bio/SearchIO/psiblast.pm $(INST_LIB)/Bio/SearchIO/psiblast.pm \
	  Bio/Cluster/ClusterFactory.pm $(INST_LIB)/Bio/Cluster/ClusterFactory.pm \
	  Bio/SeqFeature/Gene/Poly_A_site.pm $(INST_LIB)/Bio/SeqFeature/Gene/Poly_A_site.pm \
	  Bio/SeqFeature/Gene/ExonI.pm $(INST_LIB)/Bio/SeqFeature/Gene/ExonI.pm \
	  Bio/Tools/Hmmpfam.pm $(INST_LIB)/Bio/Tools/Hmmpfam.pm \
	  Bio/Expression/FeatureGroup.pm $(INST_LIB)/Bio/Expression/FeatureGroup.pm \
	  Bio/Graphics/Glyph/primers.pm $(INST_LIB)/Bio/Graphics/Glyph/primers.pm \
	  Bio/DB/GFF/Aggregator/ucsc_genscan.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_genscan.pm \
	  Bio/IdentifiableI.pm $(INST_LIB)/Bio/IdentifiableI.pm \
	  Bio/Graphics/Glyph/heterogeneous_segments.pm $(INST_LIB)/Bio/Graphics/Glyph/heterogeneous_segments.pm \
	  Bio/Ontology/InterProTerm.pm $(INST_LIB)/Bio/Ontology/InterProTerm.pm \
	  Bio/SeqIO/game/featureHandler.pm $(INST_LIB)/Bio/SeqIO/game/featureHandler.pm \
	  Bio/Graphics/Glyph/crossbox.pm $(INST_LIB)/Bio/Graphics/Glyph/crossbox.pm \
	  Bio/Coordinate/Chain.pm $(INST_LIB)/Bio/Coordinate/Chain.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  Bio/Tools/Grail.pm $(INST_LIB)/Bio/Tools/Grail.pm \
	  Bio/Tools/Blast/HTML.pm $(INST_LIB)/Bio/Tools/Blast/HTML.pm \
	  Bio/DB/InMemoryCache.pm $(INST_LIB)/Bio/DB/InMemoryCache.pm \
	  Bio/DB/GFF/Segment.pm $(INST_LIB)/Bio/DB/GFF/Segment.pm \
	  Bio/Symbol/DNAAlphabet.pm $(INST_LIB)/Bio/Symbol/DNAAlphabet.pm \
	  Bio/OntologyIO/goflat.pm $(INST_LIB)/Bio/OntologyIO/goflat.pm \
	  Bio/Tools/Blast/HSP.pm $(INST_LIB)/Bio/Tools/Blast/HSP.pm \
	  Bio/DB/FileCache.pm $(INST_LIB)/Bio/DB/FileCache.pm \
	  Bio/DB/GFF/Util/Binning.pm $(INST_LIB)/Bio/DB/GFF/Util/Binning.pm \
	  Bio/SearchIO/blast.pm $(INST_LIB)/Bio/SearchIO/blast.pm \
	  Bio/Species.pm $(INST_LIB)/Bio/Species.pm \
	  Bio/Biblio/Person.pm $(INST_LIB)/Bio/Biblio/Person.pm \
	  Bio/Map/Marker.pm $(INST_LIB)/Bio/Map/Marker.pm \
	  Bio/Tools/CodonTable.pm $(INST_LIB)/Bio/Tools/CodonTable.pm \
	  Bio/SeqIO/bsml.pm $(INST_LIB)/Bio/SeqIO/bsml.pm \
	  Bio/Tools/AlignFactory.pm $(INST_LIB)/Bio/Tools/AlignFactory.pm \
	  Bio/DB/QueryI.pm $(INST_LIB)/Bio/DB/QueryI.pm \
	  Bio/Graphics/Glyph/segments.pm $(INST_LIB)/Bio/Graphics/Glyph/segments.pm \
	  Bio/Seq/SeqWithQuality.pm $(INST_LIB)/Bio/Seq/SeqWithQuality.pm \
	  Bio/Annotation/DBLink.pm $(INST_LIB)/Bio/Annotation/DBLink.pm \
	  Bio/SearchIO/FastHitEventBuilder.pm $(INST_LIB)/Bio/SearchIO/FastHitEventBuilder.pm \
	  Bio/DB/XEMBL.pm $(INST_LIB)/Bio/DB/XEMBL.pm \
	  Bio/DB/GFF/Adaptor/dbi/mysqlopt.pm $(INST_LIB)/Bio/DB/GFF/Adaptor/dbi/mysqlopt.pm \
	  Bio/Search/Hit/BlastHit.pm $(INST_LIB)/Bio/Search/Hit/BlastHit.pm \
	  Bio/Graphics/Glyph/redgreen_segment.pm $(INST_LIB)/Bio/Graphics/Glyph/redgreen_segment.pm \
	  Bio/Variation/AAReverseMutate.pm $(INST_LIB)/Bio/Variation/AAReverseMutate.pm \
	  Bio/Tools/Prediction/Gene.pm $(INST_LIB)/Bio/Tools/Prediction/Gene.pm \
	  Bio/SeqFeature/Primer.pm $(INST_LIB)/Bio/SeqFeature/Primer.pm \
	  Bio/Search/SearchUtils.pm $(INST_LIB)/Bio/Search/SearchUtils.pm \
	  Bio/Biblio/WebResource.pm $(INST_LIB)/Bio/Biblio/WebResource.pm \
	  Bio/AlignIO/fasta.pm $(INST_LIB)/Bio/AlignIO/fasta.pm \
	  Bio/Tools/Phylo/Molphy.pm $(INST_LIB)/Bio/Tools/Phylo/Molphy.pm \
	  Bio/Search/Hit/HMMERHit.pm $(INST_LIB)/Bio/Search/Hit/HMMERHit.pm \
	  Bio/Tools/Run/StandAloneBlast.pm $(INST_LIB)/Bio/Tools/Run/StandAloneBlast.pm \
	  Bio/Root/Err.pm $(INST_LIB)/Bio/Root/Err.pm \
	  Bio/Tools/MZEF.pm $(INST_LIB)/Bio/Tools/MZEF.pm \
	  Bio/SearchIO/Writer/ResultTableWriter.pm $(INST_LIB)/Bio/SearchIO/Writer/ResultTableWriter.pm \
	  Bio/Ontology/RelationshipFactory.pm $(INST_LIB)/Bio/Ontology/RelationshipFactory.pm \
	  Bio/Seq/RichSeqI.pm $(INST_LIB)/Bio/Seq/RichSeqI.pm \
	  Bio/Tools/Prediction/Exon.pm $(INST_LIB)/Bio/Tools/Prediction/Exon.pm \
	  Bio/SearchIO/EventHandlerI.pm $(INST_LIB)/Bio/SearchIO/EventHandlerI.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  Bio/SearchIO/Writer/HitTableWriter.pm $(INST_LIB)/Bio/SearchIO/Writer/HitTableWriter.pm \
	  Bio/DB/GFF/Adaptor/biofetch.pm $(INST_LIB)/Bio/DB/GFF/Adaptor/biofetch.pm \
	  Bio/Tools/Est2Genome.pm $(INST_LIB)/Bio/Tools/Est2Genome.pm \
	  Bio/SearchIO/chadosxpr.pm $(INST_LIB)/Bio/SearchIO/chadosxpr.pm \
	  Bio/Search/HSP/HMMERHSP.pm $(INST_LIB)/Bio/Search/HSP/HMMERHSP.pm \
	  Bio/Variation/DNAMutation.pm $(INST_LIB)/Bio/Variation/DNAMutation.pm \
	  Bio/Factory/MapFactoryI.pm $(INST_LIB)/Bio/Factory/MapFactoryI.pm \
	  Bio/Structure/IO.pm $(INST_LIB)/Bio/Structure/IO.pm \
	  Bio/Tree/Tree.pm $(INST_LIB)/Bio/Tree/Tree.pm \
	  Bio/Variation/SNP.pm $(INST_LIB)/Bio/Variation/SNP.pm \
	  Bio/SearchIO/SearchWriterI.pm $(INST_LIB)/Bio/SearchIO/SearchWriterI.pm \
	  Bio/Factory/AnalysisI.pm $(INST_LIB)/Bio/Factory/AnalysisI.pm \
	  Bio/SeqFeature/Gene/Promoter.pm $(INST_LIB)/Bio/SeqFeature/Gene/Promoter.pm \
	  Bio/SeqFeature/FeaturePair.pm $(INST_LIB)/Bio/SeqFeature/FeaturePair.pm \
	  Bio/Graphics/Glyph/transcript2.pm $(INST_LIB)/Bio/Graphics/Glyph/transcript2.pm \
	  Bio/Location/Split.pm $(INST_LIB)/Bio/Location/Split.pm \
	  Bio/SeqIO/exp.pm $(INST_LIB)/Bio/SeqIO/exp.pm \
	  Bio/Graphics/Glyph/transcript.pm $(INST_LIB)/Bio/Graphics/Glyph/transcript.pm \
	  Bio/Biblio/IO/pubmedxml.pm $(INST_LIB)/Bio/Biblio/IO/pubmedxml.pm \
	  Bio/Search/Result/GenericResult.pm $(INST_LIB)/Bio/Search/Result/GenericResult.pm \
	  Bio/Cluster/SequenceFamily.pm $(INST_LIB)/Bio/Cluster/SequenceFamily.pm \
	  Bio/SeqIO/swiss.pm $(INST_LIB)/Bio/SeqIO/swiss.pm \
	  Bio/DB/Makefile.PL $(INST_LIB)/Bio/DB/Makefile.PL \
	  Bio/ClusterIO/unigene.pm $(INST_LIB)/Bio/ClusterIO/unigene.pm \
	  Bio/DB/XEMBLService.pm $(INST_LIB)/Bio/DB/XEMBLService.pm \
	  Bio/Annotation/StructuredValue.pm $(INST_LIB)/Bio/Annotation/StructuredValue.pm \
	  Bio/Tools/FootPrinter.pm $(INST_LIB)/Bio/Tools/FootPrinter.pm \
	  Bio/SeqIO/FTHelper.pm $(INST_LIB)/Bio/SeqIO/FTHelper.pm \
	  Bio/Graphics/Glyph/dot.pm $(INST_LIB)/Bio/Graphics/Glyph/dot.pm \
	  Bio/Tools/Coil.pm $(INST_LIB)/Bio/Tools/Coil.pm \
	  Bio/Coordinate/Collection.pm $(INST_LIB)/Bio/Coordinate/Collection.pm \
	  Bio/Root/IO.pm $(INST_LIB)/Bio/Root/IO.pm \
	  Bio/Tools/Lucy.pm $(INST_LIB)/Bio/Tools/Lucy.pm \
	  Bio/Biblio/MedlineBook.pm $(INST_LIB)/Bio/Biblio/MedlineBook.pm \
	  Bio/Symbol/Symbol.pm $(INST_LIB)/Bio/Symbol/Symbol.pm \
	  Bio/Ontology/GOterm.pm $(INST_LIB)/Bio/Ontology/GOterm.pm \
	  Bio/DB/GFF/Featname.pm $(INST_LIB)/Bio/DB/GFF/Featname.pm \
	  Bio/Symbol/Alphabet.pm $(INST_LIB)/Bio/Symbol/Alphabet.pm \
	  Bio/Graphics/Glyph/xyplot.pm $(INST_LIB)/Bio/Graphics/Glyph/xyplot.pm \
	  Bio/LocatableSeq.pm $(INST_LIB)/Bio/LocatableSeq.pm \
	  Bio/Factory/ObjectBuilderI.pm $(INST_LIB)/Bio/Factory/ObjectBuilderI.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  Bio/SeqIO/ztr.pm $(INST_LIB)/Bio/SeqIO/ztr.pm \
	  Bio/DB/Universal.pm $(INST_LIB)/Bio/DB/Universal.pm \
	  Bio/Tools/OddCodes.pm $(INST_LIB)/Bio/Tools/OddCodes.pm \
	  Bio/Structure/Entry.pm $(INST_LIB)/Bio/Structure/Entry.pm \
	  Bio/Biblio/JournalArticle.pm $(INST_LIB)/Bio/Biblio/JournalArticle.pm \
	  Bio/Coordinate/Pair.pm $(INST_LIB)/Bio/Coordinate/Pair.pm \
	  Bio/DB/GFF/Aggregator/processed_transcript.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/processed_transcript.pm \
	  Bio/Coordinate/ResultI.pm $(INST_LIB)/Bio/Coordinate/ResultI.pm \
	  Bio/Graphics/Glyph/arrow.pm $(INST_LIB)/Bio/Graphics/Glyph/arrow.pm \
	  Bio/Graphics/Panel.pm $(INST_LIB)/Bio/Graphics/Panel.pm \
	  Bio/ClusterI.pm $(INST_LIB)/Bio/ClusterI.pm \
	  Bio/Index/Fasta.pm $(INST_LIB)/Bio/Index/Fasta.pm \
	  Bio/Align/StatisticsI.pm $(INST_LIB)/Bio/Align/StatisticsI.pm \
	  Bio/SeqFeature/PositionProxy.pm $(INST_LIB)/Bio/SeqFeature/PositionProxy.pm \
	  Bio/Tools/SeqAnal.pm $(INST_LIB)/Bio/Tools/SeqAnal.pm \
	  Bio/LiveSeq/IO/README $(INST_LIB)/Bio/LiveSeq/IO/README \
	  Bio/DB/GFF/Adaptor/ace.pm $(INST_LIB)/Bio/DB/GFF/Adaptor/ace.pm \
	  Bio/Structure/SecStr/STRIDE/Res.pm $(INST_LIB)/Bio/Structure/SecStr/STRIDE/Res.pm \
	  Bio/Biblio/MedlineBookArticle.pm $(INST_LIB)/Bio/Biblio/MedlineBookArticle.pm \
	  Bio/Graphics/Glyph/ruler_arrow.pm $(INST_LIB)/Bio/Graphics/Glyph/ruler_arrow.pm \
	  Bio/Tools/Genemark.pm $(INST_LIB)/Bio/Tools/Genemark.pm \
	  Bio/Biblio/Patent.pm $(INST_LIB)/Bio/Biblio/Patent.pm \
	  Bio/Seq/RichSeq.pm $(INST_LIB)/Bio/Seq/RichSeq.pm \
	  Bio/DB/GFF/Aggregator.pm $(INST_LIB)/Bio/DB/GFF/Aggregator.pm \
	  Bio/DB/GFF.pm $(INST_LIB)/Bio/DB/GFF.pm \
	  Bio/Tools/BPpsilite.pm $(INST_LIB)/Bio/Tools/BPpsilite.pm \
	  Bio/Graphics/Glyph/track.pm $(INST_LIB)/Bio/Graphics/Glyph/track.pm \
	  Bio/Factory/ResultFactoryI.pm $(INST_LIB)/Bio/Factory/ResultFactoryI.pm \
	  Bio/SeqFeature/CollectionI.pm $(INST_LIB)/Bio/SeqFeature/CollectionI.pm \
	  Bio/Biblio/MedlineArticle.pm $(INST_LIB)/Bio/Biblio/MedlineArticle.pm \
	  Bio/Phenotype/PhenotypeI.pm $(INST_LIB)/Bio/Phenotype/PhenotypeI.pm \
	  Bio/Taxonomy/Taxon.pm $(INST_LIB)/Bio/Taxonomy/Taxon.pm \
	  Bio/Seq/LargeSeq.pm $(INST_LIB)/Bio/Seq/LargeSeq.pm \
	  Bio/DB/GDB.pm $(INST_LIB)/Bio/DB/GDB.pm \
	  Bio/Tools/Profile.pm $(INST_LIB)/Bio/Tools/Profile.pm \
	  Bio/Seq/EncodedSeq.pm $(INST_LIB)/Bio/Seq/EncodedSeq.pm \
	  Bio/Factory/EMBOSS.pm $(INST_LIB)/Bio/Factory/EMBOSS.pm \
	  Bio/Variation/README $(INST_LIB)/Bio/Variation/README \
	  Bio/DB/Registry.pm $(INST_LIB)/Bio/DB/Registry.pm \
	  Bio/Variation/VariantI.pm $(INST_LIB)/Bio/Variation/VariantI.pm \
	  Bio/Root/Utilities.pm $(INST_LIB)/Bio/Root/Utilities.pm \
	  Bio/Graphics/Glyph/line.pm $(INST_LIB)/Bio/Graphics/Glyph/line.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  Bio/Factory/SequenceFactoryI.pm $(INST_LIB)/Bio/Factory/SequenceFactoryI.pm \
	  Bio/Search/Hit/GenericHit.pm $(INST_LIB)/Bio/Search/Hit/GenericHit.pm \
	  Bio/Tools/RestrictionEnzyme.pm $(INST_LIB)/Bio/Tools/RestrictionEnzyme.pm \
	  Bio/Map/LinkageMap.pm $(INST_LIB)/Bio/Map/LinkageMap.pm \
	  Bio/AlignIO/stockholm.pm $(INST_LIB)/Bio/AlignIO/stockholm.pm \
	  Bio/OntologyIO/Handlers/InterProHandler.pm $(INST_LIB)/Bio/OntologyIO/Handlers/InterProHandler.pm \
	  Bio/DB/GFF/Adaptor/dbi/mysqlace.pm $(INST_LIB)/Bio/DB/GFF/Adaptor/dbi/mysqlace.pm \
	  Bio/SeqFeature/Computation.pm $(INST_LIB)/Bio/SeqFeature/Computation.pm \
	  Bio/DasI.pm $(INST_LIB)/Bio/DasI.pm \
	  Bio/Graphics/Glyph/splice_site.pm $(INST_LIB)/Bio/Graphics/Glyph/splice_site.pm \
	  Bio/TreeIO/nhx.pm $(INST_LIB)/Bio/TreeIO/nhx.pm \
	  Bio/SeqUtils.pm $(INST_LIB)/Bio/SeqUtils.pm \
	  Bio/Assembly/Scaffold.pm $(INST_LIB)/Bio/Assembly/Scaffold.pm \
	  Bio/Assembly/Contig.pm $(INST_LIB)/Bio/Assembly/Contig.pm \
	  Bio/Structure/Residue.pm $(INST_LIB)/Bio/Structure/Residue.pm \
	  Bio/Tools/Sim4/Results.pm $(INST_LIB)/Bio/Tools/Sim4/Results.pm \
	  Bio/DB/GFF/Adaptor/dbi/iterator.pm $(INST_LIB)/Bio/DB/GFF/Adaptor/dbi/iterator.pm \
	  Bio/Variation/AAChange.pm $(INST_LIB)/Bio/Variation/AAChange.pm \
	  Bio/Index/Fastq.pm $(INST_LIB)/Bio/Index/Fastq.pm \
	  Bio/SearchIO/Writer/TextResultWriter.pm $(INST_LIB)/Bio/SearchIO/Writer/TextResultWriter.pm \
	  Bio/Range.pm $(INST_LIB)/Bio/Range.pm \
	  Bio/SeqIO/game/idHandler.pm $(INST_LIB)/Bio/SeqIO/game/idHandler.pm \
	  Bio/OntologyIO/dagflat.pm $(INST_LIB)/Bio/OntologyIO/dagflat.pm \
	  Bio/SeqIO/pir.pm $(INST_LIB)/Bio/SeqIO/pir.pm \
	  Bio/Factory/SequenceStreamI.pm $(INST_LIB)/Bio/Factory/SequenceStreamI.pm \
	  Bio/Matrix/PhylipDist.pm $(INST_LIB)/Bio/Matrix/PhylipDist.pm \
	  Bio/Annotation/Comment.pm $(INST_LIB)/Bio/Annotation/Comment.pm \
	  Bio/Tools/BPlite.pm $(INST_LIB)/Bio/Tools/BPlite.pm \
	  Bio/Symbol/ProteinAlphabet.pm $(INST_LIB)/Bio/Symbol/ProteinAlphabet.pm \
	  Bio/Tools/Genewise.pm $(INST_LIB)/Bio/Tools/Genewise.pm \
	  Bio/Biblio/PubmedArticle.pm $(INST_LIB)/Bio/Biblio/PubmedArticle.pm \
	  Bio/Perl.pm $(INST_LIB)/Bio/Perl.pm \
	  Bio/Search/HSP/HSPI.pm $(INST_LIB)/Bio/Search/HSP/HSPI.pm \
	  Bio/LiveSeq/Exon.pm $(INST_LIB)/Bio/LiveSeq/Exon.pm \
	  Bio/LiveSeq/Chain.pm $(INST_LIB)/Bio/LiveSeq/Chain.pm \
	  Bio/ClusterIO/dbsnp.pm $(INST_LIB)/Bio/ClusterIO/dbsnp.pm \
	  Bio/Root/Xref.pm $(INST_LIB)/Bio/Root/Xref.pm \
	  Bio/Tools/Run/WrapperBase.pm $(INST_LIB)/Bio/Tools/Run/WrapperBase.pm \
	  Bio/AlignIO/meme.pm $(INST_LIB)/Bio/AlignIO/meme.pm \
	  Bio/Map/MapI.pm $(INST_LIB)/Bio/Map/MapI.pm \
	  Bio/Location/WidestCoordPolicy.pm $(INST_LIB)/Bio/Location/WidestCoordPolicy.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  Bio/Tools/Genomewise.pm $(INST_LIB)/Bio/Tools/Genomewise.pm \
	  Bio/DB/Flat/BDB/embl.pm $(INST_LIB)/Bio/DB/Flat/BDB/embl.pm \
	  Bio/Search/HSP/BlastHSP.pm $(INST_LIB)/Bio/Search/HSP/BlastHSP.pm \
	  Bio/LiveSeq/Mutator.pm $(INST_LIB)/Bio/LiveSeq/Mutator.pm \
	  Bio/Phenotype/Measure.pm $(INST_LIB)/Bio/Phenotype/Measure.pm \
	  Bio/SeqFeature/Gene/Intron.pm $(INST_LIB)/Bio/SeqFeature/Gene/Intron.pm \
	  Bio/Tools/Eponine.pm $(INST_LIB)/Bio/Tools/Eponine.pm \
	  Bio/DB/GFF/RelSegment.pm $(INST_LIB)/Bio/DB/GFF/RelSegment.pm \
	  Bio/Annotation/TypeManager.pm $(INST_LIB)/Bio/Annotation/TypeManager.pm \
	  Bio/DB/GFF/Aggregator/ucsc_ensgene.pm $(INST_LIB)/Bio/DB/GFF/Aggregator/ucsc_ensgene.pm \
	  Bio/Map/OrderedPosition.pm $(INST_LIB)/Bio/Map/OrderedPosition.pm \
	  Bio/SeqIO/MultiFile.pm $(INST_LIB)/Bio/SeqIO/MultiFile.pm \
	  Bio/Tools/Tmhmm.pm $(INST_LIB)/Bio/Tools/Tmhmm.pm \
	  Bio/Factory/SeqAnalysisParserFactory.pm $(INST_LIB)/Bio/Factory/SeqAnalysisParserFactory.pm \
	  Bio/SeqFeature/Gene/UTR.pm $(INST_LIB)/Bio/SeqFeature/Gene/UTR.pm \
	  Bio/Map/CytoMarker.pm $(INST_LIB)/Bio/Map/CytoMarker.pm \
	  Bio/AlignIO/selex.pm $(INST_LIB)/Bio/AlignIO/selex.pm \
	  Bio/Seq/SeqBuilder.pm $(INST_LIB)/Bio/Seq/SeqBuilder.pm \
	  Bio/Biblio/Service.pm $(INST_LIB)/Bio/Biblio/Service.pm \
	  Bio/Biblio/MedlineJournal.pm $(INST_LIB)/Bio/Biblio/MedlineJournal.pm \
	  Bio/DB/GenPept.pm $(INST_LIB)/Bio/DB/GenPept.pm \
	  Bio/Index/Blast.pm $(INST_LIB)/Bio/Index/Blast.pm \
	  Bio/Search/DatabaseI.pm $(INST_LIB)/Bio/Search/DatabaseI.pm \
	  Bio/Graphics/Glyph.pm $(INST_LIB)/Bio/Graphics/Glyph.pm \
	  Bio/Symbol/SymbolI.pm $(INST_LIB)/Bio/Symbol/SymbolI.pm \
	  Bio/Tools/EPCR.pm $(INST_LIB)/Bio/Tools/EPCR.pm \
	  Bio/Location/NarrowestCoordPolicy.pm $(INST_LIB)/Bio/Location/NarrowestCoordPolicy.pm \
	  Bio/Tools/pSW.pm $(INST_LIB)/Bio/Tools/pSW.pm \
	  Bio/AnnotatableI.pm $(INST_LIB)/Bio/AnnotatableI.pm \
	  Bio/Biblio/Proceeding.pm $(INST_LIB)/Bio/Biblio/Proceeding.pm \
	  Bio/TreeIO/newick.pm $(INST_LIB)/Bio/TreeIO/newick.pm \
	  Bio/SeqFeature/Gene/GeneStructure.pm $(INST_LIB)/Bio/SeqFeature/Gene/GeneStructure.pm \
	  Bio/Biblio/Book.pm $(INST_LIB)/Bio/Biblio/Book.pm \
	  Bio/Tools/BPlite/Iteration.pm $(INST_LIB)/Bio/Tools/BPlite/Iteration.pm \
	  Bio/SearchIO/Writer/HTMLResultWriter.pm $(INST_LIB)/Bio/SearchIO/Writer/HTMLResultWriter.pm \
	  Bio/Tree/Statistics.pm $(INST_LIB)/Bio/Tree/Statistics.pm \
	  Bio/Align/PairwiseStatistics.pm $(INST_LIB)/Bio/Align/PairwiseStatistics.pm \
	  Bio/Assembly/IO.pm $(INST_LIB)/Bio/Assembly/IO.pm \
	  Bio/Graphics/Glyph/span.pm $(INST_LIB)/Bio/Graphics/Glyph/span.pm \
	  Bio/Ontology/TermFactory.pm $(INST_LIB)/Bio/Ontology/TermFactory.pm \
	  Bio/Tools/SeqWords.pm $(INST_LIB)/Bio/Tools/SeqWords.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  Bio/DB/GFF/Adaptor/dbi/oracleace.pm $(INST_LIB)/Bio/DB/GFF/Adaptor/dbi/oracleace.pm \
	  Bio/Expression/FeatureI.pm $(INST_LIB)/Bio/Expression/FeatureI.pm \
	  Bio/Map/OrderedPositionWithDistance.pm $(INST_LIB)/Bio/Map/OrderedPositionWithDistance.pm \
	  Bio/SeqIO/game.pm $(INST_LIB)/Bio/SeqIO/game.pm \
	  Bio/Biblio/IO/pubmed2ref.pm $(INST_LIB)/Bio/Biblio/IO/pubmed2ref.pm \
	  Bio/Index/Abstract.pm $(INST_LIB)/Bio/Index/Abstract.pm \
	  Bio/Tools/HMMER/Domain.pm $(INST_LIB)/Bio/Tools/HMMER/Domain.pm \
	  Bio/SearchIO/blastxml.pm $(INST_LIB)/Bio/SearchIO/blastxml.pm \
	  Bio/DB/GFF/Adaptor/dbi/oracle.pm $(INST_LIB)/Bio/DB/GFF/Adaptor/dbi/oracle.pm \
	  Bio/Coordinate/GeneMapper.pm $(INST_LIB)/Bio/Coordinate/GeneMapper.pm \
	  Bio/Biblio/IO/medlinexml.pm $(INST_LIB)/Bio/Biblio/IO/medlinexml.pm \
	  Bio/SeqIO/ace.pm $(INST_LIB)/Bio/SeqIO/ace.pm \
	  Bio/Location/FuzzyLocationI.pm $(INST_LIB)/Bio/Location/FuzzyLocationI.pm \
	  Bio/Search/Hit/Fasta.pm $(INST_LIB)/Bio/Search/Hit/Fasta.pm \
	  Bio/Root/Exception.pm $(INST_LIB)/Bio/Root/Exception.pm \
	  Bio/Factory/TreeFactoryI.pm $(INST_LIB)/Bio/Factory/TreeFactoryI.pm \
	  Bio/Assembly/IO/ace.pm $(INST_LIB)/Bio/Assembly/IO/ace.pm \
	  Bio/DB/Biblio/soap.pm $(INST_LIB)/Bio/DB/Biblio/soap.pm \
	  Bio/Root/Object.pm $(INST_LIB)/Bio/Root/Object.pm 
	$(NOECHO) $(TOUCH) pm_to_blib


# --- MakeMaker selfdocument section:


# --- MakeMaker postamble section:
show_tests :
	@echo 'type make test_<subtest> to run'
	@echo 'BioGraphics \
OMIMentry \
Alphabet \
UniGene \
SeqAnalysisParser \
BiblioReferences \
SequenceFamily \
AAReverseMutate \
MicrosatelliteMarker \
EMBL_DB \
Symbol \
Exonerate \
largepseq \
primaryqual \
primer3 \
Map \
BlastIndex \
GFF \
RepeatMasker \
Tools \
phd \
trim \
Molphy \
StructIO \
Biblio \
Sim4 \
qual \
Relationship \
Species \
LocusLink \
XEMBL_DB \
simpleGOparser \
IUPAC \
MiniMIMentry \
WABA \
Biblio_biofetch \
StateMachine \
Mutation \
BioDBGFF \
Correlate \
Allele \
SeqFeature \
primedseq \
consed \
Seq \
MapIO \
RandomTreeFactory \
largefasta \
LocatableSeq \
DBFasta \
Ontology \
Swiss \
RefSeq \
PrimarySeq \
RangeI \
GDB \
Genomewise \
ProtDist \
ePCR \
GOterm \
Genpred \
lucy \
SeqIO \
SeqBuilder \
RelationshipType \
seqwithquality \
BPpsilite \
Variation_IO \
Annotation \
Chain \
multiple_fasta \
GeneCoordinateMapper \
Range \
Phenotype \
Index \
LiveSeq \
Assembly \
game \
OntologyEngine \
SimpleAlign \
CodonTable \
seqfeaturePrimer \
SearchDist \
AlignIO \
LinkageMap \
hmmer \
RNAChange \
SimilarityPair \
Location \
Perl \
sequencetrace \
Term \
FootPrinter \
Mutator \
CoordinateGraph \
StandAloneBlast \
BioFetch_DB \
DB \
Exception \
Node \
AnnotationAdaptor \
PAML \
SeqStats \
est2genome \
Tempfile \
OddCodes \
Tree \
Measure \
EncodedSeq \
SeqPattern \
scf \
OMIMentryAllelicVariant \
SeqUtils \
splicedseq \
SeqDiff \
BPlite \
CoordinateMapper \
LocationFactory \
flat \
ECnumber \
SeqFeatCollection \
Genewise \
PhylipDist \
CytoMap \
DNAMutation \
Structure \
ClusterIO \
Sigcleave \
BPbl2seq \
OMIMparser \
RootIO \
AAChange \
AlignStats \
RestrictionEnzyme \
RootI \
'
test_BioGraphics :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/BioGraphics.t
test_OMIMentry :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/OMIMentry.t
test_Alphabet :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Alphabet.t
test_UniGene :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/UniGene.t
test_SeqAnalysisParser :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqAnalysisParser.t
test_BiblioReferences :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/BiblioReferences.t
test_SequenceFamily :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SequenceFamily.t
test_AAReverseMutate :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/AAReverseMutate.t
test_MicrosatelliteMarker :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/MicrosatelliteMarker.t
test_EMBL_DB :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/EMBL_DB.t
test_Symbol :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Symbol.t
test_Exonerate :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Exonerate.t
test_largepseq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/largepseq.t
test_primaryqual :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/primaryqual.t
test_primer3 :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/primer3.t
test_Map :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Map.t
test_BlastIndex :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/BlastIndex.t
test_GFF :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/GFF.t
test_RepeatMasker :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/RepeatMasker.t
test_Tools :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Tools.t
test_phd :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/phd.t
test_trim :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/trim.t
test_Molphy :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Molphy.t
test_StructIO :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/StructIO.t
test_Biblio :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Biblio.t
test_Sim4 :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Sim4.t
test_qual :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/qual.t
test_Relationship :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Relationship.t
test_Species :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Species.t
test_LocusLink :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/LocusLink.t
test_XEMBL_DB :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/XEMBL_DB.t
test_simpleGOparser :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/simpleGOparser.t
test_IUPAC :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/IUPAC.t
test_MiniMIMentry :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/MiniMIMentry.t
test_WABA :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/WABA.t
test_Biblio_biofetch :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Biblio_biofetch.t
test_StateMachine :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/StateMachine.t
test_Mutation :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Mutation.t
test_BioDBGFF :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/BioDBGFF.t
test_Correlate :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Correlate.t
test_Allele :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Allele.t
test_SeqFeature :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqFeature.t
test_primedseq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/primedseq.t
test_consed :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/consed.t
test_Seq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Seq.t
test_MapIO :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/MapIO.t
test_RandomTreeFactory :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/RandomTreeFactory.t
test_largefasta :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/largefasta.t
test_LocatableSeq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/LocatableSeq.t
test_DBFasta :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/DBFasta.t
test_Ontology :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Ontology.t
test_Swiss :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Swiss.t
test_RefSeq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/RefSeq.t
test_PrimarySeq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/PrimarySeq.t
test_RangeI :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/RangeI.t
test_GDB :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/GDB.t
test_Genomewise :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Genomewise.t
test_ProtDist :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/ProtDist.t
test_ePCR :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/ePCR.t
test_GOterm :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/GOterm.t
test_Genpred :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Genpred.t
test_lucy :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/lucy.t
test_SeqIO :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqIO.t
test_SeqBuilder :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqBuilder.t
test_RelationshipType :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/RelationshipType.t
test_seqwithquality :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/seqwithquality.t
test_BPpsilite :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/BPpsilite.t
test_Variation_IO :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Variation_IO.t
test_Annotation :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Annotation.t
test_Chain :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Chain.t
test_multiple_fasta :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/multiple_fasta.t
test_GeneCoordinateMapper :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/GeneCoordinateMapper.t
test_Range :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Range.t
test_Phenotype :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Phenotype.t
test_Index :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Index.t
test_LiveSeq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/LiveSeq.t
test_Assembly :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Assembly.t
test_game :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/game.t
test_OntologyEngine :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/OntologyEngine.t
test_SimpleAlign :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SimpleAlign.t
test_CodonTable :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/CodonTable.t
test_seqfeaturePrimer :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/seqfeaturePrimer.t
test_SearchDist :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SearchDist.t
test_AlignIO :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/AlignIO.t
test_LinkageMap :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/LinkageMap.t
test_hmmer :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/hmmer.t
test_RNAChange :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/RNAChange.t
test_SimilarityPair :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SimilarityPair.t
test_Location :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Location.t
test_Perl :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Perl.t
test_sequencetrace :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/sequencetrace.t
test_Term :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Term.t
test_FootPrinter :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/FootPrinter.t
test_Mutator :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Mutator.t
test_CoordinateGraph :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/CoordinateGraph.t
test_StandAloneBlast :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/StandAloneBlast.t
test_BioFetch_DB :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/BioFetch_DB.t
test_DB :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/DB.t
test_Exception :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Exception.t
test_Node :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Node.t
test_AnnotationAdaptor :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/AnnotationAdaptor.t
test_PAML :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/PAML.t
test_SeqStats :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqStats.t
test_est2genome :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/est2genome.t
test_Tempfile :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Tempfile.t
test_OddCodes :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/OddCodes.t
test_Tree :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Tree.t
test_Measure :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Measure.t
test_EncodedSeq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/EncodedSeq.t
test_SeqPattern :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqPattern.t
test_scf :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/scf.t
test_OMIMentryAllelicVariant :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/OMIMentryAllelicVariant.t
test_SeqUtils :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqUtils.t
test_splicedseq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/splicedseq.t
test_SeqDiff :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqDiff.t
test_BPlite :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/BPlite.t
test_CoordinateMapper :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/CoordinateMapper.t
test_LocationFactory :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/LocationFactory.t
test_flat :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/flat.t
test_ECnumber :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/ECnumber.t
test_SeqFeatCollection :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqFeatCollection.t
test_Genewise :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Genewise.t
test_PhylipDist :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/PhylipDist.t
test_CytoMap :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/CytoMap.t
test_DNAMutation :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/DNAMutation.t
test_Structure :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Structure.t
test_ClusterIO :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/ClusterIO.t
test_Sigcleave :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Sigcleave.t
test_BPbl2seq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/BPbl2seq.t
test_OMIMparser :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/OMIMparser.t
test_RootIO :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/RootIO.t
test_AAChange :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/AAChange.t
test_AlignStats :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/AlignStats.t
test_RestrictionEnzyme :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/RestrictionEnzyme.t
test_RootI :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/RootI.t



# End.
